---
output: pdf_document
fontsize: 12pts
textalign: left  # this does not work...
geometry: margin = 1in
bibliography: library.bib
csl: systematic-biology.csl
link-citations: yes
# indent: true # this works
# header-includes:
#    - \setlength\parindent{24pt}  # this works too, but I'mg onna indent manually with 4 spaces
# nope, 4 spaces indents are not recognized, they do something else
linestretch: 2
header-includes:
    - \usepackage[left]{lineno}
    - \linenumbers
---
```{r eval = TRUE, echo = FALSE}
figure_runtime_main <- "Fig. 1"  # make_datelife_query_plus_get_datelife_result
```

Running head: DATELIFE: REVEALING THE DATED TREE OF LIFE <!--MAX 50 CHARACTERS, ALL CAPITALS-->

Title: DateLife: Leveraging databases and analytical tools to reveal the dated Tree of Life

Authors: Luna L. Sánchez-Reyes^1^, Brian C. O’Meara^1^

Correspondence address:

1. *Department of Ecology and Evolutionary Biology, University of Tennessee, Knoxville, 425 Hesler Biology Building, Knoxville, TN 37996, USA*

Corresponding authors: sanchez.reyes.luna@gmail.com, bomeara@utk.edu


\newpage

**abstract.-** Here goes the abstract.

**Keywords:** Tree; Phylogeny; Scaling; Open; Ages; Congruify; Supertree;


\newpage

<!--INTRODUCTION-->
Divergence time of lineages<!--named biological entities, taxon ages?--> constitutes in many ways the main knowledge necessary for evolutionary understanding.
<!--Along with phylogenetic relationships, it is the most important piece of information allowing to test alternative evolutionary hypothesis in many areas of research, from developmental to ecological questions.
Along with migration rates and patterns, it is fundamental to test biogeographical hypothesis-->
Coupled to species number and distribution, it is the basic information for the study of diversification processes, such as the tempo and mode of speciation and extinction, crucial for the understanding of how biodiversity patterns are shaped across space and time [@Morlon2014].

When organisms are preserved in a fossil form, a time frame of taxon origin can be obtained directly from the age of rock strata.
However, not all organisms fossilize well or at all.
Fossilization success alone is highly circumstancial, and varies depending on a number of parameters including the nature of the habitat, population size, species range breadth and physical characteristics of the organism. <!--fossil recovery is also highly circumstancial-->
Thus, relying only on the fossil record to obtain a time frame of lineage divergence<!--taxon ages--> for all life is<!--, alas,--> not possible.
Relative rate of DNA or aminoacid substitution constitutes another important source of information on lineage divergence. <!--describe the two components of rel subst rate: which, contain tne information on absolute rate of substitution and absolute time.-->
It is usually obtained from hypothesis of character homology (alignments) when reconstructing phylogenetic relationships. Molecular dating techniques use external data such as absolute time calibrations (e.g., fossils, geologic events) or absolute substitution rates <!--birds--> to generate dated phylogenies (chronograms) which contain information on absolute times of node divergence and taxon ages. <!--With molecular dating methods, an external calibration point is still necessary to estimate the absolute time of origin. Hence, the accumulation of data on fossils for calibration is also continual-->
<!--This method has been particularly useful to obtain time frame of origin of organisms with poor or no fossil record, but is also largely used for organisms with rich fossil records-->
<!--Models allow to take into account the nature of the fossil record: Given the characteristics of the fossilization process, the quality of dates inferred directly from it is not always considered the best.-->

In the past decades, the possibility to obtain DNA sequences in large quantities from a wide variety of organisms became a reality, which, coupled to methodological development in phylogenetic and dating inference, allowed the application of molecular dating methods on a very large amount and diversity of organisms, greatly increasing the quantity of<!--organisms with--> data on taxon ages across the tree of life.
<!-- figure showing growth on studies with age data from references table from ttol-->
To date, there is a large amount of both fossil and molecular-based data on taxon ages and phylogenetic relationships in public repositories such as Dryad, TreeBASE and Open Tree of Life (OToL).
OToL alone holds more than 200 chronograms. <!--cache updated jan 15 2018
devtools::load_all(pkg = "~/Desktop/datelife")
length(opentree_chronograms$trees)
names(opentree_chronograms)
-->
<!--How many trees? How many dated trees?-->
Methods to include living and fossil lineages are in continued development and increased usage by the community<!--total evidence and revbayes (fossilized birth-death)-->, which coupled to better sharing data practices, are greatly contributing to the accumulation in number and type of available data on taxon ages.

<!--What led to datelife development?
Describe public and research necessities covered by datelife
It is important to use available data on time frame of lineage origin:
To know the state of dating for a group of interest:
What range of estimated ages exist already? Are fossil and molecular
time frames coherent? (e.g., [@Magallon2015c]).
To construct a time tree of life.
For science communication, improve scientific discussions, time-framing
other events of importance in other research areas.
Comparative method: autocorrelation (phylogenetic relatedeness) corrections, phylogenetic signal
-->
One inspiration for the DateLife project was the TimeTree project [@Hedges2006; @Hedges2015; @Kumar2017]. This project has aggregated chronograms from 3,163 studies, encompassing 97,085 species [@Kumar2017], and continues to grow. However, even in this gold standard resource, the included taxa only include between 0.097 and 3.236% of total species diversity (following taxonomic expert opinion on the global, extant species numbers, which ranges from 3 to 100 million species [Mayr2010; Moran2011]). One advantage of TimeTree is that it includes taxa from across the tree of life, versus more specialized chronograms focusing on plants [PHYLOMATIC], birds [JETZ ET AL BIRDTREE.ORG], and other groups. Users can choose between a web interface or a mobile app to receive information on divergence times for the evolutionary history of a lineage, pairs of taxa, all lineages within a taxon, or a list of taxa. As a science communication tool, TimeTree project is a very powerful tool: it has a friendly graphical interface, with informative while colorful outputs, that allows the general public to satisfy curiosity regarding their favorite organism or group of them. It is of limited utility for scientific studies, however. The thousands of trees that have been entered are unavailable for examination or reuse; according to their FAQ, methods for allowing download of the data have been under discussion since 2015. Also, there is no Application Programming Interface (API) allowing programmatic access to any data. Moreover, large-scale, automated, data-mining is not allowed under the site's terms of use. The nearly hundred thousand taxon chronogram is not available with [@Kumar2017] or the TimeTree website, though the still substantial chronogram from [@Hedges2015] is available at OToL. 

With this inspiration, a prototype DateLife service was developed over a series of phylotastic hackathons [__CITE__] at the National Evolutionary Synthesis Center. NSF funding allowed for the further development leading to this paper. A core goal has been openness of both the data sources and the code underlying the analyses.


<!--
[@Mora2011]
eukaryotes and prokaryotes:
on earth: 1.24 million catalogued, 8.74 million predicted
on the oceans: 0.19 million catalogued, 2.21 predicted

Total ssp number estimated to be between 3 and 100 million species, as mentiones in [@Mora2011]:
ttol_spp <- 97085
(ttol_spp * 100) / 3e+6  # 3.236167
(ttol_spp * 100) / 100e+6  # 0.097085
-->
<!--
While its node-time search is very powerful, using available data to infer divergence time from lineages that have never been sequenced before, its timetree option is less powerful, their main flaw being that it does not provide source data.
However, their primary database is not fully open, giving information on a per node basis. So, it is not possible to easily access information from original studies for a group of taxa (One would need to perform a pair of taxa search for all pairs from the group of interest).
Divergence time estimates for all taxa in the TToL are precomputed and stored in the TimeTree database
All searches allow species names and higher group names. Common names are allowed in all searches but a list of taxa.
one-taxon search was nor working for dog, though.
1) ttol one lineage: it gives the evolutionary timeline of a lineage, showing the age of main divergence events occuring along the lineage evolutionary history, i.e., when it shared a comon ancestor with all main lineages from which it diverged a long time ago.
It shows results only graphically on the website.
2) ttol parwise search: allows common names, species names and higher group names
    how it handles names missing from tree: it looks in the NCBI taxonomy for the closest relative of the species or higher taxa. Then uses the closest relative to get the lineage pair age from ttol.
3) ttol group of taxa
4) ttol list of taxa: Cannot get it to work; does not work with cat, dog, turtle search, felidae, hominidae, phrynops search, some_ants species name search. What kind of files does it accepts: txt, rtf, doc, docx?
-->

<!--
retrieves information takes available data on lineage ages and phylogenetic relationships to construct a tree of all life scaled to absolute time: a chronogram of all life, while allowing the general public to search for age data within the tree for one taxon of interest (timeline option), a pair of taxa (node time option), a group of taxa (it can only be a single name of a taxonomic group), or a list of taxa of interest (timetree option for the two latter).
Good things:
Allows to get ages of species with no data, by extrapolating from closely related species with data.
Allows comparison of timelines to geologic events, such as asteroid impacts, oxygen and carbon dioxide levels, solar luminosity.
Not so good things:
Only allows to download some data from original sources if performing a pair-of-taxa search.
Only allows to see age ranges or confidence intervals in graphical form, when hovering over nodes. These CIs come from ttol. It does not have CIs from original studies. Node hovering is activated on one-taxon evolutionary history, and group-of-taxa searches (we could not get to work list-of-taxa search). Pair-of-taxa search shows the confidence interval of the only ttol node that constitutes the taxon-pair mrca.
You can only download data from pair-of-taxa and group-of-taxa search. It generates a table with ages from original studies and a tree pruned from ttol, respectively. In both cases it gives single point age estimates. We cannot download CI information from ttol.
-->

Despite its importance, analytical tools to summarize available information on taxon ages for the scientific community are still lacking.
Data might not have been exploited because:
Data is in different repositories and formats;
Lineage names are different among studies and difficult to reconcile;
Taxonomy is also different among studies and difficult to reconcile.
<!--Data curation is necessary at some point (is this always true?)-->
Also, data curation is an important part of any biological study. The research community considers it as an important or even crucial step before data analysis. Hence, automated processes for large data analysis are frequently received with skepticism.
Recent work on this area (e.g., supersmart<!--and, which others?-->) aims to:
Generate new dates using all available DNA sequence information;
Perform one global analysis using all available information;
Problems or downsides: This might be time consuming for large groups and a lot of data curation is still necessary.
<!--General issues with dating techniques:-->
Choosing correct fossils for calibration requires a lot of expertise and knowledge on the group.
Incorrect use of fossils can generate severe bias in dating results [@Sauquet2012c].
DateLife palliates this by using only information available from already published studies, which are ideally constructed using robust information, such as sequence data and curated fossil calibrations.
<!--Data curation is largely based on taxonomic knowledge; or at least on the interplay between taxonomy and phylogenetics-->
<!--Importance of datelife:-->
DateLife can summarize this information in several formats that can be easily inspected by users.
This allows rapidly obtaining a time frame of lineage divergence for a wide number of taxa.
DateLife can also generate chronograms for taxa with little available information, by using the available data as calibration points.
DateLife is the main service for scaling phylogenetic trees in Phylotastic! system [@Stoltzfus2013] <!--to construct phylogenetic trees on the fly [http://phylotastic.org/].-->
It can be used through an R package <link to documentation>, a web interface (<http://www.datelife.org/query/>) and an API<!--(still not up, right?)-->.


<!--DESCRIPTION of Datelife-->
<span style="font-variant:small-caps;">
Description
</span>

<!--(BOLD and OToL species names are homogeneous?-->

DateLife is a service for searching and processing information on ages of any taxon <!--or group of taxa-->of interest, available in chronograms from public data repositories of published peer reviewed studies. With this data, it can also generate new taxon age information<!--by using expert available phylogenies and scaling them to absolute time using information on ages obtained from published chronograms,--> by calling a variety of external services and proved methods.
It only requires as input a set of taxon names, in the form of a comma separated listing or vector, or of a phylogeny with taxon names on the tips.
Taxon names can correspond to binomial species names or clades<!--common names (will we implement it?)-->.
When taxon names are clades, DateLife pulls all accepted species names within the clade (up to OToL's limit of _____ species) from OToL's reference taxonomy using a service of rphylotastic R package<!--citation-->.
Names belonging to subspecies or any other infraspecific category are treated as species.
DateLife can process input names with the taxon name resolution service (TNRS), <!--which "corrects misspelled names and authorities, standardizes variant spellings, and converts nomenclatural synonyms to accepted names" (Boyle et al. 2013)-->which corrects misspelled names or typos, and standardizes variation in spelling and synonyms [@Boyle2013], increasing the probability to correctly find the queried taxa in the chronogram database. DateLife uses TNRS to compare names against OToL's reference taxonomy using a service from the R package rotl [@Michonneau2016]. <!--we'll implement the option to use other reference taxonomies in the future -->

DateLife's main function searches taxon names across the chronogram database specified by the user. At the moment, it queries chronograms from the OToL [@Hinchliff2015]<!-- there is an earlier reference from 1994 but it does not exist in its journal database: Sanderson MJ, Donoghue MJ, Piel WH, Eriksson T: TreeBASE: a prototype database of phylogenetic analyses and an interactive tool for browsing the phylogeny of life. Am J Bot. 1994, 81 (6): 183--> repository.
DateLife identifies chronograms having at least two taxon names, and subset them to contain only the taxa of interest. It then stores taxon age information from each chronogram individually as a patristic matrix, named with the citation of the original study. This format allows a rapid summary in a number of different ways, including: 1) citations of the original studies containing the subset chronograms, 2) a list of subset chronograms' mrca ages, 3) a list of subset chronograms in newick or phylo format, 4) a table containing all information in html or R's data frame format, or 5) as a single chronogram summarized from subset chronograms using the supertree Super Distance Matrix (SDM) approach [@Criscuolo2006] or using the median of branch lengths. <!--(how to better explain the latter?? Also, I just discovered the median gives non ultrametric trees...).-->

DateLife also stores information on input taxon presence/absence across subset chronograms. Users can choose to add ages of missing taxa to subset chronograms in different ways, depending on the amount of knowledge they want to input or how much they want to be involved in the steps of the addition process.
If users have no access to biological information (i.e., a character, DNA or protein matrix), missing taxa can be added to any chronogram simply at random, or by following taxonomic or phylogenetic knowledge from expert sources. There are a wide number of open reference taxonomies available, such as the Catalogue of Life [@Roskov2017] or the NCBI taxonomy database [@Federhen2012]. Expert phylogenies (with or without branch lengths) to be used as topological constraint (backbone) can also be obtained from a number of public repositories, such as OToL [@Hinchliff2015], TreeBASE [@Piel2002] and Dryad (<https://www.datadryad.org//>).
At the moment, DateLife only uses OToL's synthetic tree and reference taxonomy as expert knowledge to automatically add missing taxa to chronograms. Alternatively, users can input a reference taxonomy or topological constraint of their choosing or making. <!-- at some point we will implement the possibility for users to choose different reference taxonomies automatically-->
If OToL's synthetic tree is not satisfactorily resolved for the taxa of interest, DateLife can construct a sequence data matrix from DNA markers available from the Barcode of Life Database (BOLD; @Ratnasingham2007), to attempt to further resolve polytomies. It will follow OToL's synthetic tree as backbone. <!--This molecular tree can then be dated with chronosMPL from the ape package, but is it really needed?? To estimate node ages chronosMPL uses mean path length method from Britton et al. 2002-->
<!--It can rapidly construct trees with branch lengths from
a set of lineages with sequence data from the Barcode Of Life Database (BOLD, <http://www.boldsystems.org/>), using the synthetic OToL (<https://tree.opentreeoflife.org>) as backbone, and to obtain divergence dates.
It allows direct comparison of dates obtained with different markers
available in BOLD (in plants and fungi in particular).-->
To use information from a topological constraint, DateLife calls the congruification method described in [@Eastman2013] to find shared nodes between trees (congruent nodes). It then fixes their ages, and add ages to remaining nodes with a dating method that can be specified by the user. If users have access to biological data, they can input a tree with branch lengths proportional to relative substitution rates as topological constraint. In this case, age data from congruent nodes will be used as calibration points.
Age data from several chronograms can be combined and congruified to be used as calibration points in a single analysis.

Several dating methods are implemented in DateLife.
Branch Length Adjuster (BLADJ) is a simple algorithm to distribute ages of undated nodes evenly, which minimizes age variance in the chronogram [@Webb2008].
DateLife implements BLADJ from the development R version of phylocom's R package [@Webb2008], phylocomr (<https://github.com/ropensci/phylocomr>). It can only be used when there is a topological constraint with no branch lengths.
PATHd8 is a non-clock, rate-smoothing method [@Britton2007] to date trees. It is also called through R. <!--It could be used with a reference taxonomy, if we provide a tree with politomies-->
treePL, is a semi-parametric, rate-smoothing, penalized likelihood dating method [@Smith2012]. It is called through R.
MrBayes program [@Huelsenbeck2001; @Ronquist2003] can be used when adding taxa at random, following a reference taxonomy or a topological constraint. It draws ages from a pure birth model, as implemented by Jetz and collaborators [-@Jetz2012].
DateLife calls MrBayes trough an R function.

DateLife can also correct negative branch lengths in several ways.

<!--**Benchmark: Testing DateLife computing performance**-->
<span style="font-variant:small-caps;">
Benchmark
</span>
<!-- Good to test 3 features on a package: performance, speed and scalability-->

DateLife's code speed was tested on an Apple iMac
with one 3.4 GHz Intel Core i5 processor. <!--No multiprocessor tests were performed-->
We registered variation in computing time relative to number of input names <!--number of chronograms in database--> and DateLife service.
Input processing increases roughly linearly with number of input taxon names, and increases considerably if tnrs service is activated (`r figure_runtime_main`).
<!--
Speed with different number of input lineages and types of analysis:
-->
Results show that searching time increases linearly with number of input names and number of chronograms in database.

Summarizing DateLife results processing times

Adding dates processing time

get_bold_otol_tree running time
<!-- test improvement in synthetic tree after using bold sequences. Is it really better resolved?-->

<!--
Speed of web interface and of r package (in computers with different capacities?)
-->
DateLife's code performance was evaluated with a set of unit tests designed and implemented with the R package testthat [@RCoreTeam2018]. These tests were run both locally –using the devtools package [@RCoreTeam2018]– and on a public server –via GitHub– using the continuous integration tool Travis CI (<//travis-ci.org>). At present, unit tests cover around 30% (for now) of DateLife's code (<https://codecov.io/gh/phylotastic/datelife>). <!-- unit test make code robust, verifiable, debuggable, -->


<!--**Biological example: Testing DateLife accuracy**-->
<span style="font-variant:small-caps;">
Biological example
</span>

Find a clade with at least one chronogram containing all clade's species.
(Penguins look good, but they are giving weird results in SDM)

Remove this chronogram from datelife Results.

Make sdm and median trees and Compare

add taxa with different methods and Compare

Use ltts to compare for now.
Fig. X2 shows comparison of available chronograms for Felidae species and chronograms generated through DateLife

think of a test to compare trees, topology- and date-wise

<!--Bird (or reptile) chronograms, too long time...
finches is good
nothofagus
Look for all chronograms containing any birds
Or, look for chronograms containing small and old lineages
Determine which clade of birds has the more chronograms (have been dated
more times) and use that as biological example-->


<!--**Discussion/Conclusions** there is not a discussion section in this type of sys Bio contributions-->
<span style="font-variant:small-caps;">
Conclusions
</span>

Taxon ages are key to many areas of evolutionary studies: trait evolution, species diversification, biogeography, macroecology and more. Obtaining these ages is difficult, especially for those who want to use phylogenies but who are not systematists, or do not have the time to develop the necessary knowledge and data curation skills to produce new chronograms. Knowledge on taxon ages is also important for non-biological studies and the non-academic community.
The combination of new analytical techniques, availability of more fossil and molecular data, and better practices in data sharing has resulted in a steady accumulation of chronograms in public and open databases such as Dryad, TreeBASE or Open Tree of Life, for a large quantity and diversity of organisms. However, this information remains difficult to synthesize for many biologists and the non-academic community<!-- or just non biologists-->.

<!--Potential applications shown here-->Here, we have shown that DateLife allows an easy and fast obtention of all publicly available information on taxon ages, which can be used to generate new data. <!--Potential applications not shown here-->This information can be used to account for the effect of phylogenetic signal in studies of trait evolution; to explore potential speciation and extinction dynamics of interest within a clade; to obtain a time frame of biogeographical events; for science communication and outreach, amongst others.
<!--Why you should use it-->Compared to similar platforms such as time tree of life and supermart, it offers several advantages.
It is fast;
source data is completely open;
it requires no expert biological knowledge from users for any of its functionalities;
it allows exploration of alternative taxonomic and phylogenetic schemes;
it allows rapid exploration of the effect of alternative divergence time hypothesis;
it allows rapid synthesis in a number of different formats; <!-- we should add graphical options too-->
it facilitates reproducibility of analyses;

Improvements, short and long-term:
* fossils as calibrations: Using secondary calibrations can generate biased ages when using bayesian methods, mainly because we don't know what prior to give to secondary calibrations [@Schenk2016].
* bayesian congruification
* topological congruification

Problems and caveats:



<!--**Availability**-->
<span style="font-variant:small-caps;">
Availability
</span>

DateLife is free and open source and it can be used through its current website <http://www.datelife.org/query/>, or through Phylotastic!'s web portal <http://phylo.cs.nmsu.edu:3000/>.
DateLife can also be used locally through its R package. The stable version is available for installation from the CRAN repository (<https://cran.r-project.org/package=datelife>) using the command `install.packages(pkgs = "datelife")` from R. Development versions are available from GitHub repository (<https://github.com/phylotastic/datelife>) and can be installed using the devtools R package command `install_github("phylotastic/datelife")`.

<!--**Supplementary Material**-->
<span style="font-variant:small-caps;">
Supplementary Material
</span>

Supplementary material, including code files and
online-only appendices, can be found in the GitHub
repository <!-- Dryad also? since it is all in github already-->

<!--**Funding**-->
<span style="font-variant:small-caps;">
Funding
</span>

Funding was provided by NSF grant 1458603

NESCent

Open Tree of Life

University of Tennessee, Knoxville


<!--**Acknowledgements**-->
<span style="font-variant:small-caps;">
Acknowledgements
</span>


<!--**References**-->
<span style="font-variant:small-caps;">
References
</span>
