---
output:
  bookdown::pdf_document2:
    keep_tex: yes
    toc: false
    includes:
      # in_header: preamble.tex
      after_body: [Datelife_paper1_figures_noyaml.tex]
    # fig_caption: no
textalign: left  # this does not work...
geometry: margin = 1in
bibliography: library_red.bib
csl: systematic-biology.csl
link-citations: yes
# indent: true # this works
# header-includes:
#    - \setlength\parindent{24pt}  # this works too, but I'm gonna indent manually with 4 spaces
# nope, 4 spaces indents are not recognized, they do something else
linestretch: 2
header-includes:
    # - \usepackage{caption}
    - \usepackage[left]{lineno}
    - \linenumbers
  # - \documentclass[oupdraft]{sysbio} # this is duplicated in tex generated by knitr
  # - \usepackage{xr}
  # - \externaldocument{Datelife_paper1_figures}
    # - \usepackage[utf8]{inputenc}
    # - \usepackage[english]{babel}
    # - \usepackage[document]{ragged2e}
nocite: |
  @barker2012going, @barker2015new, @burns2014phylogenetics, @claramunt2015new, @gibb2015new, @Hedges2015, @hooper2017chromosomal, @Jetz2012, @price2014niche
---


```{r eval = TRUE, echo = FALSE, warning=FALSE, message=FALSE, results='hide'}
source("Datelife_paper1_global.R")
```

Running head: DATELIFE: REVEALING THE DATED TREE OF LIFE <!--MAX 50 CHARACTERS, ALL CAPITALS-->

Title: DateLife: Leveraging databases and analytical tools to reveal the dated Tree of Life

Authors: Luna L. Sánchez-Reyes^1^, Brian C. O’Meara^1^

Correspondence address:

1. *Department of Ecology and Evolutionary Biology, University of Tennessee, Knoxville, 425 Hesler Biology Building, Knoxville, TN 37996, USA*

Corresponding authors: sanchez.reyes.luna@gmail.com, bomeara@utk.edu


\newpage

**abstract.-** Here goes the abstract.

**Keywords:** Tree; Phylogeny; Scaling; Dating; Ages; Divergence times; Open Science; Congruification; Supertree; Calibrations


\newpage
Time of lineage divergence constitutes a fundamental piece of information for evolutionary
understanding in many areas of research, from developmental to conservation biology [@Felsenstein1985a; @Webb2000], from historical biogeography to species diversification studies [@posadas2006historical; @Morlon2014]. The primary information needed for time of lineage divergence estimation comes from the fossil record. Coupled to molecular phylogenies, molecular dating methods are used to reconstruct the time of divergence of extant lineages. 

Probably encouraged by the great developments in DNA sequencing techniques, phylogenetic inference and molecular dating methods, the number of studies publishing phylogenies with branch lengths proportional to geologic time (hereafter chronograms) have constantly increased in number for the last two decades [@Kumar2017].
Still, generating a chronogram is not easy unless you have specialized training. That's why there has been an urge for reuse of the vast amount and still accuulating already available information for the advantage of researchers not specialized in this area.

Wide interest from the scientific community to make chronogram information available for consultation and reuse has spurred the creation of public platforms with various goals. TreeBASE [@morell1996roots; @Piel2002], Phylomatic [@webb2005phylomatic], the Dryad repository (<http://datadryad.org/>), and the Open Tree of Life [OToL; @Hinchliff2015] platforms are dedicated to storing and making available published trees and chronograms for easy scientific reuse. For chronogram reuse for a wide range of living organisms, only OToL is generally suitable right now. Treebase does not store branch length information and dryad does not distinguish between data sets, so the time of lineage div information cannot be accessed. Phylomatic does contain chronograms but is restricted to plants and mammals, and if it tries to add more data it gives many polytomies back.

OToL also has the primary goal of synthesizing stored trees into a single tree of life, to show phylogenetic relationships among known extant species and subspecies. All or parts of OToL's synthetic tree can be reused for any purpose; however, it currently does not include information on time of divergence, so the platform does not summarize source chronogram data. The Timetree of Life platform main goal is the synthesis of a single chronogram of life [@Hedges2006]. However, both the latest version of their synthetic chronogram [@Kumar2017] and the thousands of chronograms compiled for synthesis are only publicly available for visual examination in their website or for download as images, but not for scientific reuse or reanalysis.

Other platforms such as SuperSmart [@antonelli2017supersmart] and phylogenerator [@pearse2013phylogenerator] are focused in _de novo_ chronogram inference, by reusing DNA sequence data to reconstruct phylogenetic trees. However, expert fossil information necessary for subsequent molecular dating analyses still needs to be compiled and curated by the user, rendering them a challenging tool to obtain data on time of lineage divergence for the non-specialist. Moreover, they do not provide information from available chronograms.

<!-- discussion:
The Angiosperm Phylogeny Website [APW; @stevens2001apw] also specializes in constructing a single synthetic tree, but it focuses in showing phylogenetic relationships between seed plant families only, and it does not store or make available the source trees.

Platforms dedicated to store data on fossils and fossil calibrations: PBDB and The fossil calibration database
can provide data on time of lineage divergence, but it is often not in a phylogenetic context.

To our knowledge, there is no system allowing the comparison of available source information (apw does this a little bit, but it is a verbal comparison more than a graphical or visual comparison). Other types of technical comparisons: which trees are better (which trees represent reality better), but this is of course a very difficult issue.
 -->
A platform for efficient reuse of expert, published data on time of lineage divergence should have an open source database in a format suitable for scientific reuse, and straightforward means of accessing, comparing and summarizing the source data as needed by the user.
A prototype service striving for this characteristics was initially developed over a series of hackathons at the National Evolutionary Synthesis Center [@Stoltzfus2013].
In here we present the first formal description of the `datelife` service. Constituted by and R package and web site, it features new methods to summarize chronograms, an improved system for chronogram database maintenance, new functions to visualize source data in the web interface (<http://www.datelife.org/query/>), as well as tools for comparison of source and summary trees. R packages for benchmarking of functionalities and exemplifying services were also developped.


\begin{center}
\textsc{Description}
\end{center}
<!--(BOLD and OToL species names, are they homogeneous/standardized?-->
The basic `datelife` workflow is shown in figure \ref{fig:workflow} and consists of:

1) A user providing at least two taxon names as input, either as tip labels on a tree, or as a simple comma separated character string. The tree can be in newick or phylo format, and can be with or without branch lengths.
2) `datelife` then performs a search across its database of peer reviewed and curated chronograms; identifies and gets source trees with at least two matching input names; drops unmatching taxa from positively identified source trees; and finally transforms each source tree to a patristic matrix named by the citation of the original study. This format facilitates and greatly speeds up all further analyses and summarization algorithms.
3) The user can obtain different types of summaries from the source data including: a) all source chronograms, b) mrca ages of source chronograms, c) citations of studies where source chronograms were originally published, d) a summary table with all of the above, e) a single summary tree of all source chronograms, and f) a report of succesful matches per input taxon name across source chronograms.
4) At this point, users can choose to use all or some source data as calibration points to date a tree of their own making or choosing. <!--, a taxonomic tree-->
5) Users can also simulate age and/or phylogenetic data of input taxa not found in the database. A variety of algorithms are available for this purpose.
6) Finally, users can easily view results graphically as well as construct their own graphs using inbuilt `datelife` graphic generators.

<!--To perform the former tasks:-->
`datelife`'s chronogram database is currently built from OToL's
tree repository. Among currently existing repositories (i.e., TreeBASE,
Dryad), OToL's metadata rich tree store is the only one meeting the requirements
for automatized handling of source trees. TreeBASE trees are stored without branch lengths. Dryad metadata does not allow differentiating between different types of data sets, and trees would need to be manually curated anyways.
`datelife` currently accepts scientific names, from named clades to binomial specifics.
<!--common names (implemented yet?)-->
Taxon searches are performed at the species level, so when input names correspond
to named clades, `datelife` pulls all accepted species names within the
clade from OToL's reference taxonomy to perform the search.
Currently, searches at the infraspecies level are not allowed, so input names belonging to subspecies or any other infraspecific category are collapsed to the species level.
`datelife` also processes input names with the taxon name resolution service [TNRS; @Boyle2013],
<!--which "corrects misspelled names and authorities, standardizes variant spellings,
and converts nomenclatural synonyms to accepted names" (Boyle et al. 2013)-->
which corrects potentially misspelled names and typos, and standardizes spelling
variations and synonyms <!--to a taxonomic reference of users choosing, default
to otol taxonomy -->, increasing the probability to correctly find the
queried taxa in `datelife`'s chronogram database.

Source chronogram summary tree can be assembled using the Super Distance Matrix
(SDM) supertree construction approach [@Criscuolo2006] or using the median of branch
lengths <!-- and the hierarchical clustering method of ttol-->.
Tree dating and simulation options are performed with various algorithms:
Branch Length Adjuster (BLADJ) is a simple algorithm to distribute ages of undated
nodes evenly, which minimizes age variance in the chronogram [@Webb2008].
PATHd8 is a non-clock, rate-smoothing method [@Britton2007] to date trees.
treePL, is a semi-parametric, rate-smoothing, penalized likelihood dating method
[@Smith2012].
MrBayes [@Huelsenbeck2001; @Ronquist2003] can be used when adding taxa at
random, following a reference taxonomy or a topological constraint. It draws ages
from a pure birth model, as implemented by Jetz and collaborators [-@Jetz2012].
To apply calibrations to a tree, the congruification algorithm described by @Eastman2013
is implemented to find shared nodes between trees (congruent nodes).

To gather, process, and present information, `datelife` builds up from functions
available in several R packages including rotl [@Michonneau2016], ape [@Paradis2004],
geiger [@Harmon2008], paleotree [@Bapst2012a], bold [@Chamberlain2018], phytools [@Revell2012],
taxize [@Chamberlain2013; @Chamberlain2018], phyloch [@Heibl2008], phylocomr [@Ooms2018]
and rphylotastic [@Omeara2019].

<!-- Details on each step are further developed in `datelife`'s R package documentation -->
<!-- `datelife workflow` vignette at (<https://LINK>). -->

\begin{center}
\textsc{Benchmark}
\end{center}
<!--**Benchmark: Testing DateLife computing performance**-->
<!-- Good to test 3 features on a package: performance, speed and scalability-->

`datelife`'s code speed was tested on an Apple iMac
with one 3.4 GHz Intel Core i5 processor. <!--No multiprocessor tests were performed-->
We registered variation in computing time of query processing and search through the database relative to number of queried taxon names.
Query processing increases roughly linearly with number of input taxon names, and
increases considerably if TNRS service is activated. Up to ten thousand names can be processed and searched in less than 30 minutes. A name search through the chronogram database with an already processed query can be performed in less than a minute, even with a very large number of taxon names (Fig. \ref{fig:runtime1}).
<!--Speed of other datelife workflows? NO, bc it refkects speed from other software mainly. But I will put a sentence about it...
Summarizing `datelife` results processing times: shows sdm speed, clustering methods speed;
generating new trees: shows bladj speed;
Adding dates processing time: reflects more mrbayes speed than anything;
get_bold_otol_tree running time: reflects bold downloading data speed.-->
<!--Speed with different number of chronograms in database? NO-->
<!--Speed of web interface and of r package in computers with different capacities? NOOOO-->
`datelife`'s code performance was evaluated with a set of unit tests designed and
implemented with the R package testthat [@RCoreTeam2018] that were run locally
--using the devtools package [@RCoreTeam2018], and on a public server --via
GitHub, using the continuous integration tool Travis CI (<https://travis-ci.org>). At
present, unit tests cover more than 50% of `datelife`'s code (<https://codecov.io/gh/phylotastic/datelife>). <!-- unit test make code robust, verifiable, debuggable, -->


\begin{center}
\textsc{Example}
\end{center}
<!--**Biological example: Testing DateLife accuracy**-->
<!--Bird (or reptile) chronograms, too long time...
finches is good
nothofagus
Look for all chronograms containing any birds
Or, look for chronograms containing small and old lineages
Determine which clade of birds has the more chronograms (have been dated
more times) and use that as biological example. Fringillidae is a good candidate
Find a clade with at least one chronogram containing all clade's species.
(Penguins look good, but they are giving weird results in SDM)

Remove this chronogram from datelife Results.

Make sdm and median trees and Compare

add taxa with different methods and Compare

Use ltts to compare for now.

think of a test to compare trees, topology- and date-wise

test improvement in synthetic otol tree after using bold sequences. Is it really better resolved? Probably not.
Do not include this analysis in paper, maybe in examples, but i think it is not worthy.
-->
In this section we demonstrate the types of outputs that can be obtained with `datelife`, using as an example the bird family Fringillidae of true finches. We performed a higher-taxon search to obtain all data on lineage divergence available in `datelife`'s database for all recognised species within the Fringillidae (`r length(dq$cleaned_names)` spp. according to the Open Tree of Life taxonomy). There are `r length(dr)` chronograms containing at least two Fringillidae species, published in `r length(unique(names(dr)))` different studies (Fig. \ref{fig:schronograms1}).
<!-- In case we want to cite the studies in here, but they are already cited in the figure caption: [Fig. \ref{fig:schronograms1}; @barker2012going; @barker2015new; @burns2014phylogenetics; @claramunt2015new; @gibb2015new; @Hedges2015; @hooper2017chromosomal; @Jetz2012; @price2014niche]. -->
Data from these source chronograms was used to generate two types of summary chronograms, median and SDM. As explained in the `Description`, data from source chronograms was first summarised into a single distance matrix (using either the median or the SDM method) and then the available node ages were used as calibrations points over a consensus tree topology, to obtain a dated tree with the program BLADJ (Fig. \ref{fig:summaries}). Median summary chronograms are older and have wider variation in maximum ages than chronograms obtained with SDM. In both cases, ages are coherent with source ages.
<!-- (Going from a summary distance matrix Using this method is better than using clustering algorithms  -->
<!-- and hierarchical clustering method which appears to push ages back -->
It is not certain if these chronograms can be used to perform downstream evolutionary analyses. There is currently wide interest in determining this. However, we know that these chronograms are useful for...

Data from source chronograms was also used to date tree topologies with no branch length information and trees with branch lengths in relative substitution rates (Figs. \ref{fig:cvbladj} and \ref{fig:cvbold}). As a form of cross validation, we used tree topologies from each study and calibrated them using information from all other source chronograms. In the absence of branch length data, the ages of internal nodes were approximately recovered in almost all cases (except for studies 3, and 5; Fig. \ref{fig:cvbladj}). Maximum tree ages were only approximately recovered in one case (study 2; Fig. \ref{fig:cvbladj}).
Branch lengths were successfully generated using the BOLD database for all source chronograms. However, dating with PATHd8 (using congruified calibrations) was only successful in
<!-- two studies when expanding calibrations to make them agree (studies 3 and 5), and from  -->
three cases (studies 3, 5, and 9; Fig. \ref{fig:cvbold}). From these, two trees have a different sampling than the original source chronogram, mainly because DNA data for some species is absent from the BOLD. Maximum ages are quite different from source chronograms, but this might be explained also by the differences in sampling between source chronograms and BOLD trees.
More examples and details can be consulted in <https://github.com/LunaSare/datelife_examples>.

<!--Show code here, in the paper? Or show it only in the examples? Or only in the vignette? Write down what about datelife will be shown in each of these.
Paper should show outputs from web site and package.
Vignette will show the R code used to generate the outputs displayed here and in the website.
datelife examples will explain more on what is done in every step.
-->

\begin{center}
\textsc{Deficiencies, Limitations and Prospects}
\end{center}

The main goal of `datelife` is to make expert information on time of lineage divergence easily accesible for comparison, reuse, and reanalysis, to researchers in all areas of science and with all levels of expertise in the matter.
It depends 


\begin{center}
\textsc{Conclusions}
\end{center}

Taxon ages are key to many areas of evolutionary studies: trait evolution, species
diversification, biogeography, macroecology and more. Obtaining these ages is difficult,
especially for those who want to use phylogenies but who are not systematists, or
do not have the time to develop the necessary knowledge and data curation skills
to produce new chronograms. Knowledge on taxon ages is also important for non-biological
studies and the non-academic community.
The combination of new analytical techniques, availability of more fossil and molecular
data, and better practices in data sharing has resulted in a steady accumulation
of chronograms in public and open databases such as Dryad, TreeBASE or Open Tree
of Life, for a large quantity and diversity of organisms. However, this information
remains difficult to synthesize for many biologists and the non-academic community<!-- or just non biologists-->.

<!--Potential applications shown here-->Here, we have shown that `datelife` allows
an easy and fast obtention of all publicly available information on taxon ages,
which can be used to generate new data. <!--Potential applications not shown here-->
This information can be used to account for the effect of phylogenetic signal in
studies of trait evolution; to explore potential speciation and extinction dynamics
of interest within a clade; to obtain a time frame of biogeographical events; for
science communication and outreach, amongst others.
<!--Why you should use it-->Compared to similar platforms such as time tree of life
and supermart, it offers several advantages.
It is fast;
source data is completely open;
it requires no expert biological knowledge from users for any of its functionalities;
it allows exploration of alternative taxonomic and phylogenetic schemes;
it allows rapid exploration of the effect of alternative divergence time hypothesis;
it allows rapid synthesis in a number of different formats; <!-- we should add graphical options too-->
it facilitates reproducibility of analyses;

Improvements, short and long-term:
* fossils as calibrations: Using secondary calibrations can generate biased ages when using bayesian methods, mainly because we don't know what prior to give to secondary calibrations
[@Schenk2016].
* bayesian congruification
* topological congruification

Problems and caveats:
Not many databases, only OToL
Why TreeBase is not very useful for us? Be precise.
Are these chronograms reliable to study evolutionary patterns, such as species diversification?
`datelife` can be seen as an open resource to know the current state of knowledge on lineage divergence times.
Whether chronograms obtained using this original data can be used reliably to study complicated patterns of evolution is still uncertain.
If all, la facilidad para obtener hipotesis de tiempo de divergencia nos ayudará a evaluar la capacidad de los cronogramas para estudiar otros fenomenos evolutivos.
Por ahora, no podemos aseverar que estos cronogramas puedan usarse para todo tipo de analisis.


\begin{center}
\textsc{Availability}
\end{center}

`datelife` is free and open source and it can be used through its current website
<http://www.datelife.org/query/>, through its R package, and through Phylotastic's project web portal <http://phylo.cs.nmsu.edu:3000/>.
`datelife`'s website is maintained by RStudio's shiny server and the shiny package open infrastructure, as well as Docker.
`datelife`'s R package stable version is available
for installation from the CRAN repository (<https://cran.r-project.org/package=datelife>)
using the command `install.packages(pkgs = "datelife")` from within R. Development versions
are available from the GitHub repository (<https://github.com/phylotastic/datelife>)
and can be installed using the command `devtools::install_github("phylotastic/datelife")`.


\begin{center}
\textsc{Supplementary Material}
\end{center}

Code used to generate all versions of this manuscript, the biological examples, as well as the software benchmark can be found in GitHub repositories at <https://github.com/LunaSare/datelife_paper1>, <https://github.com/LunaSare/datelife_examples>, and <https://github.com/LunaSare/datelife_benchmark>, respectively.
<!-- Vignettes
Dryad also?? since it is all in github already, maybe not necessary-->

\begin{center}
\textsc{Funding}
\end{center}

Funding was provided by US National Science Foundation (NSF) grant 1458603

NESCent

Open Tree of Life

University of Tennessee, Knoxville


\begin{center}
\textsc{Acknowledgements}
\end{center}

We thank colleagues from the O'Meara Lab at the University
of Tennesse Knoxville for suggestions, discussions and software testing.
The late National Evolutionary Synthesis Center (NESCent), which sponsored hackathons
that led to initial work on this project.
The Open Tree of Life project that provides the open, metadata rich repository of
trees used for `datelife`.
The many scientists who publish their chronograms in an open, reusable form, and
the scientists who curate them for deposition in OpenTree.
The NSF for funding nearly all the above, in addition
to the ABI grant that funded this project itself.

\newpage

\begin{center}
\textsc{References}
\end{center}
