---
output: 
  bookdown::pdf_document2:
    keep_tex: yes
    toc: false
    includes:
      # in_header: preamble.tex
      after_body: [Datelife_paper1_figures_noyaml.tex]
    # fig_caption: no
textalign: left  # this does not work...
geometry: margin = 1in
bibliography: library_red.bib
csl: systematic-biology.csl
link-citations: yes
# indent: true # this works
# header-includes:
#    - \setlength\parindent{24pt}  # this works too, but I'm gonna indent manually with 4 spaces
# nope, 4 spaces indents are not recognized, they do something else
linestretch: 2
header-includes:
    # - \usepackage{caption}
    - \usepackage[left]{lineno}
    - \linenumbers  
  # - \documentclass[oupdraft]{sysbio} # this is duplicated in tex generated by knitr
  # - \usepackage{xr}
  # - \externaldocument{Datelife_paper1_figures}
    # - \usepackage[utf8]{inputenc}
    # - \usepackage[english]{babel}
    # - \usepackage[document]{ragged2e}
nocite: | 
  @barker2012going, @barker2015new, @burns2014phylogenetics, @claramunt2015new, @gibb2015new, @Hedges2015, @hooper2017chromosomal, @Jetz2012, @price2014niche
---


```{r eval = TRUE, echo = FALSE, warning=FALSE, message=FALSE, results='hide'}
source("Datelife_paper1_global.R")
```

Running head: DATELIFE: REVEALING THE DATED TREE OF LIFE <!--MAX 50 CHARACTERS, ALL CAPITALS-->

Title: DateLife: Leveraging databases and analytical tools to reveal the dated Tree of Life

Authors: Luna L. Sánchez-Reyes^1^, Brian C. O’Meara^1^

Correspondence address:

1. *Department of Ecology and Evolutionary Biology, University of Tennessee, Knoxville, 425 Hesler Biology Building, Knoxville, TN 37996, USA*

Corresponding authors: sanchez.reyes.luna@gmail.com, bomeara@utk.edu


\newpage

**abstract.-** Here goes the abstract.

**Keywords:** Tree; Phylogeny; Scaling; Open; Ages; Congruify; Supertree;


\newpage

<!--INTRODUCTION-->
Supersmart is an open tool but not easy to use, still requires a lot of curation and knowledge.

<!--INTRODUCTION: 1st paragraph: general statement that the field knows to be true and important-->
Time of lineage divergence<!--alt lineages: named biological entities, taxon ages?-->
constitutes in many ways the fundamental/main knowledge necessary for evolutionary
understanding.
<!--Along with phylogenetic relationships, it is the most important piece of information
allowing to test alternative evolutionary hypothesis in many areas of research,
from developmental to ecological questions.
Along with migration rates and patterns, it is fundamental to test biogeographical
hypothesis-->
Coupled to species number and distribution, it is the main information necessary
for the study of diversification processes (i.e., the tempo and mode of speciation
and extinction),
<!-- crucial/vital/critical/essential  -->central for the understanding of how biodiversity
patterns are shaped across space, time and clades [@Morlon2014].
Evolutionary understanding also relies on comparative studies, for which knowing
the time context for all life is crucial. Efforts to have a whole tree of life have
been great and here are some examples.
<!--INTRODUCTION: 2nd paragraph: important advances in the field-->
In the past two decades, the possibility to obtain good quality DNA sequences
<!-- from a wide variety of organisms became a reality, which,  -->
coupled to methodological developments in phylogenetic and dating inference, allowed
the application of molecular dating methods on a very large amount and diversity
of organisms, greatly increasing the quantity of<!--organisms with--> data on taxon
ages across the tree of life.
<!-- figure showing growth on studies with age data from references table from ttol-->
To date, there is a large amount of both fossil and molecular-based data on taxon
ages and phylogenetic relationships in public repositories such as Dryad, TreeBASE
and Open Tree of Life (OToL).
OToL alone holds more than 200 chronograms. <!--cache updated jan 15 2018
devtools::load_all(pkg = "~/Desktop/datelife")
length(opentree_chronograms$trees) names(opentree_chronograms)
-->
<!--How many trees? How many dated trees?-->
Methods to include living and fossil lineages are in continued development and increased
usage by the community<!--total evidence and revbayes (fossilized birth-death)-->,
which coupled to better sharing data practices, are greatly contributing to the
accumulation in number and type of available data on taxon ages.

<!--INTRODUCTION: 3rd paragraph: problems associated-->
The TimeTree project [@Hedges2006; @Hedges2015; @Kumar2017] has aggregated chronograms
from 3,163 studies, encompassing 97,085 species [@Kumar2017], and continues to grow.
However, even in this gold standard resource, the included taxa only encompass between
0.097 and 3.236% of total species diversity (following taxonomic expert opinion
on the global, extant species numbers, which ranges from 3 to 100 million species
[Mayr2010; Moran2011]). One advantage of TimeTree is that it includes taxa from
across the tree of life, versus more specialized chronograms focusing on plants
[PHYLOMATIC], birds [JETZ ET AL BIRDTREE.ORG], and other groups. Users can choose
between a web interface or a mobile app to receive information on divergence times
for the evolutionary history of a lineage, pairs of taxa, all lineages within a
taxon, or a list of taxa. As a science communication tool, TimeTree project is
very powerful: it has a friendly graphical interface, with informative and colorful
outputs, that allows the general public to satisfy curiosity regarding a particular
organism of interest or group of them. It is of limited utility for scientific
studies, however. The thousands of trees that have been entered are unavailable
for examination or reuse; according to the creators (see TimeTree web FAQ), methods
for allowing data downloading have been under discussion for the past several years
yet the primary data remain closed. Moreover, there is no Application Programming
Interface (API) allowing programmatic access to any data, greatly impairing the
possibility of large-scale, automated data-mining, which is not allowed under TimeTree
website's terms of use. The nearly hundred thousand taxon summary chronogram generated
from TimeTree resources is not available with its publication [@Kumar2017] or the
TimeTree website, though the still substantial chronogram from a previous publication
[@Hedges2015] was made available at OToL.
<!--What led to datelife development?
Describe public and research necessities covered by datelife
It is important to use available data on time frame of lineage origin:
To know the state of dating for a group of interest:
What range of estimated ages exist already? Are fossil and molecular
time frames coherent? (e.g., [@Magallon2015c]).
To construct a time tree of life.
For science communication, improve scientific discussions, time-framing
other events of importance in other research areas.
Comparative method: autocorrelation (phylogenetic relatedeness) corrections, phylogenetic signal
-->

<!--INTRODUCTION: 4th paragraph-->
Despite its great importance, analytical tools to summarize available information
on taxon ages for the scientific community are still lacking.
We identified several aspects that might have so far delayed the exploitation of
existing data. First, original chronograms available publicly are scattered across
various repositories (otol tree store, dryad, treebase, journals supplementary data)
usually with different formats too.
Second, lineage names due to taxonomic idiosincracy can be different among studies
and manual curation of that is usually necessary.
Third, data curation
Recent advances on this area (e.g., supersmart<!--and, which others?-->) aim to:
Generate new dates using all available DNA sequence information;
Perform one global analysis using all available information;
Problems or downsides: This might be time consuming for large groups and a lot of
data curation and knowledge on the group of interest is still necessary. For example,
<!--General issues with dating techniques:--> choosing correct fossils for calibration
requires a lot of expertise and knowledge on the group. An incorrect use of fossils
can generate severe bias in dating results [@Sauquet2012c].
<!--Data curation is necessary at some point (is this always true?)-->
Hence, data curation is still an important part of any biological study. The research
community considers it as an important or even crucial step before data analysis.
Hence, automated processes for large data analysis are frequently received with skepticism.

`datelife` palliates this by only using information available from already published
studies, which are ideally constructed using robust information, such as sequence
data and thoughtfully curated fossil calibrations.
<!--Data curation is largely based on taxonomic knowledge; or at least on the interplay between taxonomy and phylogenetics-->
<!--Importance of datelife:-->

<!--INTRODUCTION: 5th paragraph: Goal, objectives and predictions-->
Rapidly increasing data on time of lineage divergence both from molecular and paleontological studies; the increasing importance of use of these data in distant areas of research, often not specialized enough to rapidly obain data on their own; and the lack of an open (both the data sources and the code underlying the analyses) easy to use tool
inspired the development of a prototype `datelife` service over a series
of phylotastic hackathons [@Stoltzfus2013] at the National Evolutionary Synthesis Center.
In this paper we present the first formal description of `datelife`, featuring an improved database of chronograms, more methods to summarize trees, and new functions to visualize data, as well as comparisons of summary trees.
`datelife` is the main service for scaling phylogenetic trees in Phylotastic! system
[@Stoltzfus2013] <!--to construct phylogenetic trees on the fly [http://phylotastic.org/].-->
It can be used through an R package <link to documentation>, a web interface
(<http://www.datelife.org/query/>) and an API<!--(still not up, right?)-->.


\begin{center}
\textsc{Description}
\end{center}
<!--(BOLD and OToL species names, are they homogeneous/standardized?-->
The basic `datelife` workflow is shown in figure \ref{fig:workflow} and consists of:

1) A user providing at least two taxon names as input, either as tip labels on a tree, or as a simple comma separated character string. The tree can be in newick or phylo format, and can be with or without branch lengths.
2) `datelife` then performs a search across its database of peer reviewed and curated chronograms; identifies and gets source trees with at least two matching input names; drops unmatching taxa from positively identified source trees; and finally transforms each source tree to a patristic matrix named by the citation of the original study. This format facilitates and greatly speeds up all further analyses and summarization algorithms.
3) The user can obtain different types of summaries from the source data including: a) all source chronograms, b) mrca ages of source chronograms, c) citations of studies where source chronograms were originally published, d) a summary table with all of the above, e) a single summary tree of all source chronograms, and f) a report of succesful matches per input taxon name across source chronograms.
4) At this point, users can choose to use all or some source data as calibration points to date a tree of their own making or choosing. <!--, a taxonomic tree-->
5) Users can also simulate age and/or phylogenetic data of input taxa not found in the database. A variety of algorithms are available for this purpose.
6) Finally, users can easily view results graphically as well as construct their own graphs using inbuilt `datelife` graphic generators.

<!--To perform the former tasks:-->
`datelife`'s chronogram database is currently built from Open Tree of Life (OToL)'s
[@Hinchliff2015] tree repository. Among currently existing repositories (e.g., TreeBase,
Dryad), OToL's metadata rich tree store is the only one meeting the requirements
for proper/accurate automatized handling of trees.
Input taxon names accepted by `datelife` are binomial species names or clades.
<!--common names (implemented yet?)-->
Taxon searches are performed at the species level, so when input names correspond
to higher clades,
`datelife` pulls all accepted species names within the
clade from OToL's reference taxonomy to perform the search.
Currently, searches at the infrapsecies level are not allowed, so input names belonging to subspecies or any other infraspecific category are treated
as species.
`datelife` also processes input names with the taxon name resolution service (TNRS),
<!--which "corrects misspelled names and authorities, standardizes variant spellings,
and converts nomenclatural synonyms to accepted names" (Boyle et al. 2013)-->
which corrects potentially misspelled names and typos, and standardizes variation
in spelling and synonyms <!--to a taxonomic reference of users choosing, default
to otol taxonomy -->[@Boyle2013], increasing the probability to correctly find the
queried taxa in `datelife`'s chronogram database.

Source chronogram summary tree can be assembled using the Super Distance Matrix
(SDM) supertree construction approach [@Criscuolo2006] or using the median of branch
lengths and the hierarchical clustering method<!-- of ttol-->. <!--Also, I discovered the median gives non ultrametric trees... but I think it is fixed now, check it!).-->
Tree dating and simulation options are performed with various algorithms:
Branch Length Adjuster (BLADJ) is a simple algorithm to distribute ages of undated
nodes evenly, which minimizes age variance in the chronogram [@Webb2008].
PATHd8 is a non-clock, rate-smoothing method [@Britton2007] to date trees.
treePL, is a semi-parametric, rate-smoothing, penalized likelihood dating method
[@Smith2012].
MrBayes [@Huelsenbeck2001; @Ronquist2003] can be used when adding taxa at
random, following a reference taxonomy or a topological constraint. It draws ages
from a pure birth model, as implemented by Jetz and collaborators [-@Jetz2012].
To apply calibrations to a tree, th econgruification algorithm described in [@Eastman2013]
is used to find shared nodes between trees (congruent nodes).

To gather, process, and present information, `datelife` builds up from functions
available in several R packages including rotl [@Michonneau2016], ape [@Paradis2004],
geiger [@Harmon2008], paleotree [@Bapst2012a], bold [@Chamberlain2018], phytools [@Revell2012],
taxize [@Chamberlain2013; @Chamberlain2018], phyloch [@Heibl2008], phylocomr [@Ooms2018]
and rphylotastic [@Omeara2019].

<!-- Details on each step are further developed in `datelife`'s R package documentation -->
<!-- `datelife workflow` vignette at (<https://LINK>). -->

\begin{center}
\textsc{Benchmark}
\end{center}
<!--**Benchmark: Testing DateLife computing performance**-->
<!-- Good to test 3 features on a package: performance, speed and scalability-->

`datelife`'s code speed was tested on an Apple iMac
with one 3.4 GHz Intel Core i5 processor. <!--No multiprocessor tests were performed-->
We registered variation in computing time of query processing and search through the database relative to number of queried taxon names.
Query processing increases roughly linearly with number of input taxon names, and
increases considerably if TNRS service is activated. Up to ten thousand names can be processed and searched in less than 30 minutes. A name search through the chronogram database with an already processed query can be performed in less than a minute, even with a very large number of taxon names (Fig. \ref{fig:runtime1}).
<!--
Speed with different number of input lineages and types of analysis:
Results show that searching time increases linearly with number of input names and
number of chronograms in database.
Summarizing `datelife` results processing times: sdm speed, clustering methods speed

generating new trees: bladj speed
Adding dates processing time: reflects more mrbayes speed than anything

get_bold_otol_tree running time: not really useful since that is an external service
-->
<!-- test improvement in synthetic tree after using bold sequences. Is it really better resolved?-->

<!--Speed of web interface and of r package (in computers with different capacities?)-->
`datelife`'s code performance was evaluated with a set of unit tests designed and
implemented with the R package testthat [@RCoreTeam2018]. These tests were run both
locally –using the devtools package [@RCoreTeam2018]– and on a public server –via
GitHub– using the continuous integration tool Travis CI (<https://travis-ci.org>). At
present, unit tests cover more than 50% of `datelife`'s code (<https://codecov.io/gh/phylotastic/datelife>). <!-- unit test make code robust, verifiable, debuggable, -->


\begin{center}
\textsc{Example}
\end{center}
<!--**Biological example: Testing DateLife accuracy**-->
<!--Bird (or reptile) chronograms, too long time...
finches is good
nothofagus
Look for all chronograms containing any birds
Or, look for chronograms containing small and old lineages
Determine which clade of birds has the more chronograms (have been dated
more times) and use that as biological example. Fringillidae is a good candidate
Find a clade with at least one chronogram containing all clade's species.
(Penguins look good, but they are giving weird results in SDM)

Remove this chronogram from datelife Results.

Make sdm and median trees and Compare

add taxa with different methods and Compare

Use ltts to compare for now.

think of a test to compare trees, topology- and date-wise
-->
In this section we demonstrate the types of outputs that can be obtained with `datelife`, using as an example the bird family Fringillidae of true finches. We performed a higher-taxon search to obtain all data on lineage divergence available from `datelife`'s database for all recognised species within the Fringillidae (`r length(dq$cleaned_names)` spp. according to the Open Tree of Life taxonomy). There are `r length(dr)` chronograms containing at least two Fringillidae species, published in `r length(unique(names(dr)))` different studies (Fig. \ref{fig:schronograms1}). 
<!-- In case we want to cite the studies in here, but they are already cited in the figure caption: [Fig. \ref{fig:schronograms1}; @barker2012going; @barker2015new; @burns2014phylogenetics; @claramunt2015new; @gibb2015new; @Hedges2015; @hooper2017chromosomal; @Jetz2012; @price2014niche]. -->
Data from these source chronograms was used to generate two types of summary chronograms, median and SDM. As explained in the `Description`, data from source chronograms was first summarised into a single distance matrix (using either the median or the SDM method) and then the available node ages were used as calibrations points over a consensus tree topology, to obtain a dated tree with the program BLADJ (Fig. \ref{fig:summaries}). Median summary chronograms are older and have wider variation in maximum ages than chronograms obtained with SDM. In both cases, ages are coherent with source ages.
<!-- (Going from a summary distance matrix Using this method is better than using clustering algorithms  -->
<!-- and hierarchical clustering method which appears to push ages back -->
It is not certain if these chronograms can be used to perform downstream evolutionary analyses. There is currently wide interest in determining this. However, it is certain that these chronograms are useful for...

Data from source chronograms was also used to date tree topologies with no branch length information and trees with branch lengths in relative substitution rates (Figs. \ref{fig:cvbladj} and \ref{fig:cvbold}). As a form of cross validation, we used tree topologies from each study and calibrated them using information from all other source chronograms. In the absence of branch length data, the ages of internal nodes were approximately recovered in almost all cases (except for studies 3, and 5; Fig. \ref{fig:cvbladj}). Maximum tree ages were only approximately recovered in one case (study 2; Fig. \ref{fig:cvbladj}).
Branch lengths were successfully generated using the BOLD database for all source chronograms. However, dating with PATHd8 (using congruified calibrations) was only successful in  
<!-- two studies when expanding calibrations to make them agree (studies 3 and 5), and from  -->
three cases (studies 3, 5, and 9; Fig. \ref{fig:cvbold}). From these, two trees have a different sampling than the original source chronogram, mainly because DNA data for some species is absent from the BOLD. Maximum ages are quite different from source chronograms, but this might be explained also by the differences in sampling between source chronograms and BOLD trees.
More examples and details can be consulted in <https://github.com/LunaSare/datelife_examples>.

<!--Show code here, in the paper? Or show it only in the examples? Or only in the vignette? Write down what about datelife will be shown in each of these.
Paper should show outputs from web site and package. 
Vignette will show the R code used to generate the outputs displayed here and in the website.
datelife examples will explain more on what is done in every step.
-->

\begin{center}
\textsc{Conclusions}
\end{center}

Taxon ages are key to many areas of evolutionary studies: trait evolution, species
diversification, biogeography, macroecology and more. Obtaining these ages is difficult,
especially for those who want to use phylogenies but who are not systematists, or
do not have the time to develop the necessary knowledge and data curation skills
to produce new chronograms. Knowledge on taxon ages is also important for non-biological
studies and the non-academic community.
The combination of new analytical techniques, availability of more fossil and molecular
data, and better practices in data sharing has resulted in a steady accumulation
of chronograms in public and open databases such as Dryad, TreeBASE or Open Tree
of Life, for a large quantity and diversity of organisms. However, this information
remains difficult to synthesize for many biologists and the non-academic community<!-- or just non biologists-->.

<!--Potential applications shown here-->Here, we have shown that `datelife` allows
an easy and fast obtention of all publicly available information on taxon ages,
which can be used to generate new data. <!--Potential applications not shown here-->
This information can be used to account for the effect of phylogenetic signal in
studies of trait evolution; to explore potential speciation and extinction dynamics
of interest within a clade; to obtain a time frame of biogeographical events; for
science communication and outreach, amongst others.
<!--Why you should use it-->Compared to similar platforms such as time tree of life
and supermart, it offers several advantages.
It is fast;
source data is completely open;
it requires no expert biological knowledge from users for any of its functionalities;
it allows exploration of alternative taxonomic and phylogenetic schemes;
it allows rapid exploration of the effect of alternative divergence time hypothesis;
it allows rapid synthesis in a number of different formats; <!-- we should add graphical options too-->
it facilitates reproducibility of analyses;

Improvements, short and long-term:
* fossils as calibrations: Using secondary calibrations can generate biased ages when using bayesian methods, mainly because we don't know what prior to give to secondary calibrations
[@Schenk2016].
* bayesian congruification
* topological congruification

Problems and caveats:
Not many databases, only OToL
Why TreeBase is not very useful for us? Be precise.
Are these chronograms reliable to study evolutionary patterns, such as species diversification?
`datelife` can be seen as an open resource to know the current state of knowledge on lineage divergence times.
Whether chronograms obtained using this original data can be used reliably to study complicated patterns of evolution is still uncertain.
If all, la facilidad para obtener hipotesis de tiempo de divergencia nos ayudará a evaluar la capacidad de los cronogramas para estudiar otros fenomenos evolutivos.
Por ahora, no podemos aseverar que estos cronogramas puedan usarse para todo tipo de analisis.


\begin{center}
\textsc{Availability}
\end{center}

`datelife` is free and open source and it can be used through its current website
<http://www.datelife.org/query/>, through its R package, and through Phylotastic's project web portal <http://phylo.cs.nmsu.edu:3000/>.
`datelife`'s website is maintained by RStudio's shiny server and the shiny package open infrastructure, as well as Docker.
`datelife`'s R package stable version is available
for installation from the CRAN repository (<https://cran.r-project.org/package=datelife>)
using the command `install.packages(pkgs = "datelife")` from within R. Development versions
are available from the GitHub repository (<https://github.com/phylotastic/datelife>)
and can be installed using the command `devtools::install_github("phylotastic/datelife")`.


\begin{center}
\textsc{Supplementary Material}
\end{center}

Code used to generate all versions of this manuscript, the biological examples, as well as the software benchmark can be found in GitHub repositories at <https://github.com/LunaSare/datelife_paper1>, <https://github.com/LunaSare/datelife_examples>, and <https://github.com/LunaSare/datelife_benchmark>, respectively.
<!-- Vignettes
Dryad also?? since it is all in github already, maybe not necessary-->

\begin{center}
\textsc{Funding}
\end{center}

Funding was provided by NSF grant 1458603

NESCent

Open Tree of Life

University of Tennessee, Knoxville


\begin{center}
\textsc{Acknowledgements}
\end{center}

We thank colleagues (students and postdocs) at the O'Meara Lab at the University
of Tennesse Knoxville for suggestions, discussions and software testing.
The late National Evolutionary Synthesis Center (NESCent), which sponsored hackathons
that led to initial work on this project.
The Open Tree of Life project that provides the open, metadata rich repository of
trees used for `datelife`.
The many scientists who publish their chronograms in an open, reusable form, and
the scientists who curate them for deposition in OpenTree.
The US National Science Foundation (NSF) for funding nearly all the above, in addition
to the ABI grant that funded this project itself.

\newpage

\begin{center}
\textsc{References}
\end{center}
