---
output:
  pdf_document: default
  html_document: default
---

Clade ages represent a fundamental piece of information for evolutionary
understanding in many areas of research, from developmental to conservation biology [@Felsenstein1985a; @Webb2000], from historical biogeography to species diversification studies [@posadas2006historical; @Morlon2014]. The primary information needed for these time estimates comes from the fossil record. Coupled with phylogenies with branch lengths based on molecular and/or morphological data, the time of divergence of extant and extinct lineages can be reconstructed with molecular dating methods.
The number of studies publishing phylogenies with branch lengths proportional to geological time (hereafter chronograms) have constantly increased in number for the last two decades [@Kumar2017].
Still, generating a chronogram is not an easy task unless you have specialized training: it requires inferring a tree, understanding what fossil data are available and their limits, and where fossils go on the tree. That is why there has been an urge for promoting and facilitating reuse of the vast amount of phylogenetic and time of lineage divergence data that has been generated and made available in publications, for the advantage of research relying on this information [@webb2005phylomatic; @Stoltzfus2013].

Wide interest from the scientific community to make information from phylogenies in general and chronograms in particular available for consultation and reuse has spurred the creation of public platforms with various goals and characteristics. TreeBASE [@piel2009treebase; @vos2012nexml], the Dryad repository (<http://datadryad.org/>), and the Open Tree of Life (OToL) repository [@Hinchliff2015; @mctavish2015phylesystem] are platforms that store and make available published phylogenies and chronograms for easy scientific reuse. Trees in all of them can be queried using automatised web procedures, which permit personalized, large scale searches that are also very fast.
OToL stores trees with branch length information from a wide range of living organisms, implementing a metadata structure that stores the branch length units (i.e., time or relative susbtitution rates). Treebase and Dryad repositories also contain trees from all groups of life, but the former did not store branch length information until recently (and lacks consistent metadata on what any branch lengths stored mean) and Dryad stores many other types of biological data using metadata that does not allow automatic distinction of types of trees and branch length units, impairing the automatised access to time of lineage divergence information.

Besides keeping a repository to easily store and share expert phylogenetic and chronogram knowledge, OToL also has the primary goal of synthesising all trees in their repository to expose to the community a single tree of all life depicting the phylogenetic relationships among known lineages.
All or parts of this synthetic tree can be reused for any purpose. However, it currently only focus on synthesizing tree topology, meaning that it does not expose branch length data of any type.
<!-- let alone time of divergence data from their stored chronograms. -->
The Timetree of Life project focuses on the synthesis of a single chronogram of life [@Hedges2006] and presents a very accessible, attractive interface. However, the thousands of chronograms this NSF-funded project have compiled for synthesis are only publicly available for visual examination in their website or for download as images, but large scale download remains prohibited by their site. The latest version of their synthetic chronogram [@Kumar2017] can be queried only through their website in a non-automatised fashion, and only subsets of it can be reused for analyses with the permission of the authors.
Other platforms such as SuperSmart [@antonelli2017supersmart] and phylogenerator [@pearse2013phylogenerator] are focused in automatised _de novo_ chronogram inference, by reusing DNA sequence data to reconstruct phylogenetic trees. However, expert fossil information necessary for subsequent molecular dating analyses still needs to be compiled and curated by the user, rendering them a challenging tool to obtain data on time of lineage divergence for the non-specialist. Additionally, neither Supersmart nor phylogenerator are designed to provide information from already created expert chronograms, since it might not be their objective.

<!-- discussion:
The Angiosperm Phylogeny Website [APW; @stevens2001apw] also specializes in constructing a single synthetic tree, but it focuses in showing phylogenetic relationships between seed plant families only, and it does not store or make available the source trees.

Platforms dedicated to store data on fossils and fossil calibrations: PBDB and The fossil calibration database
can provide data on time of lineage divergence, but it is often not in a phylogenetic context.

To our knowledge, there is no system allowing the comparison of available source information (apw does this a little bit, but it is a verbal comparison more than a graphical or visual comparison). Other types of technical comparisons: which trees are better (which trees represent reality better), but this is of course a very difficult issue.
 -->
A tool for efficient reuse of expert, published data on time of lineage divergence should have an open and fully public chronogram database storing data in a format suitable for scientific reuse, an automatised way of accessing the information, and straightforward means of comparing and summarizing chronogram information as needed by the user.
A prototype service aiming to meet this characteristics was developed over a series of hackathons at the National Evolutionary Synthesis Center [@Stoltzfus2013].
Here we present the formal description and implementation of the `datelife` service, constituted by an R package and a web site (<http://www.datelife.org/>).
<!-- There is still much room for improvement, and flaws and limitations are addressed below.  -->
The current implementation of `datelife` performs the basic tasks described above. It features a system for maintenance of an open database of chronograms pulled from public repositories, methods to summarize and compare source chronograms, and new functions to visualize and graphically compare source and summary chronograms.
<!-- , for the benefit of the broader community. -->
<!-- as well as tools for comparison of source and summary trees.  NOT YET u_u-->
<!-- R packages for benchmarking of functionalities and demonstrating services were also developed and made available.>

<!--(BOLD and OToL species names, are they standardized with each other? I think yes-->

\begin{center}
\textsc{Description}
\end{center}

The basic `datelife` workflow is shown in figure \ref{fig:workflow} and consists of:

\begin{enumerate}
\item A user providing at least two taxon names as input. Taxon names can be given at the species level or above. The user can provide them as a simple comma separated character string, or as tip labels on a tree. The tree can be provided in newick format (as a character string) or as a "phylo" R object, and can be with or without branch lengths.
\item A name search is then performed across the chronogram database. Source trees with at least two matching input names are selected, and taxa that do not match the original query are dropped from these source trees.
<!-- The resulting pruned source chronograms are hereafter referred as source chronograms.  --> <!--# This was generating some confussion, so I'm calling them pruned source chronograms now-->
Finally, each pruned source chronogram is transformed to a patristic distance matrix. This format facilitates and greatly speeds up all further analyses and summarizing algorithms. For the sake of keeping track of authorship later, the matrices are associated to the citation of the original study.
\item  At this point, the user can already obtain different summary information from their query, allowing the user to take informed decisions for the next steps of the workflow. Types of summary information provided are: a) all pruned source chronograms, b) age of the mrca (most recent common ancestor) of the pruned source chronograms, c) citations of studies where pruned source chronograms were originally published, d) a summary table with all of the above, e) a single summary chronogram of all or a subset of pruned source chronograms, f) a report of succesful matches of input taxon names across pruned source chronograms, and g) the single pruned source chronogram with the most taxa matched.
\item  Then, source chronogram data can be used as calibration points to date a tree with or without branch lengths containing some or all names from the initial query. %<!--, a taxonomic tree-->
\item  If there is no information available for any queried taxa, users can also create both age and phylogenetic data for this missing taxa with a variety of algorithms described below.
\item  Finally, users can easily save all source and summary chronograms in formats that permit easy reuse and reanalyses (newick and R "phylo" format), as well as view and compare results graphically, or construct their own graphs using \texttt{datelife}'s graphic generation functions.
\end{enumerate}

<!--To perform each task:-->
To gather, process, and present information, `datelife` builds up from functions
available in several R packages including rotl [@Michonneau2016], ape [@Paradis2004],
geiger [@Harmon2008], paleotree [@Bapst2012a], bold [@Chamberlain2018], phytools [@Revell2012], taxize [@Chamberlain2013; @Chamberlain2018], phyloch [@Heibl2008], phylocomr [@Ooms2018] and rphylotastic [@Omeara2019].

A `datelife` search currently accepts scientific names only. It can be any named clade or binomial specific. <!--common names (implemented in phylotastic)-->
The search is performed at the species level, so when input names correspond
to named clades, `datelife` pulls all accepted species names within the
clade from OToL's reference taxonomy [@rees2017automated] and uses all of those.
Searches at the infraspecies level are not currently allowed, so input names belonging to subspecies or any other infraspecific category are collapsed to the species level.
`datelife` processes input names with the taxon name resolution service [TNRS; @Boyle2013],
<!--which "corrects misspelled names and authorities, standardizes variant spellings,
and converts nomenclatural synonyms to accepted names" (Boyle et al. 2013)-->
which corrects potentially misspelled names and typos, and standardizes spelling
variations and synonyms <!--to a taxonomic reference of users choosing, default
to otol taxonomy -->, increasing the probability of correctly finding the
queried taxa in `datelife`'s chronogram database.

Then, a name matching procedure is performed across `datelife`'s chronogram database which is assembled from OToL's tree repository [known as the phylesystem; @mctavish2015phylesystem]. Compared to other existing open tree repositories OToL's metadata rich tree store is the only one that supports search, identification, and handling of chronograms in an automatised fashion. Also, the tip names in their stored trees correspond to scientific names at or below the species level. Finally, all their chronograms come from peer-reviewed published studies generated by specialists in the targeted lineages, arguably representing expert knowledge on time of lineage divergence.
<!-- The TreeBASE repository stores trees without branch lengths, so chronograms stored there do not provide relevant information on time of lineage divergence. The Dryad repository stores all types of biological data besides chronograms, and their metadata does not allow differentiating between branch lengths units.  -->
<!-- branch length units that might represent other types of time of divergence that are not proportional to absolute (geologic) time.  -->

Information from pruned source chronograms can then be summarised with simple summary statistics of tree branch length, such as the median or the mean. A  much slower, Super Distance Matrix (SDM) approach for supertree reconstruction with branch lengths [@Criscuolo2006] is also implemented via the ape package [@Paradis2004].<!-- and the hierarchical clustering method of ttol??-->
<!-- describe what SDM does and why clustering does not work here -->
The resulting summary patristic distance matrix could be clustered with classic algorithms to return a tree. However, the resulting trees are often non-ultrameric and do not reflect the source chronogram data (see datelife_examples package). Instead, we obtained a distribution of age data from the summary matrix available for nodes on a consensus tree. The Branch Length Adjuster (BLADJ) algorithm [@Webb2008] was then used to fix known node ages and then distribute nodes with unknown ages evenly over the consensus tree, minimizing age variance in the resulting chronogram.

For tree dating, the congruification algorithm described by @Eastman2013
is implemented to find shared nodes between trees (congruent nodes). The ages of these nodes are then used as calibrations to date any given tree. Currently implemented methods for tree dating are BLADJ, MrBayes [@Huelsenbeck2001; @Ronquist2003] and PATHd8 [@Britton2007], a non-clock, rate-smoothing dating method.

<!-- and simulations are performed with various algorithms: -->
<!-- MrBayes can be used when adding taxa at -->
<!-- random, following a reference taxonomy or a topological constraint. It draws ages -->
<!-- from a pure birth model, as implemented by Jetz and collaborators [-@Jetz2012]. -->


<!-- Details on each step are further developed in `datelife`'s R package documentation -->
<!-- `datelife workflow` vignette at (<https://LINK>). -->

\begin{center}
\textsc{Benchmark}
\end{center}
<!--**Benchmark: Testing DateLife computing performance**-->
<!-- Good to test 3 features on a package: performance, speed and scalability-->

`datelife`'s code speed was tested on an Apple iMac
with one 3.4 GHz Intel Core i5 processor. <!--No multiprocessor tests were performed-->
We registered variation in computing time of query processing and search through the database relative to number of queried taxon names.
Query processing time increases roughly linearly with number of input taxon names, and
increases considerably if taxonomic name resolution with TNRS [@Boyle2013] is activated.
<!-- Defining tnrs here again, bc we only talk about it two times, so it easy to forget it -->
Up to ten thousand names can be processed and searched in less than 30 minutes with the most time consuming settings.
Once names have been processed as described in methods, a name search through the chronogram database can be performed in less than a minute, even with a very large number of taxon names (Fig. \ref{fig:runtime1}).
<!--Speed of other datelife workflows? NO, bc it refkects speed from other software mainly. But I will put a sentence about it...
Summarizing `datelife` results processing times: shows sdm speed, clustering methods speed;
generating new trees: shows bladj speed;
Adding dates processing time: reflects more mrbayes speed than anything;
get_bold_otol_tree running time: reflects bold downloading data speed.-->
<!--Speed with different number of chronograms in database? NO-->
<!--Speed of web interface and of r package in computers with different capacities? NOOOO-->
`datelife`'s code performance was evaluated with a set of unit tests designed and
implemented with the R package testthat [@RCoreTeam2018] that were run both locally
with the devtools package [@RCoreTeam2018], and on a public server --via
GitHub, using the continuous integration tool Travis CI (<https://travis-ci.org>). At
present, unit tests cover more than 50% of `datelife`'s code (<https://codecov.io/gh/phylotastic/datelife>). <!-- unit test make code robust, verifiable, debuggable, -->


\begin{center}
\textsc{Example}
\end{center}
<!--**Biological example: Testing DateLife accuracy**-->
<!--Bird (or reptile) chronograms, too long time...
finches is good
nothofagus
Look for all chronograms containing any birds
Or, look for chronograms containing small and old lineages
Determine which clade of birds has the more chronograms (have been dated
more times) and use that as biological example. Fringillidae is a good candidate
Find a clade with at least one chronogram containing all clade's species.
(Penguins look good, but they are giving weird results in SDM)

Remove this chronogram from datelife Results.

Make sdm and median trees and Compare

add taxa with different methods and Compare

Use ltts to compare for now.

think of a test to compare trees, topology- and date-wise

test improvement in synthetic otol tree after using bold sequences. Is it really better resolved? Probably not.
Do not include this analysis in paper, maybe in examples, but i think it is not worthy.
-->
In this section we demonstrate the types of outputs that can be obtained with `datelife`, using as an example the bird family Fringillidae of true finches. We performed a higher-taxon search to obtain data on lineage divergence available in `datelife`'s database for all recognised species within the Fringillidae (`r length(dq$cleaned_names)` species according to the Open Tree of Life taxonomy). There are `r length(dr)` chronograms containing at least two Fringillidae species, published in `r length(unique(names(dr)))` different studies (Fig. \ref{fig:schronograms1}).
<!-- In case we want to cite the studies in here, but they are already cited in the figure caption: [Fig. \ref{fig:schronograms1}; @barker2012going; @barker2015new; @burns2014phylogenetics; @claramunt2015new; @gibb2015new; @Hedges2015; @hooper2017chromosomal; @Jetz2012; @price2014niche]. -->
Data from these source chronograms was used to generate two types of summary chronograms, median and SDM. As explained in the "Description" section, data from source chronograms was first summarised into a single distance matrix (using the median and the SDM method respectively) and then the available node ages were used as fixed ages over a consensus tree topology, to obtain a fully dated tree with the program BLADJ (Fig. \ref{fig:summaries}). Median summary chronograms are older and have wider variation in maximum ages than chronograms obtained with SDM. With both methods, ages are generally consistent with source ages, but there are some biological examples in which this is not true (see Flaws, Limitations and Prospects section).
<!-- (Going from a summary distance matrix Using this method is better than using clustering algorithms  -->
<!-- and hierarchical clustering method which appears to push ages back -->

Data from source chronograms was also used to date tree topologies with no branch length information and trees with branch lengths in relative substitution rates (Figs. \ref{fig:cvbladj} and \ref{fig:cvbold}). As a form of cross validation, we used tree topologies from each study and calibrated them using information from all other source chronograms. In the absence of branch length data, the ages of internal nodes were approximately recovered in almost all cases (except for studies 3, and 5; Fig. \ref{fig:cvbladj}).
Maximum tree ages were only approximately recovered in one case (study 2; Fig. \ref{fig:cvbladj}).
<!-- We need to think abput the root! is it comparable across source chronograms?? not all of them have the same sampling... so probably not the same root and thus not comparable. Address this -->
To exemplify dating a tree with branch lengths provided by the user, we obtained DNA sequence data using the Barcode of Life Data System [BOLD; ratnasingham2007bold] and branch lengths were successfully generated for all source chronograms. However, dating with PATHd8 [@Britton2007] using congruified calibrations, was only successful in
<!-- two studies when expanding calibrations to make them agree (studies 3 and 5), and from  -->
three cases (studies 3, 5, and 9, shown in Fig. \ref{fig:cvbold}). From these, two trees have a different sampling than the original source chronogram, mainly because DNA data for some species is absent from the BOLD. Maximum ages are quite different from source chronograms, but this might be explained also by the differences in sampling between source chronograms and BOLD trees.
More examples and code used to generate these trees were developes on an open repository that is available for consultation and reuse at <https://github.com/LunaSare/datelife_examples>.

<!--Show code here, in the paper? Or show it only in the examples? Or only in the vignette? Write down what about datelife will be shown in each of these.
Paper should show outputs from web site and package.
Vignette will show the R code used to generate the outputs displayed here and in the website.
datelife examples will explain more on what is done in every step.
-->

\begin{center}
\textsc{Flaws, Limitations and Prospects}
\end{center}

The main goal of `datelife` is to make expert information on time of lineage divergence easily accesible for comparison, reuse, and reanalysis, to researchers in all areas of science and with all levels of expertise in the matter. It is a very fast tool that fulfills the quality of openness and does not require any expert biological knowledge from users --besides the names of the organisms they want to work with-- for any of its functionalities. However, it has many flaws. Some of them can be overcome, some of them might represent limitations.

Up to the time this manuscript was written, `datelife`'s chronogram database had `r length(opentree_chronograms$trees)` chronograms, pulled entirely from OToL's tree repository, the only public tree repository from where `datelife` can currently get chronograms to construct its database. This represents `r round(length(opentree_chronograms$trees)*100/3987, digits = 2)`% of the largest existing chronogram database, TimeTree, which has a collection of 3,998 chronograms as of `r format(sys.Date(), format="%B %d %Y")`. Unforutnately, TimeTree's database is not open for scientific reuse nor automatised data mining [@Kumar2017].
In 2015, a synthetic chronogram was constructed from 2,274 chronograms available at the time on the TimeTree database [@Hedges2015]. This is the only synthetic TimeTree chronogram that has been made publicly available and deposited on the OToL repository, and is part of datelife's database now. Hence, the amount of lineages represented in datelife's database is at least as substantial as TimeTree's, ensuring that some information will be available for any given taxon or lineage. Regrettably, this does not ensure that the full state of knowledge of time of divergence of the taxon/lineage will be available.
<!--
Even though datelifes chronograms cover this many species from the tree of life compared to this many covered by TimeTree (database or synthetic chronogram?)-->
Incorporation of more published chronograms into `datelife`'s database is crucial to improve its services. One option to increase our database is the Dryad data repository.  Methods to automatically mine chronograms from Dryad could be designed and implemented. However, Dryad's metadata system has no information to automatically detect branch length units, and those would still need to be determined on a second step, by a curator.
<!-- Can we mine journal databases? -->
<!-- At the same time, the prospect is to increase awereness and keep inviting and doing work encouraging scientists to make their data sets available in OToL's database. -->
Consequently, we would like to emphasize on the importance of sharing chronogram data for the benefit of the scientific community as a whole, into repositories that require expert input and manual curation, such as OToL's tree repository [@mctavish2015phylesystem].

Another potential concern comes from summary chronograms. We currently summarize by default all source chronograms that overlap with at least two taxa. Users can subset source data if they have reasons to choose some source chronograms over others. Strictly speaking, a good chronogram should reflect the real time of lineage divergence accurately and precisely. To our knowledge, there is no objective way to determine if an expert chronogram is better than another. Some criteria that have been put forward are the level of lineage sampling and the number of calibrations used. Scientists usually also favor chronograms constructed using primary calibrations (ages obtained from the fossil or geological record) to ones constructed with secondary calibrations (ages coming from other chronograms). It has been observed with simulations that divergence times inferred with secondary calibrations are significantly younger than those inferred with primary calibrations in analyses performed with bayesian inference methods when priors are implemented in similar ways in both analyses [@schenk2016sec]. Yet, there are different ways to use secondary calibrations
<!-- give some examples or alternative ways of using calibrations-->
and that same bias might not be encountered with dating methods that do not require setting priors, i.e., Maximum Likelihood methods such as r8s [@sanderson2003r8s]. Certainly, further studies are required to fully understand the effect of using secondary calibrations on time estimates and downstream anlyses.

Furthermore, even chronograms obtained with primary fossil data can show substantial variation in time estimates between clades, as observed from the comparison of source chronograms in the Fringillidae example.
This observation is often encountered in the literature
<!-- Different source chronograms often show substantial variation in clade ages  -->
(see, for example, the ongoing debate about crown group age of angiosperms [@barba2018constraining; @magallon2015metacalibrated; @sanderson2001sources; @ramshaw1972time]. For some studies, especially ones based on branch lengths (e.g., studies of species diversification, timing of evolutionary events, phenotypic trait evolution), using a different chronogram may return different results [@title2016macrophylogenies]. Stitching together these chronograms can create a larger tree that uses information from multiple studies, but the effect of uncertainties and errors here on downstream analyses is still largely unknown.

Summarizing chronograms might also imply summarizing fundamentally distinct evolutionary hypotheses. For example, two different researchers working on the same clade both carefully select and argument their choices of fossil calibrations. Still, if one researcher decides a fossil will calibrate the ingroup of a clade, while another researcher uses teh same one to calibrate outside the clade, the resulting age estimates will probably differ substantially (the placement of calibrations is proved to deeply affect estimated times of lineage divergence). Trying to summarize the resulting chronograms into a single one using simple summary statistics might erase all types of relevant information from the source chronograms. Accordingly, the prevailing view in our research community is that we should favor time of lineage divergence estimates obtained from a single analysis, using fossil data as primary sources of calibrations, and using fossils that have been widely discussed and curated as calibrations to date other trees, making sure that all data used in the analysis reflect a coherent evolutionary history [@antonelli2017supersmart]. <!-- Will this be implemented in future `datelife` versions? Probably not-->
However, the exercise of summarizing different chronograms ha sthe potential to help getting a single global evolutionary history for a lineage by putting together evidence from different hypothesis. Choosing the elements of the chronograms that we are going tp keep and the ones that we are going to discard is key, since we are potentially loosing important parts of the evolutionary history of a lineage that might only be reflected in source chronograms and not on the summary chronogram. <!-- Example -->


<!-- still requires more research. -->
<!-- Often, there is a large discrepancy in time of lineage divergence estimates across studies from expert knowledge, that is well known and documented for different groups of organisms [e.g., angiosperms; @magallon2015metacalibrated].  -->
<!-- Comparison of available age data for a wide range of organisms shown here suggest that this is a widespread phenomenon that requires further attention. -->
<!-- In the case of the example presented here, most chronograms are new, meaning that we expect that data was generated with high quality and the software used should be very advanced. However, this is just an assumption.  -->

Alternatively, one could try to choose the "best" chronogram from a set of possible evolutionary hypotheses. Several characteristics of the data used for dating analyses as well as from the output chronogram itself, could be used to score quality of source chronograms. Some characteristics that are often cited in published studies as a measure of improved age estimates as compared to previously published estimates are: quality of alignment (missing data, GC content), lineage sampling (strategy and proportion), phylogenetic and dating inference method, number of fossils used as calibrations, support for nodes and ages, and magnitude of confidence intervals. To facilitate subsetting of source chronograms following different criteria by the users, this information should be included as metadata manually entered by curators in the future.

<!-- Still, even if all source chronograms have been generated by excellent standards and using similar methods, the evolutionary history they depict might be very different.  -->

In other areas of biological research, such as ecology and conservation biology, it has been shown that at least some data on lineage divergence represents a relevant improvement for testing alternative hypothesis using phylogenetic distance [@Webb2008]. Hence, we integrated into datelife's workflow different ways of creating branch lengths in the absence of starting branch length information for taxa lacking this information (BLADJ option). Making up branch lengths in this or other ways is accepted in scientific publications: @Jetz2012, created a time-calibrated tree of all 9,993 bird species, where 67% had molecular data adn the rest was simulated; @rabosky2018inverse created a time-calibrated tree of 31,536 ray-finned fishes, of which only 37% had molecular data; @smith2018constructing constructed a tree of 353,185 seed plants where only 23% had molecular data. Taken to the extreme, one could make a fully resolved, calibrated tree of all modern and extinct taxa using a single taxonomy and a single calibration with the polytomy resolution and branch imputation methods. There has yet to be a thorough analysis of what can go wrong when one goes beyond the data in this way, so we urge caution; we also urge readers to follow the example of many of the large tree papers cited above and make sure results are substantially similar between trees fully reconstructed with molecular or other data, and trees that are reconstructed using taxonomy by resolving polytomies at random following a statistical model.

<!-- There is evidence that different chronograms will result in very different macroevolutionary histories inferred from them [@title2016macrophylogenies]. However, it is unknown how using very different age data could affect hypothesis testing in other areas of research and more effort is needed in that directions -->

\begin{center}
\textsc{Conclusions}
\end{center}

Divergence time information is key to many areas of evolutionary studies: trait evolution,
diversification, biogeography, macroecology and more. It is also crucial for science communication and education, but generating chronograms _de novo_ is difficult,
especially for those who want to use phylogenies but who are not systematists, or
do not have the time to acquire and develop the necessary knowledge and data curation skills. Moreover, years of primarily public funded research have resulted in vast amounts of chronograms that are already available on scientific publications, but hidden to the public and scientific community for reuse.

<!--Mention again uses shown here and outline potential uses not shown here-->
`datelife` allows easy and fast summarization of publicly available information
on time of lineage divergence. This provides a straightforward way to get an informed idea on the state of knowledge of the time frame of evolution of different regions of the tree of life, and allows identification of regions that require more research or that have conflicting information.
<!-- It can also summarize public data, or use it to generate new chronograms based on trees provided by the user. -->
Both summary and newly generated trees are useful to evaluate evolutionary hypotheses in different areas of research.  `datelife` helps with awareness of the existing variation in expert time of divergence data, and will foster exploration of the effect of alternative divergence time hypothesis on the results of analyses, nurturing a culture of more cautious interpretation of evolutionary results.

<!-- `datelife` is also part of the reproducibility of science movement. It strives to facilitate usage of public data sets as well as encouraging the making of information publicly available. Acknowledging source information and describing workflows and analyses. -->

<!-- it allows exploration of alternative taxonomic and phylogenetic schemes; -->
<!-- it allows rapid ; -->

\begin{center}
\textsc{Availability}
\end{center}

`datelife` is free and open source and it can be used through its current website
<http://www.datelife.org/query/>, through its R package, and through Phylotastic's project web portal <http://phylo.cs.nmsu.edu:3000/>.
`datelife`'s website is maintained using RStudio's shiny server and the shiny package open infrastructure, as well as Docker.
`datelife`'s R package stable version `r availability`
for installation from the CRAN repository (<https://cran.r-project.org/package=datelife>)
using the command `install.packages(pkgs = "datelife")` from within R. Development versions
are available from the GitHub repository (<https://github.com/phylotastic/datelife>)
and can be installed using the command `devtools::install_github("phylotastic/datelife")`.


\begin{center}
\textsc{Supplementary Material}
\end{center}

Code used to generate all versions of this manuscript, the biological examples, as well as the benchmark of functionalities are respectively in the [datelife_paper1](https://github.com/LunaSare/datelife_paper1), [datelife_examples](https://github.com/LunaSare/datelife_examples), and [datelife_benchmark](https://github.com/LunaSare/datelife_benchmark) repositories in LLSR GitHub account.
<!-- Vignettes?
Dryad also?? since it is all in github already, maybe not necessary-->

\begin{center}
\textsc{Funding}
\end{center}

Funding was provided by the US National Science Foundation (NSF) grants ABI-1458603  to Datelife project and DBI-0905606 to the National Evolutionary Synthesis Center (NESCent), and the Phylotastic project Grant ABI-1458572.

<!-- Any separate funding from the University of Tennessee, Knoxville?? -->


\begin{center}
\textsc{Acknowledgements}
\end{center}

We thank colleagues from the O'Meara Lab at the University
of Tennesse Knoxville for suggestions, discussions and software testing.
The late National Evolutionary Synthesis Center (NESCent), which sponsored hackathons
that led to initial work on this project. The team that assembled `datelife`'s first proof of concept: Tracy Heath, Jonathan Eastman, Peter Midford, Joseph Brown, Matt Pennell, Mike Alfaro, and Luke Harmon.
The Open Tree of Life project that provides the open, metadata rich repository of
trees used for `datelife`.
The many scientists who publish their chronograms in an open, reusable form, and
the scientists who curate them for deposition in the Open Tree of Life repository.
The NSF for funding nearly all the above, in addition to the ABI grant that funded this project itself.
