\PassOptionsToPackage{unicode=true}{hyperref} % options for packages loaded elsewhere
%DIF LATEXDIFF DIFFERENCE FILE
%DIF DEL manuscript-1st-submission.tex   Tue Feb 21 00:16:03 2023
%DIF ADD manuscript.tex                  Mon May 22 12:15:40 2023
\PassOptionsToPackage{hyphens}{url}
%
\documentclass[english,man]{apa6}
\usepackage{lmodern}
\usepackage{amssymb,amsmath}
\usepackage{ifxetex,ifluatex}
\usepackage{fixltx2e} % provides \textsubscript
\ifnum 0\ifxetex 1\fi\ifluatex 1\fi=0 % if pdftex
  \usepackage[T1]{fontenc}
  \usepackage[utf8]{inputenc}
  \usepackage{textcomp} % provides euro and other symbols
\else % if luatex or xelatex
  \usepackage{unicode-math}
  \defaultfontfeatures{Ligatures=TeX,Scale=MatchLowercase}
\fi
% use upquote if available, for straight quotes in verbatim environments
\IfFileExists{upquote.sty}{\usepackage{upquote}}{}
% use microtype if available
\IfFileExists{microtype.sty}{%
\usepackage[]{microtype}
\UseMicrotypeSet[protrusion]{basicmath} % disable protrusion for tt fonts
}{}
\IfFileExists{parskip.sty}{%
\usepackage{parskip}
}{% else
\setlength{\parindent}{0pt}
\setlength{\parskip}{6pt plus 2pt minus 1pt}
}
\usepackage{hyperref}
\hypersetup{
            pdftitle={DateLife: leveraging databases and analytical tools to reveal the dated Tree of Life},
            pdfauthor={Luna L. S치nchez Reyes1,2, Emily Jane McTavish1, \& Brian O'Meara2},
            pdfborder={0 0 0},
            breaklinks=true}
\urlstyle{same}  % don't use monospace font for urls
\usepackage{graphicx,grffile}
\makeatletter
\def\maxwidth{\ifdim\Gin@nat@width>\linewidth\linewidth\else\Gin@nat@width\fi}
\def\maxheight{\ifdim\Gin@nat@height>\textheight\textheight\else\Gin@nat@height\fi}
\makeatother
% Scale images if necessary, so that they will not overflow the page
% margins by default, and it is still possible to overwrite the defaults
% using explicit options in \includegraphics[width, height, ...]{}
\setkeys{Gin}{width=\maxwidth,height=\maxheight,keepaspectratio}
\setlength{\emergencystretch}{3em}  % prevent overfull lines
\providecommand{\tightlist}{%
  \setlength{\itemsep}{0pt}\setlength{\parskip}{0pt}}
\setcounter{secnumdepth}{0}

% set default figure placement to htbp
\makeatletter
\def\fps@figure{htbp}
\makeatother

\DeclareUnicodeCharacter{0301}{*************************************}

% leave the figure captions outside\ caption{} if you want them
% to be formatted in the same way as the general text (double spaced and linenumbered)
% line numbers for captions trick from https://latex.org/forum/viewtopic.php?t=34614

\usepackage{ragged2e}

\usepackage{caption}

\DeclareCaptionFont{linenumbers}{\internallinenumbers}

\usepackage{float}
\restylefloat{table}

\captionsetup[figure]{font+=linenumbers, labelfont={sc}, labelformat={default}, labelsep=period, name={Figure}}

\captionsetup[table]{font+=linenumbers, labelfont={sc}, labelformat={default}, labelsep=period, name={Table}}
% Manuscript styling
\usepackage{upgreek}
\captionsetup{font=singlespacing,justification=justified}

% Table formatting
\usepackage{longtable}
\usepackage{lscape}
% \usepackage[counterclockwise]{rotating}   % Landscape page setup for large tables
\usepackage{multirow}		% Table styling
\usepackage{tabularx}		% Control Column width
\usepackage[flushleft]{threeparttable}	% Allows for three part tables with a specified notes section
\usepackage{threeparttablex}            % Lets threeparttable work with longtable

% Create new environments so endfloat can handle them
% \newenvironment{ltable}
%   {\begin{landscape}\centering\begin{threeparttable}}
%   {\end{threeparttable}\end{landscape}}
\newenvironment{lltable}{\begin{landscape}\centering\begin{ThreePartTable}}{\end{ThreePartTable}\end{landscape}}

% Enables adjusting longtable caption width to table width
% Solution found at http://golatex.de/longtable-mit-caption-so-breit-wie-die-tabelle-t15767.html
\makeatletter
\newcommand\LastLTentrywidth{1em}
\newlength\longtablewidth
\setlength{\longtablewidth}{1in}
\newcommand{\getlongtablewidth}{\begingroup \ifcsname LT@\roman{LT@tables}\endcsname \global\longtablewidth=0pt \renewcommand{\LT@entry}[2]{\global\advance\longtablewidth by ##2\relax\gdef\LastLTentrywidth{##2}}\@nameuse{LT@\roman{LT@tables}} \fi \endgroup}

% \setlength{\parindent}{0.5in}
% \setlength{\parskip}{0pt plus 0pt minus 0pt}

% Overwrite redefinition of paragraph and subparagraph by the default LaTeX template
% See https://github.com/crsh/papaja/issues/292
\makeatletter
\renewcommand{\paragraph}{\@startsection{paragraph}{4}{\parindent}%
  {0\baselineskip \@plus 0.2ex \@minus 0.2ex}%
  {-1em}%
  {\normalfont\normalsize\bfseries\itshape\typesectitle}}

\renewcommand{\subparagraph}[1]{\@startsection{subparagraph}{5}{1em}%
  {0\baselineskip \@plus 0.2ex \@minus 0.2ex}%
  {-\z@\relax}%
  {\normalfont\normalsize\itshape\hspace{\parindent}{#1}\textit{\addperi}}{\relax}}
\makeatother

% \usepackage{etoolbox}
\makeatletter
\patchcmd{\HyOrg@maketitle}
  {\section{\normalfont\normalsize\abstractname}}
  {\section*{\normalfont\normalsize\abstractname}}
  {}{\typeout{Failed to patch abstract.}}
\patchcmd{\HyOrg@maketitle}
  {\section{\protect\normalfont{\@title}}}
  {\section*{\protect\normalfont{\@title}}}
  {}{\typeout{Failed to patch title.}}
\makeatother

\usepackage{xpatch}
\makeatletter
\xapptocmd\appendix
  {\xapptocmd\section
    {\addcontentsline{toc}{section}{\appendixname\ifoneappendix\else~\theappendix\fi\\: #1}}
    {}{\InnerPatchFailed}%
  }
{}{\PatchFailed}
\DeclareDelayedFloatFlavor{ThreePartTable}{table}
\DeclareDelayedFloatFlavor{lltable}{table}
\DeclareDelayedFloatFlavor*{longtable}{table}
\makeatletter
\renewcommand{\efloat@iwrite}[1]{\immediate\expandafter\protected@write\csname efloat@post#1\endcsname{}}
\makeatother
\usepackage{lineno}

\linenumbers
\usepackage{csquotes}
\ifnum 0\ifxetex 1\fi\ifluatex 1\fi=0 % if pdftex
  \usepackage[shorthands=off,main=english]{babel}
\else
  % load polyglossia as late as possible as it *could* call bidi if RTL lang (e.g. Hebrew or Arabic)
  \usepackage{polyglossia}
  \setmainlanguage[]{english}
\fi

\title{DateLife: leveraging databases and analytical tools to reveal the dated Tree of Life}
\author{Luna L. S치nchez Reyes\textsuperscript{1,2}, Emily Jane McTavish\textsuperscript{1}, \& Brian O'Meara\textsuperscript{2}}
\date{}


\shorttitle{DateLife: revealing the dated Tree of Life}

\authornote{

%DIF 165c165
%DIF < School of Natural Sciences, University of California, Merced, 258 Science and Engineering Building 1, Merced, CA 95340, USA.
%DIF -------
School of Natural Sciences, University of California, Merced, Science and Engineering Building 1, Merced, CA 95340, USA. %DIF > 
%DIF -------

Department of Ecology and Evolutionary Biology, University of Tennessee, Knoxville, 446 Hesler Biology Building, Knoxville, TN 37996, USA.

The authors made the following contributions. Luna L. S치nchez Reyes: Data curation, Investigation, Software, Visualization, Validation, Writing - Original Draft Preparation, Writing - Review \& Editing; Emily Jane McTavish: Resources, Software, Writing - Review \& Editing; Brian O'Meara: Conceptualization, Funding acquisition, Methodology, Resources, Software, Supervision, Writing - Review \& Editing.

Correspondence concerning this article should be addressed to Luna L. S치nchez Reyes, . E-mail: \href{mailto:sanchez.reyes.luna@gmail.com}{\nolinkurl{sanchez.reyes.luna@gmail.com}}

}

\affiliation{\vspace{0.5cm}\textsuperscript{1} University of California, Merced, USA\\\textsuperscript{2} University of Tennessee, Knoxville, USA}
%DIF PREAMBLE EXTENSION ADDED BY LATEXDIFF
%DIF UNDERLINE PREAMBLE %DIF PREAMBLE
\RequirePackage[normalem]{ulem} %DIF PREAMBLE
\RequirePackage{color}\definecolor{RED}{rgb}{1,0,0}\definecolor{BLUE}{rgb}{0,0,1} %DIF PREAMBLE
\providecommand{\DIFaddtex}[1]{{\protect\color{blue}\uwave{#1}}} %DIF PREAMBLE
\providecommand{\DIFdeltex}[1]{{\protect\color{red}\sout{#1}}}                      %DIF PREAMBLE
%DIF SAFE PREAMBLE %DIF PREAMBLE
\providecommand{\DIFaddbegin}{} %DIF PREAMBLE
\providecommand{\DIFaddend}{} %DIF PREAMBLE
\providecommand{\DIFdelbegin}{} %DIF PREAMBLE
\providecommand{\DIFdelend}{} %DIF PREAMBLE
\providecommand{\DIFmodbegin}{} %DIF PREAMBLE
\providecommand{\DIFmodend}{} %DIF PREAMBLE
%DIF FLOATSAFE PREAMBLE %DIF PREAMBLE
\providecommand{\DIFaddFL}[1]{\DIFadd{#1}} %DIF PREAMBLE
\providecommand{\DIFdelFL}[1]{\DIFdel{#1}} %DIF PREAMBLE
\providecommand{\DIFaddbeginFL}{} %DIF PREAMBLE
\providecommand{\DIFaddendFL}{} %DIF PREAMBLE
\providecommand{\DIFdelbeginFL}{} %DIF PREAMBLE
\providecommand{\DIFdelendFL}{} %DIF PREAMBLE
%DIF HYPERREF PREAMBLE %DIF PREAMBLE
\providecommand{\DIFadd}[1]{\texorpdfstring{\DIFaddtex{#1}}{#1}} %DIF PREAMBLE
\providecommand{\DIFdel}[1]{\texorpdfstring{\DIFdeltex{#1}}{}} %DIF PREAMBLE
\newcommand{\DIFscaledelfig}{0.5}
%DIF HIGHLIGHTGRAPHICS PREAMBLE %DIF PREAMBLE
\RequirePackage{settobox} %DIF PREAMBLE
\RequirePackage{letltxmacro} %DIF PREAMBLE
\newsavebox{\DIFdelgraphicsbox} %DIF PREAMBLE
\newlength{\DIFdelgraphicswidth} %DIF PREAMBLE
\newlength{\DIFdelgraphicsheight} %DIF PREAMBLE
% store original definition of \includegraphics %DIF PREAMBLE
\LetLtxMacro{\DIFOincludegraphics}{\includegraphics} %DIF PREAMBLE
\newcommand{\DIFaddincludegraphics}[2][]{{\color{blue}\fbox{\DIFOincludegraphics[#1]{#2}}}} %DIF PREAMBLE
\newcommand{\DIFdelincludegraphics}[2][]{% %DIF PREAMBLE
\sbox{\DIFdelgraphicsbox}{\DIFOincludegraphics[#1]{#2}}% %DIF PREAMBLE
\settoboxwidth{\DIFdelgraphicswidth}{\DIFdelgraphicsbox} %DIF PREAMBLE
\settoboxtotalheight{\DIFdelgraphicsheight}{\DIFdelgraphicsbox} %DIF PREAMBLE
\scalebox{\DIFscaledelfig}{% %DIF PREAMBLE
\parbox[b]{\DIFdelgraphicswidth}{\usebox{\DIFdelgraphicsbox}\\[-\baselineskip] \rule{\DIFdelgraphicswidth}{0em}}\llap{\resizebox{\DIFdelgraphicswidth}{\DIFdelgraphicsheight}{% %DIF PREAMBLE
\setlength{\unitlength}{\DIFdelgraphicswidth}% %DIF PREAMBLE
\begin{picture}(1,1)% %DIF PREAMBLE
\thicklines\linethickness{2pt} %DIF PREAMBLE
{\color[rgb]{1,0,0}\put(0,0){\framebox(1,1){}}}% %DIF PREAMBLE
{\color[rgb]{1,0,0}\put(0,0){\line( 1,1){1}}}% %DIF PREAMBLE
{\color[rgb]{1,0,0}\put(0,1){\line(1,-1){1}}}% %DIF PREAMBLE
\end{picture}% %DIF PREAMBLE
}\hspace*{3pt}}} %DIF PREAMBLE
} %DIF PREAMBLE
\LetLtxMacro{\DIFOaddbegin}{\DIFaddbegin} %DIF PREAMBLE
\LetLtxMacro{\DIFOaddend}{\DIFaddend} %DIF PREAMBLE
\LetLtxMacro{\DIFOdelbegin}{\DIFdelbegin} %DIF PREAMBLE
\LetLtxMacro{\DIFOdelend}{\DIFdelend} %DIF PREAMBLE
\DeclareRobustCommand{\DIFaddbegin}{\DIFOaddbegin \let\includegraphics\DIFaddincludegraphics} %DIF PREAMBLE
\DeclareRobustCommand{\DIFaddend}{\DIFOaddend \let\includegraphics\DIFOincludegraphics} %DIF PREAMBLE
\DeclareRobustCommand{\DIFdelbegin}{\DIFOdelbegin \let\includegraphics\DIFdelincludegraphics} %DIF PREAMBLE
\DeclareRobustCommand{\DIFdelend}{\DIFOaddend \let\includegraphics\DIFOincludegraphics} %DIF PREAMBLE
\LetLtxMacro{\DIFOaddbeginFL}{\DIFaddbeginFL} %DIF PREAMBLE
\LetLtxMacro{\DIFOaddendFL}{\DIFaddendFL} %DIF PREAMBLE
\LetLtxMacro{\DIFOdelbeginFL}{\DIFdelbeginFL} %DIF PREAMBLE
\LetLtxMacro{\DIFOdelendFL}{\DIFdelendFL} %DIF PREAMBLE
\DeclareRobustCommand{\DIFaddbeginFL}{\DIFOaddbeginFL \let\includegraphics\DIFaddincludegraphics} %DIF PREAMBLE
\DeclareRobustCommand{\DIFaddendFL}{\DIFOaddendFL \let\includegraphics\DIFOincludegraphics} %DIF PREAMBLE
\DeclareRobustCommand{\DIFdelbeginFL}{\DIFOdelbeginFL \let\includegraphics\DIFdelincludegraphics} %DIF PREAMBLE
\DeclareRobustCommand{\DIFdelendFL}{\DIFOaddendFL \let\includegraphics\DIFOincludegraphics} %DIF PREAMBLE
%DIF COLORLISTINGS PREAMBLE %DIF PREAMBLE
\RequirePackage{listings} %DIF PREAMBLE
\RequirePackage{color} %DIF PREAMBLE
\lstdefinelanguage{DIFcode}{ %DIF PREAMBLE
%DIF DIFCODE_UNDERLINE %DIF PREAMBLE
  moredelim=[il][\color{red}\sout]{\%DIF\ <\ }, %DIF PREAMBLE
  moredelim=[il][\color{blue}\uwave]{\%DIF\ >\ } %DIF PREAMBLE
} %DIF PREAMBLE
\lstdefinestyle{DIFverbatimstyle}{ %DIF PREAMBLE
	language=DIFcode, %DIF PREAMBLE
	basicstyle=\ttfamily, %DIF PREAMBLE
	columns=fullflexible, %DIF PREAMBLE
	keepspaces=true %DIF PREAMBLE
} %DIF PREAMBLE
\lstnewenvironment{DIFverbatim}{\lstset{style=DIFverbatimstyle}}{} %DIF PREAMBLE
\lstnewenvironment{DIFverbatim*}{\lstset{style=DIFverbatimstyle,showspaces=true}}{} %DIF PREAMBLE
%DIF END PREAMBLE EXTENSION ADDED BY LATEXDIFF

\begin{document}
\maketitle

\begin{center}
Abstract
\end{center}

\DIFdelbegin \DIFdel{Achieving a high-quality reconstruction of a phylogenetic tree }\DIFdelend \DIFaddbegin \DIFadd{Chronograms --phylogenies }\DIFaddend with branch lengths proportional to \DIFdelbegin \DIFdel{absolute time (chronogram) is a difficult and time-consuming task. But the increased availability of fossil and molecular data , and time-efficient analytical techniques has resulted in many recent publications of large chronograms for a large number and wide diversity of organisms.
Knowledge of the evolutionary time frame of organisms is key for research in the natural sciences.
It also represent valuable information }\DIFdelend \DIFaddbegin \DIFadd{time-- represent key data on timing of evolutionary events for the study of natural processes in many areas of biological research.
Chronograms also provide valuable information that can be used }\DIFaddend for education, science communication, and \DIFaddbegin \DIFadd{conservation }\DIFaddend policy decisions.
\DIFdelbegin \DIFdel{When chronograms are shared in public and open databases, this wealth of expertly-curated and peer-reviewed data on evolutionary timeframe is exposed in a programatic and reusable way, as intensive and localized efforts have improved data sharing practices, as well as incentivizited open science in biology}\DIFdelend \DIFaddbegin \DIFadd{Yet, achieving a high-quality reconstruction of a chronogram is a difficult and resource-consuming task}\DIFaddend .
Here we present DateLife, a service implemented as an R package and an R Shiny \DIFdelbegin \DIFdel{website }\DIFdelend \DIFaddbegin \DIFadd{web }\DIFaddend application available at www.datelife.org, that provides \DIFdelbegin \DIFdel{functionalities }\DIFdelend \DIFaddbegin \DIFadd{services }\DIFaddend for efficient and easy \DIFdelbegin \DIFdel{finding}\DIFdelend \DIFaddbegin \DIFadd{discovery}\DIFaddend , summary, reuse, and reanalysis of \DIFaddbegin \DIFadd{node age data mined from a curated database of }\DIFaddend expert, peer-reviewed, \DIFdelbegin \DIFdel{public data on time frame of evolution}\DIFdelend \DIFaddbegin \DIFadd{and openly available chronograms}\DIFaddend .
The main DateLife workflow \DIFdelbegin \DIFdel{constructs a chronogram for any given combination of taxon names by searching a }\DIFdelend \DIFaddbegin \DIFadd{starts with one or more scientific taxon names provided by a user. Names are processed and standardized to a unified taxonomy, allowing DateLife to run a name match across its }\DIFaddend local chronogram database \DIFdelbegin \DIFdel{constructed and curated from the }\DIFdelend \DIFaddbegin \DIFadd{that is curated from }\DIFaddend Open Tree of Life\DIFdelbegin \DIFdel{Phylesystem phylogenetic database, which incorporates phylogenetic data from the TreeBASE database as well.
We implement and test methods for summarizing time data from multiple source chronograms using supertree and congruification algorithms, and using age data extracted from source chronogramsas secondary calibration points to add branch lengths proportional to absolute time to a tree topology.
DateLife will be useful to }\DIFdelend \DIFaddbegin \DIFadd{'s phylogenetic repository, and extract all chronograms that contain at least two queried taxon names, along with their metadata.
Finally, node ages from matching chronograms are mapped to corresponding nodes from a chosen tree topology using the congruification algorithm. Congruified node ages are used as secondary calibrations to date the chosen topology, with or without initial branch lengths, using different phylogenetic dating methods such as BLADJ, treePL, PATHd8 and MrBayes.
We performed a cross-validation test to compare node ages resulting from a DateLife analysis (i.e, phylogenetic dating using secondary calibrations) to those from the original chronograms (i.e, obtained with primary calibrations), and found that DateLife's node age estimates are consistent with the age estimates from the original chronograms, with the largest variation in ages occurring around topologically deeper nodes.
Results from any software for scientific analysis can only be as good as the data used as input, we highlight the importance of considering the results of a DateLife analysis in the context of the input chronograms.
We encourage the use of DateLife to help }\DIFaddend increase awareness of the existing \DIFdelbegin \DIFdel{variation in alternative hypothesis of evolutionary time }\DIFdelend \DIFaddbegin \DIFadd{disparities among alternative hypotheses of dates }\DIFaddend for the same \DIFdelbegin \DIFdel{organisms, and can foster }\DIFdelend \DIFaddbegin \DIFadd{diversification events, and to support }\DIFaddend exploration of the effect of alternative \DIFdelbegin \DIFdel{evolutionary timing hypotheses on the results of }\DIFdelend \DIFaddbegin \DIFadd{chronogram hypotheses on }\DIFaddend downstream analyses, providing a framework for a more informed interpretation of evolutionary results.

\emph{Keywords}: Tree; Phylogeny; Scaling; Dating; Ages; Divergence times; Open Science; Congruification; Supertree; Calibrations; Secondary calibrations\DIFaddbegin \DIFadd{.
}\DIFaddend 

Word count: \DIFdelbegin \DIFdel{5393
}\DIFdelend \DIFaddbegin \DIFadd{6715
}\DIFaddend 

\newpage

Chronograms --phylogenies with branch lengths proportional to time-- provide key data on evolutionary time frame for the study of natural processes in many areas of biological research, such as \DIFaddbegin \DIFadd{comparative analysis (Freckleton, Harvey, \& Pagel, 2002; Harvey, Pagel, \& others, 1991), }\DIFaddend developmental biology (Delsuc et al., 2018; Laubichler \& Maienschein, 2009), conservation biology \DIFaddbegin \DIFadd{and ecology }\DIFaddend (Felsenstein, 1985; Webb, 2000), historical biogeography (Posadas, Crisci, \& Katinas, 2006), and species diversification (Magallon \& Sanderson, 2001; Morlon, 2014).

Building a chronogram is not an easy task.
It requires obtaining and curating \DIFdelbegin \DIFdel{data }\DIFdelend \DIFaddbegin \DIFadd{a homology hypothesis }\DIFaddend to construct a phylogeny, selecting and placing appropriate calibrations on the phylogeny using independent age data points from the fossil record or other dated events, and inferring \DIFdelbegin \DIFdel{the }\DIFdelend \DIFaddbegin \DIFadd{a }\DIFaddend full dated tree\DIFdelbegin \DIFdel{;
it also generally requires }\DIFdelend \DIFaddbegin \DIFadd{. All of this entails }\DIFaddend specialized biological training, taxonomic domain knowledge, and a \DIFdelbegin \DIFdel{non-negligible }\DIFdelend \DIFaddbegin \DIFadd{significant }\DIFaddend amount of research time, computational resources and funding.

Here we present the DateLife project which has the main goal of \DIFdelbegin \DIFdel{capturing }\DIFdelend \DIFaddbegin \DIFadd{extracting and exposing }\DIFaddend age data from published chronograms, \DIFdelbegin \DIFdel{and making these }\DIFdelend \DIFaddbegin \DIFadd{making age }\DIFaddend data readily accessible to the \DIFaddbegin \DIFadd{wider }\DIFaddend community for reuse and reanalysis \DIFdelbegin \DIFdel{, for }\DIFdelend \DIFaddbegin \DIFadd{in }\DIFaddend research, teaching, \DIFdelbegin \DIFdel{and }\DIFdelend science communication and \DIFaddbegin \DIFadd{conservation }\DIFaddend policy.
DateLife's core software application is available as an R package (Sanchez-Reyes et al., 2022), and as an online Rshiny interactive website at www.datelife.org. It features key elements for scientific reproducibility, such as a \DIFaddbegin \DIFadd{curated, }\DIFaddend versioned, open and fully public \DIFdelbegin \DIFdel{source }\DIFdelend \DIFaddbegin \DIFadd{chronogram }\DIFaddend database (McTavish et al., 2015) \DIFdelbegin \DIFdel{, data stored and available in a computer readable }\DIFdelend \DIFaddbegin \DIFadd{that stores data in a computer-readable }\DIFaddend format (Vos et al., 2012)\DIFdelbegin \DIFdel{, }\DIFdelend \DIFaddbegin \DIFadd{; }\DIFaddend automated and programmatic ways of accessing \DIFdelbegin \DIFdel{the data }\DIFdelend \DIFaddbegin \DIFadd{and downloading the data, also in a computer-readable format }\DIFaddend (Stoltzfus et al., 2013)\DIFaddbegin \DIFadd{; }\DIFaddend and methods to summarize and compare the data.

\begin{center}
\textsc{Description}
\end{center}

DateLife's core software \DIFdelbegin \DIFdel{application consists of }\DIFdelend \DIFaddbegin \DIFadd{applications are implemented in }\DIFaddend the R package \texttt{datelife}\DIFdelbegin \DIFdel{. Its current stable version -- v0.6.6,
is available from The Comprehensive R Archive Network (CRAN) repository (Sanchez-Reyes et al., 2022), }\DIFdelend \DIFaddbegin \DIFadd{,
}\DIFaddend and relies on functionalities from \DIFdelbegin \DIFdel{various }\DIFdelend \DIFaddbegin \DIFadd{other }\DIFaddend biological R packages:
ape (Paradis, Claude, \& Strimmer, 2004),
bold (Chamberlain, 2018),
geiger (Pennell et al., 2014),
\DIFaddbegin \DIFadd{msa (Bodenhofer, Bonatesta, Horej코-Kainrath, \& Hochreiter, 2015),
}\DIFaddend paleotree (Bapst, 2012),
phyloch (Heibl, 2008),
phylocomr (Ooms \& Chamberlain, 2018),
phytools (Revell, 2012),
rotl (Michonneau, Brown, \& Winter, 2016), and
taxize (Chamberlain, 2018; Chamberlain \& Sz칬cs, 2013).
Figure \DIFdelbegin \DIFdel{\ref{fig:workflow} }\DIFdelend \DIFaddbegin \DIFadd{1 }\DIFaddend provides a graphical summary of the three main steps of the DateLife workflow: creating a search query, searching a database, and summarizing results from the search.

\begin{center}
\emph{Creating a Search Query}
\end{center}

DateLife starts by processing an input consisting of \DIFdelbegin \DIFdel{at least two taxonnames, which }\DIFdelend \DIFaddbegin \DIFadd{the scientific name of at least one taxon. Multiple input names }\DIFaddend can be provided as a comma separated character string or as tip labels on a tree.
If the input is a tree, it can be provided as a classic newick character string (Archie et al., 1986), or as a \enquote{phylo} R object (Paradis et al., 2004).
The input tree is not required to have branch lengths, and its topology is used in the summary steps described in the next section.

DateLife \DIFdelbegin \DIFdel{accepts scientific names that can belong to any inclusive taxonomic group (e. g., genus, family, tribe, etc.) or a binomial species name. Subspecies and variants are ignored. If an input taxon name belongs to an inclusive taxonomic group, DateLife has two alternative behaviors defined by }\DIFdelend \DIFaddbegin \DIFadd{processes input scientific names using a Taxonomic Name Resolution Service (TNRS), which increases the probability of correctly finding the queried taxon names in the chronogram database. TNRS detects, corrects and standardizes name misspellings and typos, variant spellings and authorities, and nomenclatural synonyms to a single taxonomic standard (Boyle et al., 2013). TNRS also allows to correctly choose between homonyms, by considering other taxa provided as input to infer the taxonomic context of }\DIFaddend the \DIFdelbegin %DIFDELCMD < \enquote{get species from taxon} %%%
\DIFdel{flag. If the flag is active, DateLife retrieves all species names within the inclusive taxonomic group following a standard taxonomy of choice, and adds them to the input string. Taxonomies currently supported by DateLife are }\DIFdelend \DIFaddbegin \DIFadd{homonym. DateLife implements TNRS using the }\DIFaddend Open Tree of Life (OpenTree) unified Taxonomy (OTT, \DIFaddbegin \DIFadd{Open Tree Of Life et al., 2016; }\DIFaddend Rees \& Cranston, 2017) \DIFdelbegin \DIFdel{, }\DIFdelend \DIFaddbegin \DIFadd{as standard, storing taxonomic identification numbers (OTT ids) for further processing and analysis. Other taxonomies currently supported by DateLife are }\DIFaddend the National Center of Biotechnology Information (NCBI) taxonomic database (Schoch et al., 2020), the Global Biodiversity Information Facility (GBIF) taxonomic backbone (GBIF Secretariat, 2022), and the Interim Register of Marine and \DIFdelbegin \DIFdel{Nonmarine }\DIFdelend \DIFaddbegin \DIFadd{Non-marine }\DIFaddend Genera (IRMNG) database (Rees et al., 2017).
\DIFdelbegin \DIFdel{If }\DIFdelend \DIFaddbegin 

\DIFadd{Besides binomial species names, DateLife accepts scientific names from any inclusive taxonomic group (e.g., genus, family, tribe), as well as subspecific taxonomic variants (e.g., subspecies, variants, strains). If a taxon name belongs to an inclusive taxonomic group, DateLife has two alternative behaviors defined by the }\enquote{get species from taxon} \DIFadd{flag. If the flag is active, DateLife retrieves all species names within a taxonomic group provided, from a standard taxonomy of choice, and adds them to }\DIFaddend the \DIFdelbegin \DIFdel{flag is inactive, DateLife excludes any taxon names above the species level from the }\DIFdelend search query. \DIFdelbegin %DIFDELCMD < 

%DIFDELCMD < %%%
\DIFdel{DateLife processes input scientific names using a Taxonomic Name Resolution Service (TNRS), which increases the probability of correctly finding the queried taxon names in the chronogram database. TNRS detects, corrects and standardizes name misspellings and typos, variant spellings and authorities, and nomenclatural synonyms to a single taxonomic standard (Boyle et al., 2013). DateLife implements TNRS with OTT as standard (Open Tree Of Life et al.
, 2016; Rees \& Cranston, 2017), storing taxonomic identification numbers for further processing.
}%DIFDELCMD < 

%DIFDELCMD < %%%
\DIFdel{The processed input }\DIFdelend \DIFaddbegin \DIFadd{In this case, subspecific variants are excluded.
If the flag is inactive, DateLife excludes inclusive taxon names from the search query, and species and subspecific variant names are processed as provided by the user.
The processed }\DIFaddend taxon names are saved as an R object of a newly defined class, \texttt{datelifeQuery}, that is used in the following steps. This object contains the \DIFdelbegin \DIFdel{standardized names }\DIFdelend \DIFaddbegin \DIFadd{input names standardized to a taxonomy of choice (OTT by default)}\DIFaddend , the corresponding OTT \DIFdelbegin \DIFdel{identification }\DIFdelend \DIFaddbegin \DIFadd{id }\DIFaddend numbers, and the topology of \DIFdelbegin \DIFdel{the input treeif any }\DIFdelend \DIFaddbegin \DIFadd{an input tree, if one }\DIFaddend was provided.

\begin{center}
\emph{Searching a Chronogram Database}
\end{center}

At the time of writing of this manuscript
(\DIFdelbegin \DIFdel{Jun 22, 2022}\DIFdelend \DIFaddbegin \DIFadd{May 15, 2023}\DIFaddend ),
DateLife's chronogram database latest version consist of 253 chronograms published in 187 different studies. It is curated from OpenTree's phylogenetic database, the Phylesystem, which constitutes an open source of expert and peer-reviewed phylogenetic knowledge with rich metadata (McTavish et al., 2015), which allows automatic and reproducible assembly of our chronogram database. Datelife's chronogram database is navigable as an R data object within the \texttt{datelife} R package.

A unique feature of the Phylesystem is that any user can add new published, state-of-the-art chronograms any time, through \DIFdelbegin \DIFdel{their }\DIFdelend \DIFaddbegin \DIFadd{OpenTree's }\DIFaddend curator application (\url{https://tree.opentreeoflife.org/curator}). As chronograms are added to Phylesystem, they \DIFdelbegin \DIFdel{are }\DIFdelend \DIFaddbegin \DIFadd{can be }\DIFaddend incorporated into the chronogram database of the \texttt{datelife} \DIFdelbegin \DIFdel{package}\DIFdelend \DIFaddbegin \DIFadd{R package, which is currently manually updated as new chronogram data is added to Phylesystem}\DIFaddend . The updated database is assigned a new version number, followed by a package release on CRAN.
\DIFaddbegin \DIFadd{Users can directly run }\DIFaddend \texttt{datelife} \DIFdelbegin \DIFdel{'s chronogram database is updated as new chronogram data is added to Phylesystem, at a minimum of once a month and a maximum of every 6 months.
Users can also implement functions from the }\texttt{\DIFdel{datelife}} %DIFAUXCMD
\DIFdel{R package }\DIFdelend \DIFaddbegin \DIFadd{functions }\DIFaddend to trigger an update of \DIFdelbegin \DIFdel{the }\DIFdelend \DIFaddbegin \DIFadd{their }\DIFaddend local chronogram database, to incorporate any new chronograms to \DIFdelbegin \DIFdel{the user's }\DIFdelend \DIFaddbegin \DIFadd{their }\DIFaddend DateLife analysis before an official database update is released on CRAN.

A DateLife search is implemented by matching processed taxon names provided by the user to tip labels in the chronogram database. Chronograms with at least two matching taxon names on their tip labels are identified and pruned down to preserve only the matched taxa.
These matching pruned chronograms are referred to as source chronograms.
Total distance \DIFdelbegin \DIFdel{(}\DIFdelend in units of \DIFdelbegin \DIFdel{millions of years }\DIFdelend \DIFaddbegin \DIFadd{million years (Myr}\DIFaddend ) between taxon pairs within each source chronogram are stored as a patristic distance matrix (\DIFdelbegin \DIFdel{Figure \ref{fig:workflow}}\DIFdelend \DIFaddbegin \DIFadd{Fig. 1}\DIFaddend ).
The matrix format speeds up extraction of pairwise taxon ages of any queried taxa, as opposed to searching the ancestor node of a pair of taxa in a \enquote{phylo} object or newick string.
Finally, the patristic matrices are associated to the study citation where the original chronogram was published, and stored as an R object of the newly defined class \texttt{datelifeResult}.

\begin{center}
\emph{Summarizing Search Results}
\end{center}

Summary information is extracted from the \texttt{datelifeResult} object to inform decisions for subsequent steps in the analysis workflow. Basic summary information available to the user is:

\begin{enumerate}
\def\labelenumi{\arabic{enumi}.}
\tightlist
\item
  The matching pruned chronograms as newick strings or \enquote{phylo} objects.
\item
  The ages of the root of all source chronograms. These ages can correspond to the age of the most recent common ancestor (mrca) of the user's group of interest if the source chronograms have all taxa belonging to the group. If not, the root corresponds to the mrca of a subgroup \DIFdelbegin \DIFdel{withing }\DIFdelend \DIFaddbegin \DIFadd{within }\DIFaddend the group of interest.
\item
  Study citations where original chronograms were published.
\item
  A report of input taxon names matches across source chronograms.
\item
  The source chronogram(s) with the most input taxon names.
\item
  Various single summary chronograms resulting from summarizing age data, generated using the methodology described next.
\end{enumerate}

\begin{center}
\emph{Choosing a Topology}
\end{center}

DateLife requires a tree topology to summarize age data upon.
We recommend that users provide as input a tree topology from the literature, or one of their own making. If no topology is provided, DateLife automatically extracts one from the OpenTree synthetic tree, a phylogeny \DIFaddbegin \DIFadd{currently }\DIFaddend encompassing 2.3 million taxa across all life, assembled from 1, 239 published phylogenetic trees and OpenTree's unified Taxonomy, OTT (Open Tree Of Life et al., 2019).
Alternatively, DateLife can combine topologies from source chronograms using a supertree approach \DIFdelbegin \DIFdel{.
To combine topologies from source chronograms into a single summary (or supertree)topology, the DateLife workflow }\DIFdelend \DIFaddbegin \DIFadd{(Criscuolo, Berry, Douzery, \& Gascuel, 2006).
To do this, DateLife first }\DIFaddend identifies the source chronograms that form a grove, roughly, a sufficiently overlapping set of taxa between trees, by implementing definition 2.8 for n-overlap from An칠 et al. (2009).
\DIFaddbegin \DIFadd{If the source chronograms do not form a grove, the supertree reconstruction will fail.
}\DIFaddend In rare cases, a group of trees can have multiple groves. By default, DateLife chooses the grove with the most taxa, however, the \enquote{criterion = trees} flag allows the user to choose the grove with the most trees instead.
\DIFdelbegin \DIFdel{If source chronograms do not form a grove, the supertree reconstruction will fail.
}\DIFdelend \DIFaddbegin \DIFadd{The result is a single summary (or supertree) topology, that combines topologies from source chronograms in a grove.
}\DIFaddend 

\begin{center}
\DIFdelbegin \emph{\DIFdel{Dating the Topology}}
%DIFAUXCMD
\DIFdelend \DIFaddbegin \emph{\DIFadd{Applying Secondary Calibrations}}
\DIFaddend \end{center}

\DIFdelbegin \DIFdel{Input topologies from OpenTree or the supertreeapproach described above do not include branch length estimates of any kind.
Optionally, to estimate branch lengths proportional to substitution rates on these topologies, DateLife can mine the Barcode of Life Data System, BOLD (Ratnasingham \& Hebert, 2007) to obtain genetic markers for the input taxa.
These markers are aligned with MUSCLE (Edgar, 2004) (by default) or MAFFT (Katoh, Asimenos, \& Toh, 2009).
This alignment can be used to estimate branch lengths on input topologies that lack branch lengths.
Currently, branch length reconstruction in DateLife is performed using parsimony and the likelihood of the phylogenetic tree given a sequence alignment is computed (Schliep, 2011).
While relative branch length information provides additional data for nodes without secondary date calibrations, topologies without branch lengths can also be dated.
}%DIFDELCMD < 

%DIFDELCMD < %%%
\DIFdelend Once a topology is chosen, DateLife applies the congruification method (Eastman, Harmon, \& Tank, 2013) \DIFdelbegin \DIFdel{to }\DIFdelend \DIFaddbegin \DIFadd{that }\DIFaddend find nodes belonging to the same clade across source chronograms, and \DIFdelbegin \DIFdel{extract }\DIFdelend \DIFaddbegin \DIFadd{then extracts }\DIFaddend the corresponding node ages from \DIFdelbegin \DIFdel{the }\DIFdelend patristic distance matrices stored as \DIFaddbegin \DIFadd{a }\DIFaddend \texttt{datelifeResult} \DIFdelbegin \DIFdel{. By definition, the }\DIFdelend \DIFaddbegin \DIFadd{object. Note that by definition, these }\DIFaddend matrices store total distance (time from tip to tip), \DIFdelbegin \DIFdel{hence}\DIFdelend \DIFaddbegin \DIFadd{assuming that the terminal taxa are coeval and occur at the present. Hence}\DIFaddend , node ages correspond to half the values stored in the \DIFdelbegin \DIFdel{patristic distance matrices.
This assumes that the terminal taxa are coeval and occur at the present.
}\DIFdelend \DIFaddbegin \texttt{\DIFadd{datelifeResult}} \DIFadd{matrices.
}\DIFaddend A table of congruified node ages that can be used as calibrations for a dating analysis is stored as a \texttt{congruifiedCalibrations} object.

For each congruent node, the pairwise distances that traverse that node are summarized into a single summary matrix using classic summary statistics (i.e., mean, median, minimum and maximum ages), and the Supermatrix Distance Method (SDM; Criscuolo \DIFdelbegin \DIFdel{, Berry, Douzery, \& Gascuel, }\DIFdelend \DIFaddbegin \DIFadd{et al., }\DIFaddend 2006), which deforms patristic distance matrices by minimizing variance and then averaging them.
These single summary taxon pair age matrices \DIFdelbegin \DIFdel{(Summarized calibrations ) can be applied as }\DIFdelend \DIFaddbegin \DIFadd{are stored as summarized calibrations that can be used as secondary }\DIFaddend calibrations to date a tree topology \DIFdelbegin \DIFdel{, using different }\DIFdelend \DIFaddbegin \DIFadd{- with or without initial branch lengths, using phylogenetic }\DIFaddend dating methods currently supported within DateLife: \DIFaddbegin \DIFadd{BLADJ (Webb, Ackerly, \& Kembel, 2008; Webb \& Donoghue, 2005), }\DIFaddend MrBayes (Huelsenbeck \& Ronquist, 2001; Ronquist \& Huelsenbeck, 2003), PATHd8 (Britton, Anderson, Jacquet, Lundqvist, \& Bremer, 2007), \DIFdelbegin \DIFdel{BLADJ (Webb, Ackerly, \& Kembel, 2008; Webb \& Donoghue, 2005), }\DIFdelend and treePL (Smith \& O'Meara, 2012).

\DIFdelbegin \DIFdel{By default, DateLife implements the }\DIFdelend \DIFaddbegin \begin{center}
\emph{\DIFadd{Dating a Tree Topology}}
\end{center}

\textbf{\emph{\DIFadd{Dating a tree with no branch lengths.--}}}
\DIFadd{When producing or obtaining a tree with branch lengths for a group of interest is not possible, DateLife can date a topology without branch lengths, obtained from OpenTree or by implementing the supertree approach described above, by applying the }\DIFaddend Branch Length Adjuster (BLADJ\DIFdelbegin \DIFdel{) algorithm to obtain a fully dated topology.
BLADJ fixes node ages that have calibration data , and distributes time between }\DIFdelend \DIFaddbegin \DIFadd{; Webb et al., 2008; Webb \& Donoghue, 2005) algorithm, which requires no initial branch lengths.
The algorithm starts by fixing ages for nodes with calibration data upon the given topology. Then, it distributes time for }\DIFaddend nodes with no data evenly between \DIFdelbegin \DIFdel{nodes with calibration data.
This minimizes }\DIFdelend \DIFaddbegin \DIFadd{calibrated nodes, minimizing }\DIFaddend age variance in the resulting chronogram\DIFaddbegin \DIFadd{.
This approach has proven useful for ecological analyses that require a phylogenetic time context }\DIFaddend (Webb et al., 2008).
\DIFdelbegin \DIFdel{BLADJ does not use branch lengths even when they are present in the input tree or summarizing topology.
}\DIFdelend When there is conflict \DIFdelbegin \DIFdel{in ages between nodes with calibration data}\DIFdelend \DIFaddbegin \DIFadd{between ages of calibrated nodes}\DIFaddend , BLADJ ignores node ages that are older than the age of a parent node.
\DIFdelbegin \DIFdel{BLADJ }\DIFdelend \DIFaddbegin \DIFadd{The BLADJ algorithm }\DIFaddend requires a root age estimate \DIFaddbegin \DIFadd{to run}\DIFaddend . If there is no information on the age of the root in the chronogram database, users can provide an estimate from the literature. If none is provided, DateLife \DIFdelbegin \DIFdel{assigns an arbitrary age to the root as 10\% older than the oldest age available within the group}\DIFdelend \DIFaddbegin \DIFadd{will not return a dated topology and will provide a warning message along with suggestions on how the user can provide an age for the root so that a fully dated tree can be returned.
}

\DIFadd{In the absence of genetic data, BLADJ is a very agnostic way to assign ages to nodes with no available data, as it does not require any assumptions on the underlying model of branch length distribution. It is however common practice in the literature to use a birth-death model to assign ages to nodes with no genetic data (Jetz, Thomas, Joy, Hartmann, \& Mooers, 2012; Rabosky et al., 2018; Smith \& Brown, 2018). To do so, DateLife implements MrBayes (Huelsenbeck \& Ronquist, 2001; Ronquist \& Huelsenbeck, 2003), using nodes with published age data as calibration priors on a fixed topology, a simple birth-death model with parameters that can be determined by the user, and no genetic data}\DIFaddend .

\DIFdelbegin \DIFdel{Alternative phylogenetic datingoptions supported in DateLife (MrBayes, PATHD8, TreePL) }\DIFdelend \DIFaddbegin \textbf{\emph{\DIFadd{Dating a tree with branch lengths.--}}}
\DIFadd{Relative branch lengths can provide key information for phylogenetic dating, specifically for nodes without any calibration data available.
While using initial branch length data is the golden standard for phylogenetic dating analyses, estimating trees with branch lengths proportional to substitution rates per site requires obtaining primary data, assembling and curating a homology hypothesis, and choosing and implementing a method for phylogenetic inference.
DateLife implements a workflow to streamline this process by applying open data from the Barcode of Life Data System, BOLD (Ratnasingham \& Hebert, 2007) to obtain genetic markers for input taxa.
By default, BOLD genetic sequences are aligned with MUSCLE (Edgar, 2004) using functions from the msa R package (Bodenhofer et al., 2015). Alternatively, sequences can be aligned with MAFFT (Katoh, Asimenos, \& Toh, 2009), using functions from the ape R package (Paradis et al., 2004).
The BOLD sequence alignment is then used to obtain initial branch lengths with the accelerated transformation (ACCTRAN) parsimony algorithm, which resolves ambiguous character optimization by assigning changes along branches of the tree as close to the root as possible (Agnarsson \& Miller, 2008), resulting in older internal nodes as compared to other parsimony algorithms (Forest et al., 2005). The parsimony branch lengths are then optimized using Maximum Likelihood, given the alignment, the topology and a simple Jukes-Cantor model, producing a BOLD tree with branch lengths proportional to expected number of substitutions per site. Both parsimony and ML optimizations are done with functions from the }\texttt{\DIFadd{phangorn}} \DIFadd{package (Schliep, 2011).
Due to the computing load it requires, the BOLD workflow is currently only supported through DateLife's R package. It is not yet available through the web application.
}

\DIFadd{Phylogenetic dating methods supported in DateLife that }\DIFaddend incorporate branch length information from the input topology in combination with the \DIFdelbegin \DIFdel{calibrations.
}\DIFdelend \DIFaddbegin \DIFadd{secondary calibrations include:
}\DIFaddend PATHd8\DIFdelbegin \DIFdel{is }\DIFdelend \DIFaddbegin \DIFadd{, }\DIFaddend a non-clock, rate-smoothing method \DIFaddbegin \DIFadd{to date trees }\DIFaddend (Britton et al., 2007)\DIFdelbegin \DIFdel{to date trees.
}\DIFdelend \DIFaddbegin \DIFadd{;
}\DIFaddend treePL (Smith \& O'Meara, 2012), \DIFdelbegin \DIFdel{is }\DIFdelend a semi-parametric, rate-smoothing, penalized likelihood dating method (Sanderson, 2002)\DIFdelbegin \DIFdel{.
The }\DIFdelend \DIFaddbegin \DIFadd{;
and }\DIFaddend MrBayes (Huelsenbeck \& Ronquist, 2001; Ronquist \& Huelsenbeck, 2003)\DIFdelbegin \DIFdel{approach in DateLife uses the calibrations as priors on node ages}\DIFdelend \DIFaddbegin \DIFadd{, a Bayesian inference program implementing Markov chain Monte Carlo (MCMC) methods to estimate a posterior distribution of model parameters}\DIFaddend .

\begin{center}
\emph{Visualizing Results}
\end{center}

Finally, users can save all source and summary chronograms in formats \DIFdelbegin \DIFdel{that permit reuse and reanalyses (newick and }\DIFdelend \DIFaddbegin \DIFadd{allowing for reuse and reanalysis, such as newick and the }\DIFaddend R \enquote{phylo} format\DIFdelbegin \DIFdel{), as well as visualize and compare results graphically, or }\DIFdelend \DIFaddbegin \DIFadd{. Input and summary chronograms can be visualized and compared graphically, and users can }\DIFaddend construct their own graphs using DateLife's chronogram plot generation functions available from the R package \texttt{datelifeplot} (Sanchez-Reyes \& O'Meara, 2022).

\DIFdelbegin %DIFDELCMD < \newpage
%DIFDELCMD < 

%DIFDELCMD < %%%
\DIFdelend \begin{center}
\textsc{Benchmark}
\end{center}

\DIFaddbegin \DIFadd{R package }\DIFaddend \texttt{datelife} \DIFdelbegin \DIFdel{'s R package }\DIFdelend code speed was tested on an Apple iMac
with one 3.4 GHz Intel Core i5 processor.
We registered variation in computing time of query processing and search through the database relative to number of queried taxon names.
Query processing time increases roughly linearly with number of input taxon names, and
increases considerably if Taxonomic Name Resolution Service (TNRS) is activated.
Up to ten thousand names can be processed and searched in less than 30 minutes with the most time consuming settings.
Once names have been processed as described in methods, a name search through the chronogram database can be performed in less than a minute, even with a very large number of taxon names (Fig. \DIFdelbegin \DIFdel{\ref{fig:runtime_main}}\DIFdelend \DIFaddbegin \DIFadd{2}\DIFaddend ).

\texttt{datelife}'s code performance was evaluated with a set of unit tests designed and
implemented with the R package testthat (R Core Team, 2018) that were run both locally
with the devtools package (R Core Team, 2018), and on a public server using the continuous integration tool of GitHub actions (\url{https://docs.github.com/en/actions}).
At present, unit tests cover more than 40\% of \texttt{datelife}'s code (\url{https://codecov.io/gh/phylotastic/datelife}).
Unit testing helps identify potential issues as code is updated or, more critically, as services code relies upon may change.

\begin{center}
\textsc{Case Studies}
\end{center}

We illustrate the DateLife workflow using a family within the \DIFdelbegin \DIFdel{Passeriform }\DIFdelend \DIFaddbegin \DIFadd{passeriform }\DIFaddend birds encompassing the true finches, Fringillidae, as case study. On a small example, we analysed 6 bird species, and results from each step of the workflow are shown in \DIFdelbegin \DIFdel{Fig. \ref{fig:figure2}. }\DIFdelend \DIFaddbegin \DIFadd{Figure 3. }\DIFaddend As a second example, we analysed 289 bird species in the family Fringillidae that are included in the NCBI taxonomy. The resulting summary chronogram is shown in \DIFdelbegin \DIFdel{Fig. \ref{fig:fringillidages}}\DIFdelend \DIFaddbegin \DIFadd{Figure 4}\DIFaddend , and results from previous steps of the workflow are available as Supplementary Figures.

\begin{center}
\emph{A Small Example}
\end{center}

\textbf{\emph{Creating a search query.--}}
We chose 6 bird species within the Passeriformes. The sample includes
two species of cardinals:
the black-thighed grosbeak -- \emph{Pheucticus tibialis} and
the crimson-collared grosbeak -- \emph{Rhodothraupis celaeno};
three species of buntings:
the yellowhammer -- \emph{Emberiza citrinella},
the pine bunting -- \emph{Emberiza leucocephalos} and
the yellow-throated bunting -- \emph{Emberiza elegans};
and one species of tanager, the vegetarian finch -- \emph{Platyspiza crassirostris}.
Processing of input names found that \emph{Emberiza elegans} is synonym for \emph{Schoeniclus elegans} in the default reference taxonomy (OTT v3.3, June 1, 2021). For a detailed discussion on the state of the synonym, refer to Avibase (Avibase, 2022; Lepage, 2004; Lepage, Vaidya, \& Guralnick, 2014).
Discovering this synonym allowed assigning five age data points for the parent node of \emph{Emberiza elegans}, shown as \emph{Schoeniclus elegans} in \DIFdelbegin \DIFdel{figure \ref{fig:figure2}A}\DIFdelend \DIFaddbegin \DIFadd{Figure 3A}\DIFaddend , which would not have had any data otherwise.

\DIFdelbegin %DIFDELCMD < \newpage
%DIFDELCMD < 

%DIFDELCMD < %%%
\DIFdelend \textbf{\emph{Searching the database.--}}
DateLife used the processed input names to search the local chronogram database and found 9 matching chronograms \DIFdelbegin \DIFdel{in }\DIFdelend \DIFaddbegin \DIFadd{from }\DIFaddend 6 different studies (Fig. \DIFdelbegin \DIFdel{\ref{fig:figure2}B}\DIFdelend \DIFaddbegin \DIFadd{3B}\DIFaddend ). Three studies matched five input names (Barker, Burns, Klicka, Lanyon, \& Lovette, 2015; Hedges, Marin, Suleski, Paymer, \& Kumar, 2015; Jetz \DIFdelbegin \DIFdel{, Thomas, Joy, Hartmann, \& Mooers, }\DIFdelend \DIFaddbegin \DIFadd{et al., }\DIFaddend 2012), one study matched four input names (Hooper \& Price, 2017) and two studies matched two input names (Barker, Burns, Klicka, Lanyon, \& Lovette, 2013; Burns et al., 2014). No studies matched all input names. Together, source chronograms provide 28 unique age data points, covering all nodes on our chosen tree topology to date (Table \DIFdelbegin \DIFdel{\ref{tbl:table1}}\DIFdelend \DIFaddbegin \DIFadd{1}\DIFaddend ).

\DIFdelbegin %DIFDELCMD < \vspace{3mm}
%DIFDELCMD < 

%DIFDELCMD < %%%
\DIFdelend \textbf{\emph{Summarizing search results.--}}
DateLife obtained OpenTree's synthetic tree topology for these taxa (Fig. \DIFdelbegin \DIFdel{\ref{fig:figure2}C}\DIFdelend \DIFaddbegin \DIFadd{3C}\DIFaddend ), and congruified and mapped age data to nodes in this chosen topology (Table \DIFdelbegin \DIFdel{\ref{tbl:table1}}\DIFdelend \DIFaddbegin \DIFadd{1}\DIFaddend ).
The name processing step allowed including five data points for node \enquote{n4} (parent of \emph{Schoeniclus elegans}; Fig. \DIFdelbegin \DIFdel{\ref{fig:figure2}A}\DIFdelend \DIFaddbegin \DIFadd{3A}\DIFaddend ) that would not have had any data otherwise due to name mismatch.
Age summary statistics per node were calculated (Table \DIFdelbegin \DIFdel{\ref{tbl:table2}}\DIFdelend \DIFaddbegin \DIFadd{2}\DIFaddend ) and used as calibrations to date the tree topology using the BLADJ algorithm.
As expected, more inclusive nodes (e.g., node \enquote{n1}) have more variance in age data than less inclusive nodes (e.g., node \enquote{n5}).
Summary age data for node \enquote{n2} were excluded as final calibration because they are older than age data of the more inclusive node, \enquote{n1} (Fig. \DIFdelbegin \DIFdel{\ref{fig:figure2}C4}\DIFdelend \DIFaddbegin \DIFadd{3C4}\DIFaddend ).

\newpage

\DIFdelbegin %DIFDELCMD < \newpage
%DIFDELCMD < 

%DIFDELCMD < %%%
\DIFdelend \begin{center}
\emph{An Example with the Family of True Finches}
\end{center}

\textbf{\emph{Creating a query.--}}
To obtain ages for all species within the family of true finches, Fringillidae, we ran a DateLife query using the \enquote{get species from taxon} flag,
which gets all recognized species names within a named group from a taxonomy of choice.
Following the NCBI taxonomy, our DateLife query has 289 Fringillidae species \DIFaddbegin \DIFadd{names}\DIFaddend .
This taxon-constrained approach implies that the \DIFdelbegin \DIFdel{final results of a }\DIFdelend full DateLife analysis will be \DIFdelbegin \DIFdel{done }\DIFdelend \DIFaddbegin \DIFadd{performed }\DIFaddend using a tree topology and ages \DIFdelbegin \DIFdel{for the species in a named }\DIFdelend \DIFaddbegin \DIFadd{available for species names from a given taxonomic }\DIFaddend group, which do not necessarily correspond to a monophyletic group. Users can change this \DIFdelbegin \DIFdel{behaviour by providing a monophyletic tree }\DIFdelend \DIFaddbegin \DIFadd{behavior by providing all species names corresponding to a monophyletic group }\DIFaddend as input for a DateLife search, or \DIFdelbegin \DIFdel{as a tree topology for }\DIFdelend a \DIFaddbegin \DIFadd{monophyletic tree to construct a }\DIFaddend DateLife summary.

\textbf{\emph{Searching the database.--}}
Next, we used the processed species names in our DateLife query to identify chronograms with at least two Fringillidae species \DIFaddbegin \DIFadd{as tip taxa}\DIFaddend .
The DateLife search identified \DIFdelbegin \DIFdel{13 chronograms containing at least two Fringillidae species}\DIFdelend \DIFaddbegin \DIFadd{19 chronograms matching this criteria}\DIFaddend , published in \DIFdelbegin \DIFdel{9 }\DIFdelend \DIFaddbegin \DIFadd{13 }\DIFaddend different studies (Barker et al., 2013, 2015; Burns et al., 2014; Claramunt \& Cracraft, 2015; Gibb et al., 2015; Hedges et al., 2015; Hooper \& Price, 2017; Jetz et al., 2012; \DIFaddbegin \DIFadd{Kimball et al., 2019; Oliveros et al., 2019; }\DIFaddend Price et al., 2014\DIFaddbegin \DIFadd{; Roquet, Lavergne, \& Thuiller, 2014; Uyeda, Pennell, Miller, Maia, \& McClain, 2017}\DIFaddend ).
Once identified, DateLife pruned \DIFaddbegin \DIFadd{these }\DIFaddend matching chronograms to \DIFdelbegin \DIFdel{keep Fringillidae species names on tips only}\DIFdelend \DIFaddbegin \DIFadd{remove tips that do not belong to the queried taxon names}\DIFaddend , and transformed these pruned chronograms to pairwise distance matrices, revealing 1, 206 different age data points available for species within the Fringillidae (\DIFdelbegin \DIFdel{Supplementray }\DIFdelend \DIFaddbegin \DIFadd{Supplementary }\DIFaddend Table S1).

\textbf{\emph{Summarizing search results.--}}
The final step \DIFdelbegin \DIFdel{is to congruify and summarize }\DIFdelend \DIFaddbegin \DIFadd{entailed congruifying and summarizing }\DIFaddend the age data available for the Fringillidae species into \DIFaddbegin \DIFadd{two }\DIFaddend single summary chronograms, using \DIFaddbegin \DIFadd{two }\DIFaddend different types of summary ages, median and SDM.
As explained in the \enquote{Description} section, a tree topology to summarize age data upon is required.
By default, DateLife uses the topology from OpenTree's synthetic tree that contains \DIFdelbegin \DIFdel{the species in the search queryto summarize age data upon}\DIFdelend \DIFaddbegin \DIFadd{all taxa from the search query}\DIFaddend .
According to OpenTree's synthetic tree, species belonging to the family Fringillidae do not form a monophyletic group (Fig. \DIFdelbegin \DIFdel{\ref{fig:fringillidae-topologies}). }\DIFdelend \DIFaddbegin \DIFadd{5). Hence, a topology containing only the 289 species from the original query was extracted from Open Tree of Life's synthetic tree v12.3 (Open Tree Of Life et al., 2019).
}\DIFaddend 

\DIFdelbegin %DIFDELCMD < \vspace{50mm}
%DIFDELCMD < 

%DIFDELCMD < %%%
\DIFdelend Age data from source chronograms was congruified to OpenTree's topology (Fig. \DIFdelbegin \DIFdel{\ref{fig:fringillidae-topologies}B}\DIFdelend \DIFaddbegin \DIFadd{5B}\DIFaddend ), reducing the age data set to 818 different data points (\DIFdelbegin \DIFdel{Supplementray }\DIFdelend \DIFaddbegin \DIFadd{Supplementary }\DIFaddend Table S2). For each congruent node, age summary statistics were calculated and used as fixed secondary calibrations over the chosen tree topology, to obtain a fully dated phylogeny with the program BLADJ (Fig. \DIFdelbegin \DIFdel{\ref{fig:fringillidages}}\DIFdelend \DIFaddbegin \DIFadd{4}\DIFaddend ).

\begin{center}
\textsc{Cross-Validation Test}
\end{center}

We performed a cross validation \DIFdelbegin \DIFdel{analysis of the DateLife workflow }\DIFdelend \DIFaddbegin \DIFadd{test of a DateLife analysis }\DIFaddend using the Fringillidae \DIFdelbegin \DIFdel{chronograms.
We used the }\DIFdelend \DIFaddbegin \DIFadd{source chronograms obtained above (Supplementary Figs. S2-S20).
We used as inputs for a DateLife analysis all }\DIFaddend individual tree topologies from each of the 19 source chronograms from 13 studies\DIFdelbegin \DIFdel{as inputs}\DIFdelend , treating their node ages as unknown.
We \DIFdelbegin \DIFdel{then estimated dates for these topologies using the node ages }\DIFdelend \DIFaddbegin \DIFadd{congruified node ages extracted from chronograms }\DIFaddend from \DIFdelbegin \DIFdel{the chronograms from the other studies as calibrations and smoothing using BLADJ }\DIFdelend \DIFaddbegin \DIFadd{all other studies upon the individual topologies, effectively excluding original ages from each topology. Finally, average node ages per node were applied as secondary calibrations and smoothed with the BLADJ algorithm}\DIFaddend .
We found that node ages from \DIFdelbegin \DIFdel{original study}\DIFdelend \DIFaddbegin \DIFadd{the original studies}\DIFaddend , and ages estimated using all other age data available are \DIFaddbegin \DIFadd{largely }\DIFaddend correlated (Fig. \DIFdelbegin \DIFdel{\ref{fig:cvXY}}\DIFdelend \DIFaddbegin \DIFadd{5}\DIFaddend ).
For five studies, \DIFdelbegin \DIFdel{Datelife }\DIFdelend \DIFaddbegin \DIFadd{DateLife }\DIFaddend tended to underestimate ages for topologically deeper nodes (those with many descendant taxa, aka \enquote{closer to the root}) relative to the original estimate, and overestimate ages for nodes closer to the tips. Accordingly, root ages are generally older in the original study than estimated using cross-validated ages (Supplementary Fig. S1).
\DIFaddbegin \DIFadd{In general, topologically deeper nodes display the largest age variation between node ages from the original chronograms and ages summarized with DateLife.
}\DIFaddend 

\begin{center}
\textsc{Discussion}
\end{center}

DateLife\DIFdelbegin \DIFdel{makes }\DIFdelend \DIFaddbegin \DIFadd{'s goal is to improve availability and accessibility of }\DIFaddend state-of-the-art data on evolutionary time frame \DIFdelbegin \DIFdel{easily accessible for comparison, reuse, and reanalysis, to researchers in }\DIFdelend \DIFaddbegin \DIFadd{of organisms, to allow users from }\DIFaddend all areas of science and with all levels of expertise \DIFdelbegin \DIFdel{in the matter. It is }\DIFdelend \DIFaddbegin \DIFadd{to compare, reuse, and reanalyse expert age data for their own applications. As such, it is designed as }\DIFaddend an open service that does not require any expert biological knowledge \DIFdelbegin \DIFdel{from users }\DIFdelend --besides the \DIFaddbegin \DIFadd{scientific }\DIFaddend names of the species or group \DIFdelbegin \DIFdel{they }\DIFdelend \DIFaddbegin \DIFadd{users }\DIFaddend want to work with, \DIFdelbegin \DIFdel{for }\DIFdelend \DIFaddbegin \DIFadd{to use }\DIFaddend any of its \DIFdelbegin \DIFdel{functionality}\DIFdelend \DIFaddbegin \DIFadd{functionalities}\DIFaddend .

A total of 99,474 unique terminal taxa are represented in DateLife's database.
Incorporation of more chronograms into the database will continue to improve DateLife's services. One option to increase the number of chronograms in the DateLife database is the Dryad data repository. Methods to automatically mine chronograms from Dryad could be designed and implemented. However, Dryad's metadata system has no information to automatically detect branch length units, and those would still need to be determined manually by a human curator.
We would like to emphasize on the importance of sharing chronogram data, including systematically curated metadata, into open repositories, such as OpenTree's Phylesystem (McTavish et al., 2015) for the benefit of the scientific community as a whole.

\DIFaddbegin \DIFadd{As we envision that DateLife will have many interesting applications in research and beyond, we emphasize that DateLife's results --as well as any insights gleaned from them, largely depend on the quality of the source chronograms: low quality chronograms will produce low quality results. This }\enquote{garbage in, garbage out} \DIFadd{problem has long been recognised in supertree methods for summarizing phylogenetic trees (Bininda-Emonds et al., 2004).
We note that this is a surfacing issue of any automated tool for biological data analysis. For example, DNA riddled with sequencing errors will produce generally poor alignments that will return biased evolutionary hypothesis, independently of the quality of the analysis software used. Again, we urge readers and DateLife users to explore all input chronograms before using a summary chronogram resulting from a DateLife workflow.
Finally, uncertainty and variability of chronogram node age estimates might pose larger issues in some research areas than others. For example, in ecological and conservation biology studies, it has been shown that incorporating some chronogram data provides better results than when not using any age data at all, even if the node ages are not good quality (Webb et al., 2008). In the following sections we discuss the particularities of divergence times from DateLife's summary chronograms and their impact on certain evolutionary analyses, for consideration of the readers and users in different research areas.
}

\DIFaddend \begin{center}
\emph{Age Variation in Source Chronograms}
\end{center}

Conflict in estimated ages among alternative studies is common in the literature. See, for example, the robust ongoing debate about crown group age of angiosperms (Barba-Montoya, Reis, Schneider, Donoghue, \& Yang, 2018; Magall칩n, G칩mez-Acevedo, S치nchez-Reyes, \& Hern치ndez-Hern치ndez, 2015; Ramshaw et al., 1972; Sanderson \& Doyle, 2001; Sauquet, Ram칤rez-Barahona, \& Magall칩n, 2021).
\DIFdelbegin \DIFdel{Source }\DIFdelend \DIFaddbegin \DIFadd{Alternative source }\DIFaddend chronograms available for the same \DIFdelbegin \DIFdel{organisms }\DIFdelend \DIFaddbegin \DIFadd{taxa }\DIFaddend have potentially been estimated implementing \DIFdelbegin \DIFdel{calibrations very differently}\DIFdelend \DIFaddbegin \DIFadd{different types of calibrations, which affects the resulting node age estimates}\DIFaddend .
For example, \DIFdelbegin \DIFdel{the chronograms from }\DIFdelend \DIFaddbegin \DIFadd{in the DateLife analysis of the Fringillidae shown above, the chronograms from one study (}\DIFaddend Burns et al.\DIFdelbegin \DIFdel{(}\DIFdelend \DIFaddbegin \DIFadd{, }\DIFaddend 2014) were inferred using molecular substitution rate estimates across birds (Weir \& Schluter, 2008), and have much older age estimates for the same nodes than chronograms that were inferred using \DIFdelbegin \DIFdel{fossils as }\DIFdelend \DIFaddbegin \DIFadd{fossil }\DIFaddend calibrations (Figs. \DIFdelbegin \DIFdel{\ref{fig:fringillidages}, \ref{fig:cvXY}}\DIFdelend \DIFaddbegin \DIFadd{4, 5}\DIFaddend ; Supplementary Figs. S1, S5).
\DIFdelbegin %DIFDELCMD < 

%DIFDELCMD < %%%
\DIFdel{Different calibrationimplementations might also }\DIFdelend \DIFaddbegin \DIFadd{Another source of conflict in estimated node ages can arise from different placements for the same calibration, which would }\DIFaddend imply fundamentally distinct evolutionary hypotheses (Antonelli et al., 2017).
For example, two independent researchers working on the same clade should both carefully select and justify their choices of fossil calibration placement.
Yet, if one researcher concludes that a fossil should calibrate the ingroup of a clade, while another researcher concludes that the same fossil should calibrate the outgroup of the clade, the resulting age estimates will differ, as the placement of calibrations as stem or crown group \DIFdelbegin \DIFdel{has been proven }\DIFdelend \DIFaddbegin \DIFadd{is known }\DIFaddend to significantly affect \DIFaddbegin \DIFadd{estimates of }\DIFaddend time of lineage divergence \DIFdelbegin \DIFdel{estimates }\DIFdelend (Sauquet, 2013).
\DIFaddbegin \DIFadd{Finally, placement of calibrations also affects uncertainty of node age estimates. For example, nodes that are sandwiched between a calibrated node and a calibrated root have less freedom of movement and hence narrower confidence intervals (Vos \& Mooers, 2004), which inflates precision for nodes without calibrations but does not necessarily improve accuracy of the estimated ages.
}\DIFaddend 

\DIFaddbegin \DIFadd{DateLife's summary chronograms are intended to represent all variation in estimated node ages from source chronograms. Node age distribution ranges allow to visually explore ages from source chronograms individually and contextualize and compare them against other chronograms. Researchers that wish to use summary chronograms in downstream evolutionary analysis may select multiple trees sampled from the summary distribution of node ages, to account for variation in source chronograms.
}

\DIFaddend \begin{center}
\emph{Primary vs Secondary Calibrations}
\end{center}

While most chronograms in DateLife's database are constructed using primary calibrations\DIFdelbegin \DIFdel{(}\DIFdelend \DIFaddbegin \DIFadd{, i.e., }\DIFaddend molecular substitution rates or \DIFdelbegin \DIFdel{ages }\DIFdelend \DIFaddbegin \DIFadd{absolute ages in Myr }\DIFaddend obtained from the fossil record or geological events\DIFdelbegin \DIFdel{), DateLife summarizes }\DIFdelend \DIFaddbegin \DIFadd{, DateLife also constructs summary }\DIFaddend chronograms using secondary calibrations\DIFdelbegin \DIFdel{(ages coming from other chronograms).Graur and Martin (2004) cautioned on the increased error and uncertainty in estimated ages when using secondary calibrations in dating analyses.Schenk (2016) showed that, in simulations, divergence times inferred using secondary calibrations are significantly younger than those inferred with primary calibrations, when obtained with Bayesian inference methods, and when priors are implemented in similar ways in both analyses.
Accordingly, the scientific community seems to have }\DIFdelend \DIFaddbegin \DIFadd{, i.e., node ages in Myr extracted from existing chronograms.
The scientific community has generally }\DIFaddend more confidence in chronograms \DIFdelbegin \DIFdel{obtained }\DIFdelend \DIFaddbegin \DIFadd{generated }\DIFaddend from a single analysis, \DIFdelbegin \DIFdel{using fossil data as primary sources of calibrations (}\DIFdelend \DIFaddbegin \DIFadd{where carefully chosen primary calibrations constitute the main source of absolute time information used in the dating analysis (Antonelli et al., 2017; Garz칩n-Ordu침a, Silva-Brand칚o, Willmott, Freitas, \& Brower, 2015; Graur \& Martin, 2004; Sauquet, 2013; Sauquet et al., 2012; }\DIFaddend Schenk, 2016\DIFdelbegin \DIFdel{)}\DIFdelend \DIFaddbegin \DIFadd{; Shaul \& Graur, 2002).
However, implementation of primary calibrations does not come without difficulties: it requires high expertise and training to discover, place and apply calibrations appropriately (Hipsley \& M칲ller}\DIFaddend , \DIFdelbegin \DIFdel{and using }\DIFdelend \DIFaddbegin \DIFadd{2014; Ksepka et al., 2011). One way around it is to use }\DIFaddend fossils that have been widely discussed and \DIFaddbegin \DIFadd{previously }\DIFaddend curated as calibrations to date other trees \DIFdelbegin \DIFdel{, }\DIFdelend \DIFaddbegin \DIFadd{(Ksepka et al., 2011; Sauquet, 2013), and }\DIFaddend making sure that all data reflect a coherent evolutionary history (Sauquet, 2013), as for example done by Antonelli et al. (2017).
\DIFdelbegin \DIFdel{There have been attempts to create fossil calibration databases }\DIFdelend \DIFaddbegin \DIFadd{The Fossil Calibration Database provides availability of 220 primary calibration points encompassing flowering plants and metazoans, that have been curated by experts and used for dating analysis in peer-reviewd publications }\DIFaddend (Ksepka et al., 2015)\DIFdelbegin \DIFdel{, though these still }\DIFdelend \DIFaddbegin \DIFadd{. This database facilitates the use of expert primary fossil calibrations in new phylogenetic dating analyses.
Yet, users still require the expertise to locate appropriate nodes in their phylogenies to corresponding fossils available in the database, potentially delaying increased usage of the database in research.
Open databases for effective reuse of fossil calibration data still }\DIFaddend have room to grow \DIFaddbegin \DIFadd{and should constitute a key addition to the phylogenetics tool set}\DIFaddend .

\DIFaddbegin \DIFadd{In some cases, secondary calibrations might represent the only information available for a dating analysis (e.g., groups with a poor fossil record). Recently, Powell, Waskin, and Battistuzzi (2020) showed in a simulation analysis that secondary calibrations are as good as primary calibrations. }\DIFaddend It seems that using several \DIFaddbegin \DIFadd{secondary calibrations }\DIFaddend (as opposed to just \DIFdelbegin \DIFdel{a few) secondary calibrations }\DIFdelend \DIFaddbegin \DIFadd{one) }\DIFaddend can provide sufficient information to alleviate or even neutralize potential biases \DIFdelbegin \DIFdel{(}\DIFdelend \DIFaddbegin \DIFadd{described before (Graur \& Martin, 2004; }\DIFaddend Sauquet, 2013\DIFdelbegin \DIFdel{).
Certainly, further studies are required to }\DIFdelend \DIFaddbegin \DIFadd{; Shaul \& Graur, 2002). Our cross validation analysis also provides some insight into the application of secondary calibrations. In general, node ages summarized with DateLife and those from the original studies appear positively correlated (Supplementary Figs. S2-S20). We also note that nodes closer to the root tend to be slightly younger than ages from the original studies. In contrast, nodes closer to the tips tend to be slightly older when estimated using secondary calibrations than ages from the original studies. The only exception to this trend was observed in Burns et al. (2014) chronogram, which generally displays much younger node ages when estimated using secondary calibrations than the original study (Supplementary Figs. S1, S5), supporting previous observations (Sauquet et al., 2012; Schenk, 2016).
}

\DIFadd{We unite to the call of our colleagues: further research is needed to }\DIFaddend fully understand the \DIFdelbegin \DIFdel{effect of secondary calibrations on outputs from different }\DIFdelend \DIFaddbegin \DIFadd{effects of using secondary calibrations with alternative }\DIFaddend tree dating methods, \DIFdelbegin \DIFdel{and on downstream analyses . }\DIFdelend \DIFaddbegin \DIFadd{as well as usage of resulting chronograms in downstream analyses (Hipsley \& M칲ller, 2014; Powell et al., 2020; Schenk, 2016; Shaul \& Graur, 2002). }\DIFaddend It is possible that secondary calibrations can be \DIFdelbegin \DIFdel{safely used }\DIFdelend \DIFaddbegin \DIFadd{used successfully }\DIFaddend with dating methods that do not require setting priors, such as penalized likelihood (Sanderson, 2003)\DIFdelbegin \DIFdel{, }\DIFdelend \DIFaddbegin \DIFadd{; }\DIFaddend with methods that do not make any assumptions on the ages and fix them to a node on a tree topology, such as BLADJ (Webb et al., 2008; Webb \& Donoghue, 2005)\DIFdelbegin \DIFdel{, or }\DIFdelend \DIFaddbegin \DIFadd{; and with }\DIFaddend methods that summarize age data unto a tree topology\DIFdelbegin \DIFdel{.
}%DIFDELCMD < 

%DIFDELCMD < %%%
\DIFdel{Our cross validation analysis might provide some insight in this regard. When ages are estimated with secondary calibrations, nodes closer to the root do tend to be slightly younger than ages estimated with primary calibrations.
However, nodes closer to the tip tend to be older when estimated using secondary calibrations with a dating method that does not make any prior assumptions on the nature of the calibrations themselves (Supplementary Figures S2-S20). The only excpetion to tjis was observed on results of the cross validation analysis of the Burns et al. (2014) chronogram, which results in much younger node ages when estimated using secondary calibrations (Supplementary Figs. S1}\DIFdelend , \DIFdelbegin \DIFdel{S5).
}\DIFdelend \DIFaddbegin \DIFadd{such as DateLife.
}\DIFaddend 

\begin{center}
\emph{Sumarizing Chronograms}
\end{center}

By default, DateLife currently summarizes all source chronograms that overlap with at least two species names. Users can exclude source chronograms if they have reasons to do so.
Strictly speaking, a good chronogram should reflect the real time of lineage divergence accurately and precisely.
To our knowledge, there are no tested measures to determine independently when a chronogram is better than another. Yet, several characteristics of the data used for dating analyses, as well as from the output chronogram itself, could be used to score the quality of source chronograms.

Some measures that have been proposed are the proportion of lineage sampling and the number of calibrations used (Magall칩n, 2010; Magall칩n et al., 2015).
Some characteristics that are often cited in published studies as a measure of improved age estimates as compared to previously published estimates are: quality of alignment (missing data, GC content), lineage sampling (strategy and proportion), phylogenetic and dating inference method, number of fossils used as calibrations, support for nodes and ages, and magnitude of confidence intervals.

DateLife provides an opportunity to capture concordance and conflict among date estimates, which can also be used as a metric for chronogram reliability.
Its open database of chronograms allows other researchers to do such analyses themselves reproducibly, and without needing permission. Though, of course, they should follow proper citation practices, especially for the source chronogram studies.

The exercise of summarizing age data from across multiple studies provides the opportunity to work with a \DIFdelbegin \DIFdel{more inclusive chronogram ,
}\DIFdelend \DIFaddbegin \DIFadd{chronogram }\DIFaddend that reflects a unified evolutionary history for a lineage, by putting together evidence from different hypotheses.
The largest, and taxonomically broadest chronogram currently available from OpenTree was constructed summarizing age data from 2,274 published chronograms using NCBI's taxonomic tree as backbone (Hedges et al., 2015).
A summarizing exercise may also amplify the effect of uncertainty and errors in source data, and blur parts of the evolutionary history of a lineage that might only be reflected in source chronograms and lost on the summary chronogram (Sauquet et al., 2021).

\begin{center}
\emph{Effects \DIFaddbegin \DIFadd{of Taxon Sampling }\DIFaddend on Downstream Analyses}
\end{center}

\DIFdelbegin \DIFdel{For downstream analyses, using alternative chronogram may deeply affect our inferences (Title \& Rabosky, 2016), particularly when studying phenomena dependent on the timing of species diversification events, such as macroevolutionary processes.
}%DIFDELCMD < 

%DIFDELCMD < %%%
\DIFdel{In ecology and conservation biology, incorporating at least some data on lineage divergence times represents a relevant improvement for testing alternative hypothesis using phylogenetic distance (Webb }\DIFdelend \DIFaddbegin \DIFadd{Analysis of species diversification of simulated and empirical phylogenies suggest that using a more completely sampled phylogeny provides estimates that are closer to the true diversification history than when analysing incompletely sampled phylogenies (Chang, Rabosky, \& Alfaro, 2020; Cusimano, Stadler, \& Renner, 2012; Sun }\DIFaddend et al., \DIFdelbegin \DIFdel{2008).
}\DIFdelend \DIFaddbegin \DIFadd{2020).
Ideally, phylogenies should be completed using genetic data, but this is a time-consuming and difficult task to achieve for many biological groups.
}\DIFaddend Hence, DateLife's workflow features different ways of \DIFdelbegin \DIFdel{estimating node ages in }\DIFdelend \DIFaddbegin \DIFadd{assigning divergence times to taxa with missing }\DIFaddend the absence of \DIFaddbegin \DIFadd{branch length data and }\DIFaddend calibrations and branch \DIFdelbegin \DIFdel{length information }\DIFdelend \DIFaddbegin \DIFadd{lengths }\DIFaddend for certain taxa.
\DIFdelbegin %DIFDELCMD < \enquote{Making up} %%%
\DIFdel{branch lengths }\DIFdelend \DIFaddbegin 

\DIFadd{Completing a phylogeny using a stochastic birth-death polytomy resolver and a backbone taxonomy }\DIFaddend is a common practice in scientific publications: Jetz et al. (2012), created a chronogram of all 9, 993 bird species, where 67\% had molecular data and the rest was simulated; Rabosky et al. (2018) created a chronogram of 31, 536 ray-finned fishes, of which only 37\% had molecular data; Smith and Brown (2018) constructed a chronogram of 353, 185 seed plants where only 23\% had molecular data. \DIFaddbegin \DIFadd{These stochastically resolved chronograms return diversification rates estimates that appear less biased than those estimated from their incompletely sampled counterparts, even with methods that account for missing lineages by using sampling fractions (Chang et al., 2020; Cusimano et al., 2012), but can also introduce spurious patterns of early bursts of diversification (Cusimano \& Renner, 2010; Sun et al., 2020).
}\DIFaddend 

\DIFdelbegin \DIFdel{Notably, }\DIFdelend \DIFaddbegin \DIFadd{Taxonomy-based stochastic polytomy resolvers also introduce topological differences in phylogenetic trees.
The study of macroevolutionary processes largely depends on an understanding of the timing of species diversification events, and different phylogenetic and chronogram hypothesis can provide very different overviews of the macroevolutionary history of a biological group.
For example, alternative topologies in chronograms from the same biological group can infer very different species diversification patterns (Rabosky, 2015; Title \& Rabosky, 2016).
Similarly, there are worries that patterns of morphological evolution cannot be accurately inferred with phylogenies that have been resolved stochastically over a taxonomic backbone, as any patterns would be erased by randomization (Rabosky, 2015). We note that the same applies for geography- and morphology-dependent diversification analysis. Hence, we suggest that phylogenies that have been processed with taxonomy-based stochastic polytomy resolvers, including certain summary chronograms from a DateLife analysis, can be useful as null or neutral models, representing the case of a diversification process that is independent of traits and geographical scenario.
}

\DIFadd{Taxonomy-based stochastic polytomy resolvers have been used to advance research in evolution, still, }\DIFaddend risks come with this practice.
Taken to the extreme, one could \DIFdelbegin \DIFdel{make }\DIFdelend \DIFaddbegin \DIFadd{generate }\DIFaddend a fully resolved, calibrated tree of all modern and extinct taxa using a single taxonomy\DIFdelbegin \DIFdel{and }\DIFdelend \DIFaddbegin \DIFadd{, }\DIFaddend a single calibration, \DIFdelbegin \DIFdel{using polytomy resolution and branch estimation methods. There has yet to be a thorough analysis of what can go wrong when one extends inferences beyond the data in this way, so we urge caution; we also urge readers }\DIFdelend \DIFaddbegin \DIFadd{and assigning branch lengths following a birth-death diversification model. Clearly, this can lead to a misrepresentation of the true evolutionary history.
We urge DateLife users }\DIFaddend to follow the example of the large tree papers cited above, by carefully considering the statistical assumptions being made, \DIFaddbegin \DIFadd{potential biases, }\DIFaddend and assessing the consistency of \DIFdelbegin \DIFdel{the }\DIFdelend \DIFaddbegin \DIFadd{DateLife's }\DIFaddend results with prior work.

\begin{center}
\textsc{Conclusions}
\end{center}

Knowledge of the evolutionary time frame of organisms is key to many research areas: trait evolution, species diversification, biogeography, macroecology and more. It is also crucial for education, science communication and policy, but generating chronograms is difficult, especially for those who want to use phylogenies but who are not systematists, or do not have the time to acquire and develop the necessary knowledge and skills to construct them on their own. Importantly, years of primarily \DIFdelbegin \DIFdel{public }\DIFdelend \DIFaddbegin \DIFadd{publicly }\DIFaddend funded research have resulted in vast amounts of chronograms that are already available \DIFdelbegin \DIFdel{on }\DIFdelend \DIFaddbegin \DIFadd{in }\DIFaddend scientific publications, but \DIFdelbegin \DIFdel{hidden to }\DIFdelend \DIFaddbegin \DIFadd{functionally hidden from }\DIFaddend the public and scientific community for reuse.

The DateLife project allows for easy and fast \DIFdelbegin \DIFdel{summary }\DIFdelend \DIFaddbegin \DIFadd{summarization }\DIFaddend of public and state-of-the-art data on time of lineage divergence.
It provides a straightforward way to get an informed idea on the state of knowledge of the time frame of evolution of different regions of the tree of life, and allows \DIFdelbegin \DIFdel{identification of }\DIFdelend \DIFaddbegin \DIFadd{identifying }\DIFaddend regions that require more research, or that have conflicting information.
It is available as an R package, and as a web-based R shiny application at www.datelife.org\DIFaddbegin \DIFadd{.
}\DIFaddend Both summary and newly generated trees are useful to evaluate evolutionary hypotheses in different areas of research. The DateLife project \DIFdelbegin \DIFdel{helps with }\DIFdelend \DIFaddbegin \DIFadd{will improve }\DIFaddend awareness of the existing variation in expert \DIFaddbegin \DIFadd{estimations }\DIFaddend time of divergence data, and \DIFdelbegin \DIFdel{will }\DIFdelend foster exploration of the effect of alternative divergence time \DIFdelbegin \DIFdel{hypothesis }\DIFdelend \DIFaddbegin \DIFadd{hypotheses }\DIFaddend on the results of analyses, nurturing a culture of more cautious interpretation of evolutionary results.

\begin{center}
\textsc{Availability}
\end{center}

The DateLife software is free and open source\DIFdelbegin \DIFdel{and it }\DIFdelend \DIFaddbegin \DIFadd{. It }\DIFaddend can be used \DIFaddbegin \DIFadd{online }\DIFaddend through its R shiny web application \DIFaddbegin \DIFadd{hosted }\DIFaddend at \url{http://www.datelife.org}, \DIFaddbegin \DIFadd{and locally }\DIFaddend through the \texttt{datelife} R package, \DIFdelbegin \DIFdel{and through Phylotastic's project web portal }%DIFDELCMD < \url{https://phylo.cs.nmsu.edu/}%%%
\DIFdel{.}\DIFdelend \DIFaddbegin \DIFadd{available from Zenodo (}\url{https://doi.org/10.5281/zenodo.593938} \DIFadd{and the CRAN repository (Sanchez-Reyes et al., 2022).
}\DIFaddend DateLife's web application is maintained using RStudio's shiny server and the shiny package open infrastructure, as well as Docker and OpenTree's infrastructure (\DIFdelbegin \DIFdel{dates}\DIFdelend \DIFaddbegin \DIFadd{datelife}\DIFaddend .opentreeoflife.org\DIFdelbegin \DIFdel{/datelife}\DIFdelend ).
\texttt{datelife}'s \DIFdelbegin \DIFdel{R package stable version is available for installation }\DIFdelend \DIFaddbegin \DIFadd{stable version can be installed }\DIFaddend from the CRAN repository \DIFdelbegin \DIFdel{(}%DIFDELCMD < \url{https://cran.r-project.org/package=datelife}%%%
\DIFdel{) }\DIFdelend using the command \texttt{install.packages(pkgs\ =\ "datelife")} from within R.
Development versions are available from \DIFdelbegin \DIFdel{the }\DIFdelend \DIFaddbegin \DIFadd{DateLife's }\DIFaddend GitHub repository (\url{https://github.com/phylotastic/datelife}) and can be installed using the command \texttt{devtools::install\_github("phylotastic/datelife")}.

\begin{center}
\textsc{Supplementary Material}
\end{center}

\DIFdelbegin \DIFdel{Code used to generate all versions of this manuscript, the }\DIFdelend \DIFaddbegin \DIFadd{Supplementary Figures can be viewed and dowloaded from their Zenodo repository (}\url{https://doi.org/10.5281/zenodo.6683667}\DIFadd{).
Supplementary material, including code, }\DIFaddend biological examples, \DIFaddbegin \DIFadd{benchmark results, data files and online-only appendices, can be downloaded from the Dryad data repository (}\url{https://doi.org/10.5061/dryad.cnp5hqc6w}\DIFadd{), }\DIFaddend as well as \DIFdelbegin \DIFdel{the benchmark of functionalities are available at }\href{https://github.com/LunaSare/datelifeMS1}{\DIFdel{datelifeMS1}}%DIFAUXCMD
\DIFdel{, }\href{https://github.com/LunaSare/datelife_examples}{\DIFdel{datelife\_examples}}%DIFAUXCMD
\DIFdel{, and }\href{https://github.com/LunaSare/datelife_benchmark}{\DIFdel{datelife\_benchmark}} %DIFAUXCMD
\DIFdel{repositories in LLSR's GitHub account}\DIFdelend \DIFaddbegin \DIFadd{in the Zenodo stable repositories that host the reproducible manuscript (}\url{https://doi.org/10.5281/zenodo.7435094}\DIFadd{), the biological examples (}\url{https://doi.org/10.5281/zenodo.7435101}\DIFadd{), and the software benchmark (}\url{https://doi.org/10.5281/zenodo.7435106}\DIFadd{). Development versions corresponding to all of the above are hosted on GitHub, accesible at }\url{https://github.com/LunaSare/datelifeMS1}\DIFadd{, }\url{https://github.com/LunaSare/datelife_examples}\DIFadd{, and }\url{https://github.com/LunaSare/datelife_benchmark}\DIFaddend .

\begin{center}
\textsc{Funding}
\end{center}

Funding was provided by the US National Science Foundation (NSF) grants ABI-1458603 to the \DIFdelbegin \DIFdel{Datelife }\DIFdelend \DIFaddbegin \DIFadd{DateLife }\DIFaddend project; DBI-0905606 to the National Evolutionary Synthesis Center (NESCent)\DIFdelbegin \DIFdel{, }\DIFdelend \DIFaddbegin \DIFadd{; }\DIFaddend ABI-1458572 to the Phylotastic project\DIFdelbegin \DIFdel{, }\DIFdelend \DIFaddbegin \DIFadd{; }\DIFaddend and ABI-1759846 to the Open Tree of Life project.

\begin{center}
\textsc{Acknowledgements}
\end{center}

\DIFaddbegin \DIFadd{We thank Isabel Sanmart칤n, Daniele Silvestro, Rutger Vos and an anonymous reviewer, for comments that greatly improved this manuscript.
}\DIFaddend The DateLife project was born as a prototype tool aiming to provide \DIFdelbegin \DIFdel{these services }\DIFdelend \DIFaddbegin \DIFadd{the services describe in this paper}\DIFaddend , and was initially developed over a series of hackathons at the National Evolutionary Synthesis Center, NC, USA (Stoltzfus et al., 2013).
We thank colleagues from the O'Meara Lab at the University
of \DIFdelbegin \DIFdel{Tennesse }\DIFdelend \DIFaddbegin \DIFadd{Tennessee }\DIFaddend Knoxville for suggestions, discussions and software testing.
The late National Evolutionary Synthesis Center (NESCent), which sponsored hackathons
that led to initial work on this project. The team that assembled DateLife's first proof of concept: Tracy Heath, Jonathan Eastman, Peter Midford, Joseph Brown, Matt Pennell, Mike Alfaro, and Luke Harmon.
The Open Tree of Life project that provides the open, metadata rich repository of
trees used to construct DateLife's chronogram database.
The many scientists who publish their chronograms in an open, reusable form, and the scientists who curate them for deposition in the Open Tree of Life repository.
The NSF for funding nearly all the above, in addition to the ABI grant that funded this project itself.

\newpage

\hypertarget{references}{%
\section{References}\label{references}}

\begingroup
\setlength{\parindent}{-0.5in}
\setlength{\leftskip}{0.5in}

\hypertarget{refs}{}
\leavevmode\DIFaddbegin \hypertarget{ref-agnarsson2008acctran}{}%DIF > 
\DIFadd{Agnarsson, I., \& Miller, J. A. (2008). Is ACCTRAN better than DELTRAN? }\emph{\DIFadd{Cladistics}}\DIFadd{, }\emph{24}(6)\DIFadd{, 1032--1038.
}

\leavevmode\DIFaddend \hypertarget{ref-alstrom2014discovery}{}%
Alstr칬m, P., Hooper, D. M., Liu, Y., Olsson, U., Mohan, D., Gelang, M., \ldots{} Price, T. D. (2014). Discovery of a relict lineage and monotypic family of passerine birds. \emph{Biology Letters}, \emph{10}(3), 20131067.

\leavevmode\hypertarget{ref-ane2009groves}{}%
An칠, C., Eulenstein, O., Piaggio-Talice, R., \& Sanderson, M. J. (2009). Groves of phylogenetic trees. \emph{Annals of Combinatorics}, \emph{13}(2), 139--167.

\leavevmode\hypertarget{ref-antonelli2017supersmart}{}%
Antonelli, A., Hettling, H., Condamine, F. L., Vos, K., Nilsson, R. H., Sanderson, M. J., \ldots{} Vos, R. A. (2017). Toward a self-updating platform for estimating rates of speciation and migration, ages, and relationships of Taxa. \emph{Systematic Biology}, \emph{66}(2), 153--166. \url{https://doi.org/10.1093/sysbio/syw066}

\leavevmode\hypertarget{ref-archie1986newick}{}%
Archie, J., Day, W. H., Felsenstein, J., Maddison, W., Meacham, C., Rohlf, F. J., \& Swofford, D. (1986). The Newick tree format. Retrieved from \href{\%7Bhttps://evolution.genetics.washington.edu/phylip/newicktree.html\%7D}{\{https://evolution.genetics.washington.edu/phylip/newicktree.html\}}

\leavevmode\hypertarget{ref-avibase-emberiza}{}%
Avibase. (2022). Yellow-throated Bunting. \emph{Avibase - the World Bird Database}, (Online Resource). Retrieved from \href{\%7Bhttps://avibase.bsc-eoc.org/species.jsp?lang=EN\&avibaseid=82D1EE0049D8D927\%7D}{\{https://avibase.bsc-eoc.org/species.jsp?lang=EN\&avibaseid=82D1EE0049D8D927\}}

\leavevmode\hypertarget{ref-Bapst2012a}{}%
Bapst, D. W. (2012). Paleotree: An R package for paleontological and phylogenetic analyses of evolution. \emph{Methods in Ecology and Evolution}, \emph{3}(5), 803--807. \url{https://doi.org/10.1111/j.2041-210X.2012.00223.x}

\leavevmode\hypertarget{ref-barba2018constraining}{}%
Barba-Montoya, J., Reis, M. dos, Schneider, H., Donoghue, P. C., \& Yang, Z. (2018). Constraining uncertainty in the timescale of angiosperm evolution and the veracity of a cretaceous terrestrial revolution. \emph{New Phytologist}, \emph{218}(2), 819--834.

\leavevmode\hypertarget{ref-barker2014mitogenomic}{}%
Barker, F. K. (2014). Mitogenomic data resolve basal relationships among passeriform and passeridan birds. \emph{Molecular Phylogenetics and Evolution}, \emph{79}, 313--324.

\leavevmode\hypertarget{ref-barker2013going}{}%
Barker, F. K., Burns, K. J., Klicka, J., Lanyon, S. M., \& Lovette, I. J. (2013). Going to extremes: Contrasting rates of diversification in a recent radiation of new world passerine birds. \emph{Systematic Biology}, \emph{62}(2), 298--320.

\leavevmode\hypertarget{ref-barker2015new}{}%
Barker, F. K., Burns, K. J., Klicka, J., Lanyon, S. M., \& Lovette, I. J. (2015). New insights into new world biogeography: An integrated view from the phylogeny of blackbirds, cardinals, sparrows, tanagers, warblers, and allies. \emph{The Auk: Ornithological Advances}, \emph{132}(2), 333--348.

\leavevmode\hypertarget{ref-barker2004phylogeny}{}%
Barker, F. K., Cibois, A., Schikler, P., Feinstein, J., \& Cracraft, J. (2004). Phylogeny and diversification of the largest avian radiation. \emph{Proceedings of the National Academy of Sciences}, \emph{101}(30), 11040--11045.

\leavevmode\hypertarget{ref-beresford2005african}{}%
Beresford, P., Barker, F., Ryan, P., \& Crowe, T. (2005). African endemics span the tree of songbirds (passeri): Molecular systematics of several evolutionary ``enigmas''. \emph{Proceedings of the Royal Society B: Biological Sciences}, \emph{272}(1565), 849--858.

\leavevmode\DIFaddbegin \hypertarget{ref-bininda2004garbage}{}%DIF > 
\DIFadd{Bininda-Emonds, O. R., Jones, K. E., Price, S. A., Cardillo, M., Grenyer, R., \& Purvis, A. (2004). Garbage in, garbage out: Data issues in supertree construction. }\emph{\DIFadd{Phylogenetic Supertrees: Combining Information to Reveal the Tree of Life}}\DIFadd{, 267--280.
}

\leavevmode\hypertarget{ref-bodenhofer2015msa}{}%DIF > 
\DIFadd{Bodenhofer, U., Bonatesta, E., Horej코-Kainrath, C., \& Hochreiter, S. (2015). Msa: An r package for multiple sequence alignment. }\emph{\DIFadd{Bioinformatics}}\DIFadd{, }\emph{31}(24)\DIFadd{, 3997--3999.
}

\leavevmode\DIFaddend \hypertarget{ref-Boyle2013}{}%
Boyle, B., Hopkins, N., Lu, Z., Raygoza Garay, J. A., Mozzherin, D., Rees, T., \ldots{} Enquist, B. J. (2013). The taxonomic name resolution service: An online tool for automated standardization of plant names. \emph{BMC Bioinformatics}, \emph{14}(1). \url{https://doi.org/10.1186/1471-2105-14-16}

\leavevmode\hypertarget{ref-Britton2007}{}%
Britton, T., Anderson, C. L., Jacquet, D., Lundqvist, S., \& Bremer, K. (2007). Estimating Divergence Times in Large Phylogenetic Trees. \emph{Systematic Biology}, \emph{56}(788777878), 741--752. \url{https://doi.org/10.1080/10635150701613783}

\leavevmode\hypertarget{ref-bryson2014diversification}{}%
Bryson Jr, R. W., Chaves, J., Smith, B. T., Miller, M. J., Winker, K., P칠rez-Em치n, J. L., \& Klicka, J. (2014). Diversification across the new world within the `blue'cardinalids (aves: Cardinalidae). \emph{Journal of Biogeography}, \emph{41}(3), 587--599.

\leavevmode\hypertarget{ref-burleigh2015building}{}%
Burleigh, J. G., Kimball, R. T., \& Braun, E. L. (2015). Building the avian tree of life using a large-scale, sparse supermatrix. \emph{Molecular Phylogenetics and Evolution}, \emph{84}, 53--63.

\leavevmode\hypertarget{ref-burns2014phylogenetics}{}%
Burns, K. J., Shultz, A. J., Title, P. O., Mason, N. A., Barker, F. K., Klicka, J., \ldots{} Lovette, I. J. (2014). Phylogenetics and diversification of tanagers (passeriformes: Thraupidae), the largest radiation of neotropical songbirds. \emph{Molecular Phylogenetics and Evolution}, \emph{75}, 41--77.

\leavevmode\hypertarget{ref-Chamberlain2018}{}%
Chamberlain, S. (2018). \emph{bold: Interface to Bold Systems API}. Retrieved from \url{https://CRAN.R-project.org/package=bold}

\leavevmode\hypertarget{ref-Chamberlain2013}{}%
Chamberlain, S. A., \& Sz칬cs, E. (2013). taxize : taxonomic search and retrieval in R {[}version 2; referees: 3 approved{]}. \emph{F1000Research}, \emph{2}(191), 1--29. \url{https://doi.org/10.12688/f1000research.2-191.v2}

\leavevmode\DIFaddbegin \hypertarget{ref-chang2020estimating}{}%DIF > 
\DIFadd{Chang, J., Rabosky, D. L., \& Alfaro, M. E. (2020). Estimating diversification rates on incompletely sampled phylogenies: Theoretical concerns and practical solutions. }\emph{\DIFadd{Systematic Biology}}\DIFadd{, }\emph{69}(3)\DIFadd{, 602--611.
}

\leavevmode\DIFaddend \hypertarget{ref-chaves2013biogeography}{}%
Chaves, J. A., Hidalgo, J. R., \& Klicka, J. (2013). Biogeography and evolutionary history of the n eotropical genus s altator (a ves: T hraupini). \emph{Journal of Biogeography}, \emph{40}(11), 2180--2190.

\leavevmode\hypertarget{ref-claramunt2015new}{}%
Claramunt, S., \& Cracraft, J. (2015). A new time tree reveals earth history's imprint on the evolution of modern birds. \emph{Science Advances}, \emph{1}(11), e1501005.

\leavevmode\hypertarget{ref-Criscuolo2006}{}%
Criscuolo, A., Berry, V., Douzery, E. J., \& Gascuel, O. (2006). SDM: A fast distance-based approach for (super)tree building in phylogenomics. \emph{Systematic Biology}, \emph{55}(5), 740--755. \url{https://doi.org/10.1080/10635150600969872}

\leavevmode\DIFaddbegin \hypertarget{ref-cusimano2010slowdowns}{}%DIF > 
\DIFadd{Cusimano, N., \& Renner, S. S. (2010). Slowdowns in diversification rates from real phylogenies may not be real. }\emph{\DIFadd{Systematic Biology}}\DIFadd{, }\emph{59}(4)\DIFadd{, 458--464.
}

\leavevmode\hypertarget{ref-cusimano2012new}{}%DIF > 
\DIFadd{Cusimano, N., Stadler, T., \& Renner, S. S. (2012). A new method for handling missing species in diversification analysis applicable to randomly or nonrandomly sampled phylogenies. }\emph{\DIFadd{Systematic Biology}}\DIFadd{, }\emph{61}(5)\DIFadd{, 785--792.
}

\leavevmode\DIFaddend \hypertarget{ref-delsuc2018phylogenomic}{}%
Delsuc, F., Philippe, H., Tsagkogeorga, G., Simion, P., Tilak, M.-K., Turon, X., \ldots{} Douzery, E. J. (2018). A phylogenomic framework and timescale for comparative studies of tunicates. \emph{BMC Biology}, \emph{16}(1), 1--14.

\leavevmode\hypertarget{ref-Eastman2013}{}%
Eastman, J. M., Harmon, L. J., \& Tank, D. C. (2013). Congruification: Support for time scaling large phylogenetic trees. \emph{Methods in Ecology and Evolution}, \emph{4}(7), 688--691. \url{https://doi.org/10.1111/2041-210X.12051}

\leavevmode\hypertarget{ref-edgar2004muscle}{}%
Edgar, R. C. (2004). MUSCLE: Multiple sequence alignment with high accuracy and high throughput. \emph{Nucleic Acids Research}, \emph{32}(5), 1792--1797.

\leavevmode\hypertarget{ref-Felsenstein1985a}{}%
Felsenstein, J. (1985). Phylogenies and the Comparative Method. \emph{The American Naturalist}, \emph{125}(1), 1--15. Retrieved from \url{http://www.jstor.org/stable/2461605}

\leavevmode\DIFaddbegin \hypertarget{ref-forest2005teasing}{}%DIF > 
\DIFadd{Forest, F., Savolainen, V., Chase, M. W., Lupia, R., Bruneau, A., \& Crane, P. R. (2005). Teasing apart molecular-versus fossil-based error estimates when dating phylogenetic trees: A case study in the birch family (betulaceae). }\emph{\DIFadd{Systematic Botany}}\DIFadd{, }\emph{30}(1)\DIFadd{, 118--133.
}

\leavevmode\hypertarget{ref-freckleton2002phylogenetic}{}%DIF > 
\DIFadd{Freckleton, R. P., Harvey, P. H., \& Pagel, M. (2002). Phylogenetic analysis and comparative data: A test and review of evidence. }\emph{\DIFadd{The American Naturalist}}\DIFadd{.
}

\leavevmode\hypertarget{ref-garzon2015incompatible}{}%DIF > 
\DIFadd{Garz칩n-Ordu침a, I. J., Silva-Brand칚o, K. L., Willmott, K. R., Freitas, A. V., \& Brower, A. V. (2015). Incompatible ages for clearwing butterflies based on alternative secondary calibrations. }\emph{\DIFadd{Systematic Biology}}\DIFadd{, }\emph{64}(5)\DIFadd{, 752--767.
}

\leavevmode\DIFaddend \hypertarget{ref-gbif2022taxonomy}{}%
GBIF Secretariat. (2022). GBIF Backbone Taxonomy. \emph{Checklist dataset}, (Online Resource accessed via GBIF.org). Retrieved from \href{\%7Bhttps://doi.org/10.15468/39omei\%20\%7D}{\{https://doi.org/10.15468/39omei \}}

\leavevmode\hypertarget{ref-gibb2015new}{}%
Gibb, G. C., England, R., Hartig, G., McLenachan, P. A., Taylor Smith, B. L., McComish, B. J., \ldots{} Penny, D. (2015). New zealand passerines help clarify the diversification of major songbird lineages during the oligocene. \emph{Genome Biology and Evolution}, \emph{7}(11), 2983--2995.

\leavevmode\hypertarget{ref-graur2004reading}{}%
Graur, D., \& Martin, W. (2004). Reading the entrails of chickens: Molecular timescales of evolution and the illusion of precision. \emph{TRENDS in Genetics}, \emph{20}(2), 80--86.

\leavevmode\hypertarget{ref-hackett2008phylogenomic}{}%
Hackett, S. J., Kimball, R. T., Reddy, S., Bowie, R. C., Braun, E. L., Braun, M. J., \ldots{} others. (2008). A phylogenomic study of birds reveals their evolutionary history. \emph{Science}, \emph{320}(5884), 1763--1768.

\leavevmode\DIFaddbegin \hypertarget{ref-harvey1991comparative}{}%DIF > 
\DIFadd{Harvey, P. H., Pagel, M. D., \& others. (1991). }\emph{\DIFadd{The comparative method in evolutionary biology}} \DIFadd{(Vol. 239). Oxford university press Oxford.
}

\leavevmode\DIFaddend \hypertarget{ref-Hedges2015}{}%
Hedges, S. B., Marin, J., Suleski, M., Paymer, M., \& Kumar, S. (2015). Tree of life reveals clock-like speciation and diversification. \emph{Molecular Biology and Evolution}, \emph{32}(4), 835--845. \url{https://doi.org/10.1093/molbev/msv037}

\leavevmode\hypertarget{ref-Heibl2008}{}%
Heibl, C. (2008). \emph{PHYLOCH: R language tree plotting tools and interfaces to diverse phylogenetic software packages.} Retrieved from \url{http://www.christophheibl.de/Rpackages.html}

\leavevmode\DIFaddbegin \hypertarget{ref-hipsley2014beyond}{}%DIF > 
\DIFadd{Hipsley, C. A., \& M칲ller, J. (2014). Beyond fossil calibrations: Realities of molecular clock practices in evolutionary biology. }\emph{\DIFadd{Frontiers in Genetics}}\DIFadd{, }\emph{\DIFadd{5}}\DIFadd{, 138.
}

\leavevmode\DIFaddend \hypertarget{ref-hooper2017chromosomal}{}%
Hooper, D. M., \& Price, T. D. (2017). Chromosomal inversion differences correlate with range overlap in passerine birds. \emph{Nature Ecology \& Evolution}, \emph{1}(10), 1526.

\leavevmode\hypertarget{ref-Huelsenbeck2001}{}%
Huelsenbeck, J. P., \& Ronquist, F. (2001). MRBAYES: Bayesian inference of phylogenetic trees. \emph{Bioinformatics}, \emph{17}(8), 754--755. \url{https://doi.org/10.1093/bioinformatics/17.8.754}

\leavevmode\hypertarget{ref-Jetz2012}{}%
Jetz, W., Thomas, G., Joy, J. J., Hartmann, K., \& Mooers, A. (2012). The global diversity of birds in space and time. \emph{Nature}, \emph{491}(7424), 444--448. \url{https://doi.org/10.1038/nature11631}

\leavevmode\hypertarget{ref-johansson2008phylogenetic}{}%
Johansson, U. S., Fjelds친, J., \& Bowie, R. C. (2008). Phylogenetic relationships within passerida (aves: Passeriformes): A review and a new molecular phylogeny based on three nuclear intron markers. \emph{Molecular Phylogenetics and Evolution}, \emph{48}(3), 858--876.

\leavevmode\hypertarget{ref-katoh2009multiple}{}%
Katoh, K., Asimenos, G., \& Toh, H. (2009). Multiple alignment of dna sequences with mafft. In \emph{Bioinformatics for dna sequence analysis} (pp. 39--64). Springer.

\leavevmode\hypertarget{ref-kimball2019phylogenomic}{}%
Kimball, R. T., Oliveros, C. H., Wang, N., White, N. D., Barker, F. K., Field, D. J., \ldots{} others. (2019). A phylogenomic supertree of birds. \emph{Diversity}, \emph{11}(7), 109.

\leavevmode\hypertarget{ref-klicka2014comprehensive}{}%
Klicka, J., Barker, F. K., Burns, K. J., Lanyon, S. M., Lovette, I. J., Chaves, J. A., \& Bryson Jr, R. W. (2014). A comprehensive multilocus assessment of sparrow (aves: Passerellidae) relationships. \emph{Molecular Phylogenetics and Evolution}, \emph{77}, 177--182.

\leavevmode\DIFaddbegin \hypertarget{ref-ksepka2011synthesizing}{}%DIF > 
\DIFadd{Ksepka, D. T., Benton, M. J., Carrano, M. T., Gandolfo, M. A., Head, J. J., Hermsen, E. J., \ldots{} others. (2011). }\emph{\DIFadd{Synthesizing and databasing fossil calibrations: Divergence dating and beyond}}\DIFadd{. The Royal Society.
}

\leavevmode\DIFaddend \hypertarget{ref-ksepka2015fossil}{}%
Ksepka, D. T., Parham, J. F., Allman, J. F., Benton, M. J., Carrano, M. T., Cranston, K. A., \ldots{} others. (2015). The fossil calibration database---a new resource for divergence dating. \emph{Systematic Biology}, \emph{64}(5), 853--859.

\leavevmode\hypertarget{ref-lamichhaney2015evolution}{}%
Lamichhaney, S., Berglund, J., Alm칠n, M. S., Maqbool, K., Grabherr, M., Martinez-Barrio, A., \ldots{} others. (2015). Evolution of darwin's finches and their beaks revealed by genome sequencing. \emph{Nature}, \emph{518}(7539), 371--375.

\leavevmode\hypertarget{ref-laubichler2009form}{}%
Laubichler, M. D., \& Maienschein, J. (2009). \emph{Form and function in developmental evolution}. Cambridge University Press.

\leavevmode\hypertarget{ref-lepage2004avibase}{}%
Lepage, D. (2004). \emph{Avibase: The world bird database}. Bird Studies Canada.

\leavevmode\hypertarget{ref-lepage2014avibase}{}%
Lepage, D., Vaidya, G., \& Guralnick, R. (2014). Avibase--a database system for managing and organizing taxonomic concepts. \emph{ZooKeys}, (420), 117.

\leavevmode\hypertarget{ref-lerner2011multilocus}{}%
Lerner, H. R., Meyer, M., James, H. F., Hofreiter, M., \& Fleischer, R. C. (2011). Multilocus resolution of phylogeny and timescale in the extant adaptive radiation of hawaiian honeycreepers. \emph{Current Biology}, \emph{21}(21), 1838--1844.

\leavevmode\hypertarget{ref-lovette2010comprehensive}{}%
Lovette, I. J., P칠rez-Em치n, J. L., Sullivan, J. P., Banks, R. C., Fiorentino, I., C칩rdoba-C칩rdoba, S., \ldots{} others. (2010). A comprehensive multilocus phylogeny for the wood-warblers and a revised classification of the parulidae (aves). \emph{Molecular Phylogenetics and Evolution}, \emph{57}(2), 753--770.

\leavevmode\hypertarget{ref-magallon2001absolute}{}%
Magallon, S., \& Sanderson, M. (2001). Absolute diversification rates in angiosperm clades. \emph{Evolution}, \emph{55}(9), 1762--1780.

\leavevmode\hypertarget{ref-magallon2010using}{}%
Magall칩n, S. (2010). Using fossils to break long branches in molecular dating: A comparison of relaxed clocks applied to the origin of angiosperms. \emph{Systematic Biology}, \emph{59}(4), 384--399.

\leavevmode\hypertarget{ref-magallon2015metacalibrated}{}%
Magall칩n, S., G칩mez-Acevedo, S., S치nchez-Reyes, L. L., \& Hern치ndez-Hern치ndez, T. (2015). A metacalibrated time-tree documents the early rise of flowering plant phylogenetic diversity. \emph{New Phytologist}, \emph{207}(2), 437--453.

\leavevmode\hypertarget{ref-mctavish2015phylesystem}{}%
McTavish, E. J., Hinchliff, C. E., Allman, J. F., Brown, J. W., Cranston, K. A., Holder, M. T., \ldots{} Smith, S. (2015). Phylesystem: A git-based data store for community-curated phylogenetic estimates. \emph{Bioinformatics}, \emph{31}(17), 2794--2800.

\leavevmode\hypertarget{ref-Michonneau2016}{}%
Michonneau, F., Brown, J. W., \& Winter, D. J. (2016). rotl: an R package to interact with the Open Tree of Life data. \emph{Methods in Ecology and Evolution}, \emph{7}(12), 1476--1481. \url{https://doi.org/10.1111/2041-210X.12593}

\leavevmode\hypertarget{ref-Morlon2014}{}%
Morlon, H. (2014). Phylogenetic approaches for studying diversification. \emph{Ecology Letters}, \emph{17}(4), 508--525. \url{https://doi.org/10.1111/ele.12251}

\leavevmode\hypertarget{ref-moyle2016tectonic}{}%
Moyle, R. G., Oliveros, C. H., Andersen, M. J., Hosner, P. A., Benz, B. W., Manthey, J. D., \ldots{} Faircloth, B. C. (2016). Tectonic collision and uplift of wallacea triggered the global songbird radiation. \emph{Nature Communications}, \emph{7}(1), 1--7.

\leavevmode\hypertarget{ref-oliveros2019earth}{}%
Oliveros, C. H., Field, D. J., Ksepka, D. T., Barker, F. K., Aleixo, A., Andersen, M. J., \ldots{} others. (2019). Earth history and the passerine superradiation. \emph{Proceedings of the National Academy of Sciences}, \emph{116}(16), 7916--7925.

\leavevmode\hypertarget{ref-Ooms2018}{}%
Ooms, J., \& Chamberlain, S. (2018). \emph{Phylocomr: Interface to 'phylocom'}. Retrieved from \url{https://CRAN.R-project.org/package=phylocomr}

\leavevmode\hypertarget{ref-opentreeAPIs}{}%
Open Tree Of Life, Redelings, B., Cranston, K. A., Allman, J., Holder, M. T., \& McTavish, E. J. (2016). Open Tree of Life APIs v3.0. \emph{Open Tree of Life Project}, (Online Resources). Retrieved from \href{\%7Bhttps://github.com/OpenTreeOfLife/germinator/wiki/Open-Tree-of-Life-Web-APIs\%7D}{\{https://github.com/OpenTreeOfLife/germinator/wiki/Open-Tree-of-Life-Web-APIs\}}

\leavevmode\hypertarget{ref-opentreeoflife2019synth}{}%
Open Tree Of Life, Redelings, B., S치nchez Reyes, L. L., Cranston, K. A., Allman, J., Holder, M. T., \& McTavish, E. J. (2019). Open tree of life synthetic tree v12.3. \emph{Zenodo}. Retrieved from \url{https://doi.org/10.5281/zenodo.3937742}

\leavevmode\hypertarget{ref-odeen2011evolution}{}%
칐deen, A., H친stad, O., \& Alstr칬m, P. (2011). Evolution of ultraviolet vision in the largest avian radiation-the passerines. \emph{BMC Evolutionary Biology}, \emph{11}(1), 1--8.

\leavevmode\hypertarget{ref-paradis2004}{}%
Paradis, E., Claude, J., \& Strimmer, K. (2004). APE: analyses of phylogenetics and evolution in R language. \emph{Bioinformatics}, \emph{20}(2), 289--290.

\leavevmode\hypertarget{ref-parchman2007coevolution}{}%
Parchman, T. L., Benkman, C. W., \& Mezquida, E. T. (2007). Coevolution between hispaniolan crossbills and pine: Does more time allow for greater phenotypic escalation at lower latitude? \emph{Evolution}, \emph{61}(9), 2142--2153.

\leavevmode\hypertarget{ref-packert2012horizontal}{}%
P칛ckert, M., Martens, J., Sun, Y.-H., Severinghaus, L. L., Nazarenko, A. A., Ting, J., \ldots{} Tietze, D. T. (2012). Horizontal and elevational phylogeographic patterns of himalayan and southeast asian forest passerines (aves: Passeriformes). \emph{Journal of Biogeography}, \emph{39}(3), 556--573.

\leavevmode\hypertarget{ref-pennell2014geiger}{}%
Pennell, M. W., Eastman, J. M., Slater, G. J., Brown, J. W., Uyeda, J. C., FitzJohn, R. G., \ldots{} Harmon, L. J. (2014). Geiger v2. 0: An expanded suite of methods for fitting macroevolutionary models to phylogenetic trees. \emph{Bioinformatics}, \emph{30}(15), 2216--2218.

\leavevmode\hypertarget{ref-posadas2006historical}{}%
Posadas, P., Crisci, J. V., \& Katinas, L. (2006). Historical biogeography: A review of its basic concepts and critical issues. \emph{Journal of Arid Environments}, \emph{66}(3), 389--403.

\leavevmode\hypertarget{ref-powell2014comprehensive}{}%
Powell, A. F., Barker, F. K., Lanyon, S. M., Burns, K. J., Klicka, J., \& Lovette, I. J. (2014). A comprehensive species-level molecular phylogeny of the new world blackbirds (icteridae). \emph{Molecular Phylogenetics and Evolution}, \emph{71}, 94--112.

\leavevmode\DIFaddbegin \hypertarget{ref-powell2020quantifying}{}%DIF > 
\DIFadd{Powell, C. L. E., Waskin, S., \& Battistuzzi, F. U. (2020). Quantifying the error of secondary vs. Distant primary calibrations in a simulated environment. }\emph{\DIFadd{Frontiers in Genetics}}\DIFadd{, }\emph{\DIFadd{11}}\DIFadd{, 252.
}

\leavevmode\DIFaddend \hypertarget{ref-price2014niche}{}%
Price, T. D., Hooper, D. M., Buchanan, C. D., Johansson, U. S., Tietze, D. T., Alstr칬m, P., \ldots{} others. (2014). Niche filling slows the diversification of himalayan songbirds. \emph{Nature}, \emph{509}(7499), 222.

\leavevmode\hypertarget{ref-pulgarin2013multilocus}{}%
Pulgar칤n-R, P. C., Smith, B. T., Bryson Jr, R. W., Spellman, G. M., \& Klicka, J. (2013). Multilocus phylogeny and biogeography of the new world pheucticus grosbeaks (aves: Cardinalidae). \emph{Molecular Phylogenetics and Evolution}, \emph{69}(3), 1222--1227.

\leavevmode\DIFaddbegin \hypertarget{ref-rabosky2015no}{}%DIF > 
\DIFadd{Rabosky, D. L. (2015). No substitute for real data: A cautionary note on the use of phylogenies from birth--death polytomy resolvers for downstream comparative analyses. }\emph{\DIFadd{Evolution}}\DIFadd{, }\emph{69}(12)\DIFadd{, 3207--3216.
}

\leavevmode\DIFaddend \hypertarget{ref-rabosky2018inverse}{}%
Rabosky, D. L., Chang, J., Title, P. O., Cowman, P. F., Sallan, L., Friedman, M., \ldots{} others. (2018). An inverse latitudinal gradient in speciation rate for marine fishes. \emph{Nature}, \emph{559}(7714), 392.

\leavevmode\hypertarget{ref-ramshaw1972time}{}%
Ramshaw, J., Richardson, D., Meatyard, B., Brown, R., Richardson, M., Thompson, E., \& Boulter, D. (1972). The time of origin of the flowering plants determined by using amino acid sequence data of cytochrome c. \emph{New Phytologist}, \emph{71}(5), 773--779.

\leavevmode\hypertarget{ref-ratnasingham2007bold}{}%
Ratnasingham, S., \& Hebert, P. D. (2007). BOLD: The barcode of life data system (http://www. Barcodinglife. Org). \emph{Molecular Ecology Notes}, \emph{7}(3), 355--364.

\leavevmode\hypertarget{ref-RCoreTeam2018}{}%
R Core Team. (2018). \emph{R: a language and environment for statistical computing}. Vienna, Austria: R Foundation for Statistical Computing.

\leavevmode\hypertarget{ref-rees2017automated}{}%
Rees, \& Cranston, K. (2017). Automated assembly of a reference taxonomy for phylogenetic data synthesis. \emph{Biodiversity Data Journal}, (5).

\leavevmode\hypertarget{ref-rees2017irmng}{}%
Rees, Vandepitte, L., Decock, W., \& Vanhoorne, B. (2017). IRMNG 2006--2016: 10 Years of a Global Taxonomic Database. \emph{Biodiversity Informatics}, \emph{12}.

\leavevmode\hypertarget{ref-Revell2012}{}%
Revell, L. J. (2012). Phytools: An r package for phylogenetic comparative biology (and other things). \emph{Methods in Ecology and Evolution}, \emph{3}, 217--223.

\leavevmode\hypertarget{ref-Ronquist2003}{}%
Ronquist, F., \& Huelsenbeck, J. P. (2003). MrBayes 3: Bayesian phylogenetic inference under mixed models. \emph{Bioinformatics}, \emph{19}(12), 1572--1574. \url{https://doi.org/10.1093/bioinformatics/btg180}

\leavevmode\DIFaddbegin \hypertarget{ref-roquet2014one}{}%DIF > 
\DIFadd{Roquet, C., Lavergne, S., \& Thuiller, W. (2014). One tree to link them all: A phylogenetic dataset for the european tetrapoda. }\emph{\DIFadd{PLoS Currents}}\DIFadd{, }\emph{\DIFadd{6}}\DIFadd{.
}

\leavevmode\DIFaddend \hypertarget{ref-datelifeplot}{}%
Sanchez-Reyes, L. L., \& O'Meara, B. (2022). \texttt{datelifeplot}: Methods to plot chronograms and outputs of the \texttt{datelife} package. \emph{R Package Release V0.2.2}. Retrieved from \url{https://zenodo.org/badge/latestdoi/381501451}

\leavevmode\hypertarget{ref-sanchez2022}{}%
Sanchez-Reyes, L. L., O'Meara, B., Eastman, J., Heath, T., Wright, A., Schliep, K., \ldots{} Alfaro, M. (2022). \texttt{datelife}: Scientific Data on Time of Lineage Divergence for Your Taxa. In \emph{R package version 0.6.6}. Retrieved from \href{https://CRAN.R-project.org/package=datelife\%20and\%20https://doi.org/10.5281/zenodo.593938}{https://CRAN.R-project.org/package=datelife and https://doi.org/10.5281/zenodo.593938}

\leavevmode\hypertarget{ref-sanderson2002}{}%
Sanderson, M. (2002). Estimating Absolute Rates of Molecular Evolution and Divergence Times: A Penalized Likelihood Approach. \emph{Molecular Biology and Evolution}, \emph{19}(1), 101--109. \url{https://doi.org/10.1093/oxfordjournals.molbev.a003974}

\leavevmode\hypertarget{ref-sanderson2003r8s}{}%
Sanderson, M. (2003). r8s: Inferring Absolute Rates of Molecular Evolution and Divergence Times in the Absence of a Molecular Clock. \emph{Bioinformatics}, \emph{19}(2), 301--302.

\leavevmode\hypertarget{ref-sanderson2001sources}{}%
Sanderson, M., \& Doyle, J. (2001). Sources of error and confidence intervals in estimating the age of angiosperms from rbcL and 18S rDNA data. \emph{American Journal of Botany}, \emph{88}(8), 1499--1516.

\leavevmode\hypertarget{ref-sauquet2013practical}{}%
Sauquet, H. (2013). A practical guide to molecular dating. \emph{Comptes Rendus Palevol}, \emph{12}(6), 355--367.

\leavevmode\DIFaddbegin \hypertarget{ref-sauquet2012testing}{}%DIF > 
\DIFadd{Sauquet, H., Ho, S. Y. W., Gandolfo, M. a, Jordan, G. J., Wilf, P., Cantrill, D. J., \ldots{} Udovicic, F. (2012). Testing the impact of calibration on molecular divergence times using a fossil-rich group: the case of Nothofagus (Fagales). }\emph{\DIFadd{Systematic Biology}}\DIFadd{, }\emph{61}(2)\DIFadd{, 289--313. }\url{https://doi.org/10.1093/sysbio/syr116}

\leavevmode\DIFaddend \hypertarget{ref-sauquet2021age}{}%
Sauquet, H., Ram칤rez-Barahona, S., \& Magall칩n, S. (2021). \emph{The age of flowering plants is unknown}.

\leavevmode\hypertarget{ref-schenk2016sec}{}%
Schenk, J. J. (2016). Consequences of secondary calibrations on divergence time estimates. \emph{PLoS ONE}, \emph{11}(1). \url{https://doi.org/10.1371/journal.pone.0148228}

\leavevmode\hypertarget{ref-schliep2011phangorn}{}%
Schliep, K. P. (2011). Phangorn: Phylogenetic analysis in r. \emph{Bioinformatics}, \emph{27}(4), 592--593.

\leavevmode\hypertarget{ref-schoch2020ncbi}{}%
Schoch, C. L., Ciufo, S., Domrachev, M., Hotton, C. L., Kannan, S., Khovanskaya, R., \ldots{} others. (2020). NCBI Taxonomy: a Comprehensive Update on Curation, Resources and Tools. \emph{Database}, \emph{2020}.

\leavevmode\hypertarget{ref-selvatti2015paleogene}{}%
Selvatti, A. P., Gonzaga, L. P., \& Moraes Russo, C. A. de. (2015). A paleogene origin for crown passerines and the diversification of the oscines in the new world. \emph{Molecular Phylogenetics and Evolution}, \emph{88}, 1--15.

\leavevmode\DIFaddbegin \hypertarget{ref-shaul2002playing}{}%DIF > 
\DIFadd{Shaul, S., \& Graur, D. (2002). Playing chicken (gallus gallus): Methodological inconsistencies of molecular divergence date estimates due to secondary calibration points. }\emph{\DIFadd{Gene}}\DIFadd{, }\emph{300}(1-2)\DIFadd{, 59--61.
}

\leavevmode\DIFaddend \hypertarget{ref-smith2018constructing}{}%
Smith, S., \& Brown, J. (2018). Constructing a broadly inclusive seed plant phylogeny. \emph{American Journal of Botany}, \emph{105}(3), 302--314.

\leavevmode\hypertarget{ref-Smith2012}{}%
Smith, S., \& O'Meara, B. (2012). TreePL: Divergence time estimation using penalized likelihood for large phylogenies. \emph{Bioinformatics}, \emph{28}(20), 2689--2690. \url{https://doi.org/10.1093/bioinformatics/bts492}

\leavevmode\hypertarget{ref-Stoltzfus2013}{}%
Stoltzfus, A., Lapp, H., Matasci, N., Deus, H., Sidlauskas, B., Zmasek, C. M., \ldots{} Jordan, G. (2013). Phylotastic! Making tree-of-life knowledge accessible, reusable and convenient. \emph{BMC Bioinformatics}, \emph{14}. \url{https://doi.org/10.1186/1471-2105-14-158}

\leavevmode\DIFaddbegin \hypertarget{ref-sun2020estimating}{}%DIF > 
\DIFadd{Sun, M., Folk, R. A., Gitzendanner, M. A., Soltis, P. S., Chen, Z., Soltis, D. E., \& Guralnick, R. P. (2020). Estimating rates and patterns of diversification with incomplete sampling: A case study in the rosids. }\emph{\DIFadd{American Journal of Botany}}\DIFadd{, }\emph{107}(6)\DIFadd{, 895--909.
}

\leavevmode\DIFaddend \hypertarget{ref-tietze2013complete}{}%
Tietze, D. T., P칛ckert, M., Martens, J., Lehmann, H., \& Sun, Y.-H. (2013). Complete phylogeny and historical biogeography of true rosefinches (aves: Carpodacus). \emph{Zoological Journal of the Linnean Society}, \emph{169}(1), 215--234.

\leavevmode\hypertarget{ref-title2016macrophylogenies}{}%
Title, P. O., \& Rabosky, D. L. (2016). Do Macrophylogenies Yield Stable Macroevolutionary Inferences? An Example from Squamate Reptiles. \emph{Systematic Biology}, syw102. \url{https://doi.org/10.1093/sysbio/syw102}

\leavevmode\hypertarget{ref-treplin2008molecular}{}%
Treplin, S., Siegert, R., Bleidorn, C., Thompson, H. S., Fotso, R., \& Tiedemann, R. (2008). Molecular phylogeny of songbirds (aves: Passeriformes) and the relative utility of common nuclear marker loci. \emph{Cladistics}, \emph{24}(3), 328--349.

\leavevmode\DIFaddbegin \hypertarget{ref-uyeda2017evolution}{}%DIF > 
\DIFadd{Uyeda, J. C., Pennell, M. W., Miller, E. T., Maia, R., \& McClain, C. R. (2017). The evolution of energetic scaling across the vertebrate tree of life. }\emph{\DIFadd{The American Naturalist}}\DIFadd{, }\emph{190}(2)\DIFadd{, 185--199.
}

\leavevmode\DIFaddend \hypertarget{ref-vos2012nexml}{}%
Vos, R. A., Balhoff, J. P., Caravas, J. A., Holder, M. T., Lapp, H., Maddison, W. P., \ldots{} others. (2012). NeXML: Rich, extensible, and verifiable representation of comparative data and metadata. \emph{Systematic Biology}, \emph{61}(4), 675--689. \url{https://doi.org/10.1093/sysbio/sys025}

\leavevmode\DIFaddbegin \hypertarget{ref-vos2004reconstructing}{}%DIF > 
\DIFadd{Vos, R. A., \& Mooers, A. 칒. (2004). Reconstructing divergence times for supertrees: A molecular approach. }\emph{\DIFadd{Phylogenetic Supertrees: Combining Information to Reveal the Tree of Life}}\DIFadd{, 281--299.
}

\leavevmode\DIFaddend \hypertarget{ref-Webb2000}{}%
Webb, C. (2000). Exploring the Phylogenetic Structure of Ecological Communities : An Example for Rain Forest Trees. \emph{The American Naturalist}, \emph{156}(2), 145--155.

\leavevmode\hypertarget{ref-Webb2008}{}%
Webb, C., Ackerly, D., \& Kembel, S. (2008). Phylocom: Software for the analysis of phylogenetic community structure and trait evolution. \emph{Bioinformatics}, \emph{24}(18), 2098--2100. \url{https://doi.org/10.1093/bioinformatics/btn358}

\leavevmode\hypertarget{ref-webb2005phylomatic}{}%
Webb, C., \& Donoghue, M. (2005). Phylomatic: Tree assembly for applied phylogenetics. \emph{Molecular Ecology Notes}, \emph{5}(1), 181--183.

\leavevmode\hypertarget{ref-weir2008calibrating}{}%
Weir, J., \& Schluter, D. (2008). Calibrating the avian molecular clock. \emph{Molecular Ecology}, \emph{17}(10), 2321--2328.

\leavevmode\hypertarget{ref-zuccon2012phylogenetic}{}%
Zuccon, D., Pr콩s-Jones, R., Rasmussen, P. C., \& Ericson, P. G. (2012). The phylogenetic relationships and generic limits of finches (fringillidae). \emph{Molecular Phylogenetics and Evolution}, \emph{62}(2), 581--596.

\endgroup
\DIFdelbegin %DIFDELCMD < 

%DIFDELCMD < \newpage
%DIFDELCMD < 

%DIFDELCMD < \captionof{table}{Ages of congruified nodes. See Figure 3, step C2.}
%DIFDELCMD < \label{tbl:table1}
%DIFDELCMD < \includegraphics[width=\linewidth]{../tables/table-fringillidae-small-example.pdf}
%DIFDELCMD < 

%DIFDELCMD < \vspace{15mm}
%DIFDELCMD < 

%DIFDELCMD < \captionof{table}{Summary of congruified nodes ages. See Figure 3, step C3.}
%DIFDELCMD < 

%DIFDELCMD < \label{tbl:table2}
%DIFDELCMD < \includegraphics[width=\linewidth]{../tables/table-fringillidae-small-example-summary.png}
%DIFDELCMD < 

%DIFDELCMD < \begin{minipage}[t]{17.3cm}
%DIFDELCMD < \includegraphics{../figures/figure-workflow/figure1-new.pdf}
%DIFDELCMD < \captionof{figure}{\textit{Main DateLife workflow. Analyses can be performed via DateLife's interactive website at \url{www.datelife.org}, or using the \texttt{datelife} R package. Details on the R functions used to perform the analyses are available from \texttt{datelife}'s R package vignettes at \url{https://phylotastic.org/datelife}.}}
%DIFDELCMD < \label{fig:workflow}
%DIFDELCMD < \end{minipage}
%DIFDELCMD < 

%DIFDELCMD < \includegraphics{../figures/fig_runtime_main.pdf}
%DIFDELCMD < \captionof{figure}{\textit{DateLife's benchmarking results. Computation time used to process a query and a search across \texttt{datelife}'s chronogram database, relative to number of input taxon names. For each N = \{10, 100, 200, ..., 1 000, ... , 9 000, 10 000\}, we sampled N species names from the class Aves a hundred times, and then performed a datelife search processing the input names with Taxon Names Resolution Service (TNRS; light gray), and without processing names (dark gray). For comparison, we performed a search using an input that had been pre-processed with TNRS (light blue).}}
%DIFDELCMD < \label{fig:runtime_main}
%DIFDELCMD < 

%DIFDELCMD < \begin{center}
%DIFDELCMD < \includegraphics{../figures/figure-small-example/figure2.pdf}
%DIFDELCMD < \end{center}
%DIFDELCMD < \vspace{10mm}
%DIFDELCMD < \captionof{figure}{\textit{DateLife analysis results for a small sample of A) 6 bird species within the Passeriformes. B) Processed species names were found across 9 chronograms within 6 independent studies (Barker et al. (\protect\hyperlink{ref-barker2012going}{2012}), Barker et al. (\protect\hyperlink{ref-barker2015new}{2015}), Burns et al. (\protect\hyperlink{ref-burns2014phylogenetics}{2014}), Hedges et al. (\protect\hyperlink{ref-Hedges2015}{2015}), Hooper and Price (\protect\hyperlink{ref-hooper2017chromosomal}{2017}), Jetz et al. (\protect\hyperlink{ref-Jetz2012}{2012}).) C) This revealed 28 source age data points for the queried species names. Summarized age data is used as secondary calibrations to date a tree topology obtained from OpenTree's synthetic tree, resulting in a summary chronogram of source ages.}}
%DIFDELCMD < 

%DIFDELCMD < \label{fig:figure2}
%DIFDELCMD < 

%DIFDELCMD < \begin{minipage}{18.2cm}
%DIFDELCMD < \includegraphics{../figures/fringillidae-topologies/fringillidae-topology.pdf}
%DIFDELCMD < \captionof{figure}{Tree topologies extracted from Open Tree of Life's (OpenTree) synthetic phylogenetic tree.
%DIFDELCMD < A) Topology of 2,333 tips and 1,305 internal nodes, encompassing bird species within the family Fringillidae following the NCBI taxonomy (black), as well as all other bird species that share the same Most Recent Common Ancestor (MRCA) node in OpenTree's synthetic tree (purple).
%DIFDELCMD < B) Subtree topology of 289 tips and 253 internal nodes, resulting from pruning species that do not belong to the family Fringillidae according to the NCBI taxonomy (purple branches in topology A).
%DIFDELCMD < Bird species within the Fringillidae are paraphyletic
%DIFDELCMD < (Alstr칬m et al. \protect\hyperlink{ref-alstrom2014discovery}{2014},
%DIFDELCMD < Barker, Cibois, Schikler, Feinstein, \& Cracraft \protect\hyperlink{ref-barker2004phylogeny}{2004},
%DIFDELCMD < Barker et al. \protect\hyperlink{ref-barker2013going}{2013},
%DIFDELCMD < Barker \protect\hyperlink{ref-barker2014mitogenomic}{2014},
%DIFDELCMD < Barker et al. \protect\hyperlink{ref-barker2015new}{2015},
%DIFDELCMD < Beresford, Barker, Ryan, \& Crowe \protect\hyperlink{ref-beresford2005african}{2005},
%DIFDELCMD < Bryson Jr et al. \protect\hyperlink{ref-bryson2014diversification}{2014},
%DIFDELCMD < Burleigh, Kimball, \& Braun \protect\hyperlink{ref-burleigh2015building}{2015},
%DIFDELCMD < Burns et al. \protect\hyperlink{ref-burns2014phylogenetics}{2014},
%DIFDELCMD < Chaves, Hidalgo, \& Klicka \protect\hyperlink{ref-chaves2013biogeography}{2013},
%DIFDELCMD < Claramunt \& Cracraft \protect\hyperlink{ref-claramunt2015new}{2015},
%DIFDELCMD < Gibb et al. \protect\hyperlink{ref-gibb2015new}{2015},
%DIFDELCMD < Hackett et al. \protect\hyperlink{ref-hackett2008phylogenomic}{2008},
%DIFDELCMD < Jetz et al. \protect\hyperlink{ref-Jetz2012}{2012},
%DIFDELCMD < Johansson, Fjelds친, \& Bowi \protect\hyperlink{ref-johansson2008phylogenetic}{200},
%DIFDELCMD < Kimball et al. \protect\hyperlink{ref-kimball2019phylogenomic}{2019},
%DIFDELCMD < Klicka et al. \protect\hyperlink{ref-klicka2014comprehensive}{2014},
%DIFDELCMD < Lamichhaney et al. \protect\hyperlink{ref-lamichhaney2015evolution}{2015},
%DIFDELCMD < Lerner, Meyer, James, Hofreiter, \& Fleischer \protect\hyperlink{ref-lerner2011multilocus}{2011},
%DIFDELCMD < Lovette et al. \protect\hyperlink{ref-lovette2010comprehensive}{2010},
%DIFDELCMD < Moyle et al. \protect\hyperlink{ref-moyle2016tectonic}{2016},
%DIFDELCMD < 칐deen, H친stad, \& Alstr칬m \protect\hyperlink{ref-odeen2011evolution}{2011},
%DIFDELCMD < Oliveros et al. \protect\hyperlink{ref-oliveros2019earth}{2019},
%DIFDELCMD < P칛ckert et al. \protect\hyperlink{ref-packert2012horizontal}{2012},
%DIFDELCMD < Parchman, Benkman, \& Mezquida \protect\hyperlink{ref-parchman2007coevolution}{2007},
%DIFDELCMD < Powell et al. \protect\hyperlink{ref-powell2014comprehensive}{2014},
%DIFDELCMD <   Price et al. \protect\hyperlink{ref-price2014niche}{2014},
%DIFDELCMD <   Pulgar칤n-R, Smith, Bryson Jr, Spellman, \& Klicka \protect\hyperlink{ref-pulgarin2013multilocus}{2013},
%DIFDELCMD <   Selvatti, Gonzaga, \& Moraes Russo \protect\hyperlink{ref-selvatti2015paleogene}{2015},
%DIFDELCMD <   Tietze, P칛ckert, Martens, Lehmann, \& Sun \protect\hyperlink{ref-tietze2013complete}{2013},
%DIFDELCMD <   Treplin et al. \protect\hyperlink{ref-treplin2008molecular}{2008},
%DIFDELCMD <   Zuccon, Pr콩s-Jones, Rasmussen, \& Ericson \protect\hyperlink{ref-zuccon2012phylogenetic}{2012}).}
%DIFDELCMD < \label{fig:fringillidae-topologies}
%DIFDELCMD < \end{minipage}
%DIFDELCMD < 

%DIFDELCMD < \begin{minipage}{18cm}
%DIFDELCMD < \begin{center}
%DIFDELCMD < \includegraphics{../figures/figure-fringillidae/median_and_calibration_ages-congruified.png}
%DIFDELCMD < \end{center}
%DIFDELCMD < 

%DIFDELCMD < \captionof{figure}{Fringillidae median summary chronogram generated with DateLife. It has 289 tips and 253 nodes, from which 212 have age data from at least one published chronogram. These nodes have age data points from 19 different chronograms from 13 different studies (colored circles), which were summarized into 194 data points that were used as secondary calibrations (purple slashes) with the Branch Length Adjuster (BLADJ ) software from Webb et al., (\protect\hyperlink{ref-Webb2008}{2008}).}
%DIFDELCMD < \label{fig:fringillidages}
%DIFDELCMD < \end{minipage}
%DIFDELCMD < 

%DIFDELCMD < \newpage
%DIFDELCMD < 

%DIFDELCMD < \begin{minipage}{18cm}
%DIFDELCMD < \begin{center}
%DIFDELCMD < \includegraphics{../figures/fringillidae-cross-validation/fig-cross-validation-xy-plots.pdf}
%DIFDELCMD < \end{center}
%DIFDELCMD < \captionof{figure}{Results from cross validation analysis. Each plot compares the original age estimate (x axis) with the age obtained with a DateLife analysis (y axis), per node.}
%DIFDELCMD < \label{fig:cvXY}
%DIFDELCMD < \end{minipage}
%DIFDELCMD < %%%
\DIFdelend 


\end{document}
