---
title             : "DateLife: leveraging databases and analytical tools to reveal the dated Tree of Life"
shorttitle        : "DateLife: revealing the dated Tree of Life"

author:
  - name          : "Luna L. SÃ¡nchez Reyes"
    affiliation   : "1,2"
    corresponding : yes    # Define only one corresponding author
    address       : ""
    email         : "sanchez.reyes.luna@gmail.com"
    role:         # Contributorship roles (e.g., CRediT, https://casrai.org/credit/)
      - Data curation
      - Investigation
      - Software
      - Visualization
      - Validation
      - Writing - Original Draft Preparation
      - Writing - Review & Editing
  - name          : "Emily Jane McTavish"
    affiliation   : "1"
    role:
      - Resources
      - Software
      - Writing - Review & Editing
  - name          : "Brian O'Meara"
    affiliation   : "2"
    role:
      - Conceptualization
      - Funding acquisition
      - Methodology
      - Resources
      - Software
      - Supervision
      - Writing - Review & Editing

affiliation:
  - id            : "1"
    institution   : "University of California, Merced"
  - id            : "2"
    institution   : "University of Tennessee, Knoxville"

authornote: |
  School of Natural Sciences, University of California, Merced, Science and Engineering Building 1.

  Department of Ecology and Evolutionary Biology, University of Tennessee, Knoxville, 425 Hesler Biology Building, Knoxville, TN 37996, USA.

abstract: |
  Date estimates for times of evolutionary divergences are key data for research in the natural sciences. These estimates also provide valuable information for for education, science communication and policy decisions.
  Although achieving a high-quality reconstruction of a phylogenetic tree with branch lengths proportional to absolute time (chronogram), is a difficult and time-consuming task, the increased availability of fossil and molecular data, and time-efficient analytical techniques has resulted in many recent publications of large chronograms for a large number and wide diversity of organisms.
  When these estimates are shared in public, open databases this wealth of expertly-curated and peer-reviewed data on time of evolutionary origin is exposed in a programatic and reusable way.
  Intensive and localized efforts have improved data sharing practices, as well as incentivizited open science in biology.
  Here we present DateLife, a service implemented as an R package and an Rshiny website application available at www.datelife.org/query/, that provides functionalities for efficient and easy finding, summary, reuse, and reanalysis of expert, peer-reviewed, public data on time of evolutionary origin.
  The main DateLife workflow constructs a chronogram for any given combination of taxon names, by searching a local chronogram database constructed and curated from the Open Tree of Life Phylesystem phylogenetic database, which incorporates phylogenetic data from TreeBASE database as well.
  We implement and test methods for summarizing time data from multiple source chronograms using supertree and congruification algorithms, and using age data extracted from source chronograms as secondary calibration points to add branch lengths proportional to absolute time to a tree topology.
  DateLife will be useful to increase awereness on the existing variation in expert time of divergence data, and can foster exploration of the effect of alternative divergence time hypothesis on the results of analyses, providing a framework for a more informed interpretation of evolutionary results.

  <!--  R has become mainstream in the biological-data analyst toolkit. Hence, to improve accessibility of time of evolutionary origin data we developed an R package that facilitates interaction, reanalysis and reuse of it, as well as incorporation of these data into the evolutionary analysis workflow. -->

bibliography      : ["paper_references.bib"]
keywords          : "Tree; Phylogeny; Scaling; Dating; Ages; Divergence times; Open Science; Congruification; Supertree; Calibrations; Secondary calibrations"
wordcount         : "`r wordcountaddin::word_count()`"
floatsintext      : no
figurelist        : no
tablelist         : no
footnotelist      : no
linenumbers       : yes
mask              : no
draft             : no

documentclass     : "apa6"
classoption       : "man"
output            :
  papaja::apa6_pdf:
    includes      :
      in_header: "preamble.tex"
      after_body  : "figures-alt.tex"

nocite: |
  @barker2013going, @barker2015new, @burns2014phylogenetics, @claramunt2015new, @gibb2015new, @Hedges2015, @hooper2017chromosomal, @Jetz2012, @price2014niche
---

```{r setup, include = FALSE}
library("papaja")
r_refs("paper_references.bib")
```

```{r analysis-preferences}
# Seed for random number generation
set.seed(50)
knitr::opts_chunk$set(cache.extra = knitr::rand_seed)
data(opentree_chronograms, package = "datelife")
rpackage_available <- TRUE
availability <- ifelse(rpackage_available, "is available", "will be available")
load("../data/dquery.rda")
load("../data/dres.rda")
```

# Introduction
<!-- If you're aiming for syst bio you don't want to start so general Readers know what chronograms and phylogenies are!.-->
Chronograms --phylogenies with branch lengths proportional to time-- provide key data for the study of natural processes in many areas of biological research, such as developmental biology [@laubichler2009form; @delsuc2018phylogenomic], conservation biology [@Felsenstein1985a; @Webb2000], historical biogeography [@posadas2006historical], and species diversification [@magallon2001absolute; @Morlon2014].

Building a chronogram is not an easy task.
It requires obtaining and curating data to construct a phylogeny; selecting and placing appropriate calibrations on the phylogeny using independent age data points from the fossil record or other dated events, and inferring the full dated tree.
Estimating accurate chronograms generally requires specialized biological training, taxonomic domain knowledge, and a non-negligible amount of research time, computational resources and funding.

Here we present the DateLife software application, available as an R package and as an online Rshiny interactive website at www.datelife.org/query/, which captures data from published chronograms, and make these data readily accessible to users.
DateLife features a versioned, open and fully public chronogram database [@mctavish2015phylesystem] storing age information in a computer readable format [@vos2012nexml], an automated and programmatic way of accessing the data [@Stoltzfus2013] and methods to summarize and compare age data.


# Description

The DateLife algorithm is fully implemented using the R language. The latest stable version of the R package `datelife` is available from the CRAN repository (v`r packageVersion("datelife")`; @datelifeR), and relies on functionalities from various biological R packages:
ape [@paradis2004],
bold [@Chamberlain2018],
geiger [@Harmon2008],
paleotree [@Bapst2012a],
phyloch [@Heibl2008],
phylocomr [@Ooms2018],
phytools [@Revell2012],
rotl [@Michonneau2016], and
taxize [@Chamberlain2013; @Chamberlain2018].
Figure \ref{fig:workflow} provides a graphical summary of the three main steps of the DateLife algorithm: providing an input, searching a chronogram database, and summarizing results from the search.

<!--_**Providing an input.--**_ -->

## Providing an input

DateLife starts with an input query consisting of at least two taxon names, which can be provided as a comma separated character string, or as tip labels on a tree. If the input is a tree, it can be provided as a classic newick character string [@archie1986newick], or as a "phylo" R object [@paradis2004]. The input tree is not required to have branch lengths, and its topology is used in the summary steps described below.

DateLife accepts scientific names as input.
These names can belong to any inclusive taxonomic group (e.g., genus, family, tribe, etc.) or binomial specific. Subspecies and variants are ignored. If an input taxon name belongs to an inclusive taxonomic group the algorithm has two alternative behaviors defined by the "get species from taxon" flag. If the flag is active, the DateLife algorithm retrieves all species names within the inclusive taxonomic group and adds them to the input. If the flag is inactive, DateLife ignores the inclusive taxon names from the input.

Input scientific names are processed using a Taxonomic Name Resolution Service (TNRS), which increases the probability of correctly finding the queried taxon names in the chronogram database. TNRS detects, corrects and standardizes name misspellings and typos, variant spellings and authorities, and nomenclatural synonyms to a single taxonomic standard. DateLife implements TNRS using OpenTree's taxonomy as standard [@opentreeAPIs; @rees2017automated].
<!--_I reworded a bit, and removed a Boyle citation because it sounded like you were using their TNRS - but you aren't right?  -->

The processed input taxon names are saved as an R object of a newly defined class \texttt{datelifeQuery} that is used in the following steps. This object contains the processed names, the corresponding OpenTree taxonomic id numbers, and the topology of the input tree if any was provided.

<!--_**Searching the database.--**_  -->

## Searching the database

A DateLife search consists of matching processed taxon names to tip labels in a chronogram database. Chronograms with at least two matching tip labels are identified and pruned down to preserve only the matched tips.
These matching pruned chronograms are now referred to as source chronograms.
Total distance (in units of millions of years) between taxon pairs within each source chronogram are stored as a patristic distance matrix (Figure \ref{fig:workflow}).
The matrix format speeds up extraction of pairwise taxon ages of any queried taxa, as opposed to searching the ancestor node of a pair of taxa in a "phylo" object or newick string.
Finally, the patristic matrices are associated to the study citation where the original chronogram was published, and stored as an R object of the newly defined class \texttt{datelifeResult}.


DateLife's chronogram database latest version consist of `r length(opentree_chronograms$trees)` chronograms published in `r length(unique(opentree_chronograms$studies))` different studies. It is constructed from OpenTree's phylogenetic database, the Phylesystem, which constitutes an open source of expert phylogenetic knowledge with rich metadata [@mctavish2015phylesystem] that allows automatic and reproducible construction of a chronogram database.
New chronograms can be added to Phylesystem by any user and are immediately publicly available. This permits an immediate update of DateLife's chronogram database to include those new data within a run.



<!-- _**Summarizing search results.--**_ -->

## Summarizing search results

Summary information is extracted from the \texttt{datelifeResult} object to inform decisions for the subsequent steps in the user workflow. Basic summary information available to the user is:

1. The matching pruned chronograms as newick strings or "phylo" objects.
2. The ages of the root of all source chronograms. This can correspond to the age of the most recent common ancestor (mrca) of your group of interest if the source chronograms have all taxa belonging to the group. If not, the root corresponds to the mrca of a subgroup withing your group of interest.
3. Study citations where original chronograms were published.
4. A report of input taxon names matches across source chronograms.
5. The source chronogram(s) with the most input taxon names.
6. Last but not least, age data from source chronograms are summarized and used to generate a single summary chronogram:

_**Choosing a topology.--**_ DateLife requires a tree topology to summarize age data upon. We recommend that users provide one as input from the literature, or one of their own making. If no topology is provided, DateLife automatically subsets one from the OpenTree synthetic tree [@opentreeoflife2019synth]. Alternatively, DateLife can reconstruct a tree with branch lengths proportional to substitution rates from a starting tree topology using genetic data from the Barcode of Life Data System [BOLD; @ratnasingham2007bold], or combine topologies from source chronogras using a supertree approach.

_**Reconstructing branch lengths.--**_ DateLife starts by mining the BOLD database to obtain genetic markers for the input taxa, and aligning them with MUSCLE [@edgar2004muscle; or MAFFT @katoh2009multiple]. Currently, branch length reconstruction is performed with parsimony and the likelihood of the phylogenetic tree given a sequence alignment is computed [@schliep2011phangorn].

_**Combining source chronograms.--**_ To combine topologies from source chronograms into a single summary topology, the DateLife algorithm starts by identifying the source chronograms that form a grove, roughly, a sufficiently overlapping set of taxa between trees, by implementing definition 2.8 for n-overlap from AnÃ© et al. [-@ane2009groves]. In rare cases, a group of trees can have multiple groves. By default, DateLife chooses the grove with the most taxa, however, the "criterion = trees" flag allows the user to choose the grove with the most trees instead.
If source chronograms do not form a grove, the supertree reonstruction will fail.

_**Congruifying nodes.--**_ Once with a chosen topology, DateLife applies the congruification method [@Eastman2013] to find nodes belonging to the same clade across source chronograms, and extract the corresponding node ages from the patristic distance matrices stored as `datelifeResult`. Since by definition the matrices store total distance (time from tip to tip), node ages correspond to half the values stored in the patristic distance matrices.
A table of congruified node ages that can be used as secondary calibrations is stored as a `congruifiedCalibrations` object.

_**Summarizing congruified ages.--**_ Congruified pairwise node ages are used to calculate summary statistics (mean, median, minimum and maximum ages) per taxon pair. <!-- we should calculate summary statistics per node -->
<!--??? How is 'calibrations' defined here? the calibrations from congruification, or the pairwise distances??? is calibrations from congruification-->
into a single summary matrix using different methods.
Summarizing options implemented include Super Distance Matrix method [SDM, @Criscuolo2006] and .

Ages per taxon pair can be used as calibrations to date the chosen tree topology.
<!--
For each node in the output tree topology, the pairwise distances that traverse that node are ..???
Under the SDM criterion ??? what happens??
Using the minimum age criterion ..??
-->


_**Dating the tree topology.--**_ 
By default, DateLife implements the Branch Length Adjuster (BLADJ) algorithm to obtain a fully datd topology. BLADJ fixes node ages that have calibration data, and distrivutes time between nodes with no data evenly between nodes with calibration data. This minimizes age variance in the resulting chronogram [@Webb2008].
When there is conflict in ages between nodes with calibration data, the algorithm ignores ages that are older than ages of parent nodes and/or younger than ages from descendant nodes.

If there is no information on the age of the root in the chronogram database, users can provide an estimate from the literature. If none is provided, DateLife assigns an arbitrary age to the root as 10% older than the oldest age available within the group.

Summarized calibrations can be applied as secondary calibrations with different dating methods currently supported within DateLife: MrBayes [@Huelsenbeck2001; @Ronquist2003], PATHd8 [@Britton2007], BLADJ [@Webb2008; @webb2005phylomatic], and treePL [@Smith2012].
<!--Huh? can you run MrBayes in DateLife? You should explain how the BOLD dating with PathD8 works here, and what is needed for the other ones!-->

<!--PATHd8 is a non-clock, rate-smoothing method [@Britton2007] to date trees.
treePL, is a semi-parametric, rate-smoothing, penalized likelihood dating method
[@Smith2012].
MrBayes program [@Huelsenbeck2001; @Ronquist2003] can be used when adding taxa at
random, following a reference taxonomy or a topological constraint. It draws ages
from a pure birth model, as implemented by Jetz and collaborators [-@Jetz2012].

If there is no information available for some input taxon name, users can also create age and phylogenetic data for the missing branches with MrBayes.
-->

_**Visualizing results.--**_
Finally, users can save all source and summary chronograms in formats that permit reuse and reanalyses (newick and R "phylo" format), as well as view and compare results graphically, or construct their own graphs using \texttt{datelife}'s chronogram plot generation functions.


<!-- TODO ADD FIGURE: mock example explaining difference between using summary ages and raw ages to calibrate a topology. Then show it in the biological example. This will show what happens if a tree (phylogenetic conflict) or node (age) do not agree.-->


# Benchmark

`datelife`'s code speed was tested on an Apple iMac
with one 3.4 GHz Intel Core i5 processor. <!--No multiprocessor tests were performed-->
We registered variation in computing time of query processing and search through the database relative to number of queried taxon names.
Query processing time increases roughly linearly with number of input taxon names, and
increases considerably if Taxonomic Name Resolution Service (TNRS) is activated.
<!-- Defining tnrs here again, bc we only talk about it two times, so it easy to forget it -->
Up to ten thousand names can be processed and searched in less than 30 minutes with the most time consuming settings.
Once names have been processed as described in methods, a name search through the chronogram database can be performed in less than a minute, even with a very large number of taxon names (Fig. \ref{fig:runtime_main}).
<!--Speed of other datelife workflows? NO, bc it reflects speed from other software mainly. But I will put a sentence about it...
Summarizing `datelife` results processing times: shows sdm speed, clustering methods speed;
generating new trees: shows bladj speed;
Adding dates processing time: reflects more mrbayes speed than anything;
get_bold_otol_tree running time: reflects bold downloading data speed.-->
<!--Speed with different number of chronograms in database? NO-->
<!--Speed of web interface and of r package in computers with different capacities? NOOOO-->
`datelife`'s code performance was evaluated with a set of unit tests designed and
implemented with the R package testthat [@RCoreTeam2018] that were run both locally
with the devtools package [@RCoreTeam2018], and on a public server --via
GitHub, using the continuous integration tool Travis CI (<https://travis-ci.org>). At
present, unit tests cover more than 40% of `datelife`'s code (<https://codecov.io/gh/phylotastic/datelife>). <!-- unit test make code robust, verifiable, debuggable, -->


# Case study

We illustrate the DateLife algorithm using a group within the Passeriform birds encompassing the family of true finches, Fringillidae and allies as case study. The first example analyses 6 bird species and shows all steps of the algorithm. The second example is a real life application
<!--We used `r cite_r("r-references.bib")` for all our analyses.-->

## Small example

We randomly chose 6 bird species related to the family Fringillidae of true finches. The sample includes
two species of cardinals:
the black-thighed grosbeak -- *Pheucticus tibialis* and
the crimson-collared grosbeak -- *Rhodothraupis celaeno*;
three species of buntings:
the yellowhammer -- *Emberiza citrinella*,
the pine bunting --  *Emberiza leucocephalos* and
the yellow-throated bunting --  *Emberiza elegans*;
and one species of tanager, the vegetarian finch -- *Platyspiza crassirostris*.

Processing input names found that *Emberiza elegans* is synonym for *Schoeniclus elegans* in the default reference taxonomy [Open Tree of Life Taxonomy v3.3, June 1, 2021]. For a detailed discussion on the state of the synonym refer to Avibase [@lepage2004avibase; @lepage2014avibase; @avibase-emberiza]. 
DateLife used the processed input names to search the local chronogram database and found 9 matching chronograms in 6 different studies. Three studies matched five input names [@barker2015new; @Hedges2015; @Jetz2012], one study matched four input names [@hooper2017chromosomal] and two studies matched two input names [@barker2013going; @burns2014phylogenetics]. No studies matched all input names. Together, matching chronograms have 28 unique age data points. All nodes have age data.
As fixed tree topology, DateLife used OpenTree's synthetic tree as default and mapped age data to nodes in the tree.
As expected, more inclusive nodes (e.g., node "n1") have more age data than less inclusive nodes (e.g., node "n5").
The processing step allowed discovering five data points for node "n4" that would not have had any data otherwise.
Age summary statistics per node were calculated and tested as secondary calibrations to date the tree topology using the BLADJ algorithm.
Age data for node "n2" was excluded as final calibration because it is older than age data of a more inclusive node.

## Real life application

A college educator wishes to obtain state-of-the-art data on time of evolutionary origin of species belonging to the true finches for their class. They decide to use `datelife` because they are teaching best practices for reproducibility.
Students have the option to go to the website at www.datelife.org and perform an interactive run. However, the educator also wants the students to practice their R skills.
The first step is to run a `datelife` query using the "get species from taxon" flag. This will get all recognised species names within their chosen inclusive taxon. The Fringillidae has `r length(dquery$cleaned_names)` species, according to the Open Tree of Life taxonomy.
Once with a curated set of species taxon names, the next step is to run a `datelife` search that will find all chronograms that contain at least two species names. The algorithm proceeds to prune the trees to keep matching species names on tips only, and transform the pruned trees to pairwise distance matrices.
 There are `r length(dres)` chronograms containing at least two Fringillidae species, published in `r length(unique(names(dres)))` different studies [@barker2013going; @barker2015new; @burns2014phylogenetics; @claramunt2015new; @gibb2015new; @Hedges2015; @hooper2017chromosomal; @Jetz2012; @price2014niche].
The final step is to summarize the available information using two alternative types of summary chronograms, median and SDM. As explained in the "Description" section, data from source chronograms is first summarised into a single distance matrix and then the available node ages are used as fixed node calibrations over a consensus tree topology, to obtain a fully dated tree with the program BLADJ (Fig. \ref{fig:fringillidages}). Median summary chronograms are older and have wider variation in maximum ages than chronograms obtained with SDM.
????????????????? Say some things about the results!
<!-- (Going from a summary distance matrix Using this method is better than using clustering algorithms  -->
<!-- and hierarchical clustering method which appears to push ages back -->

# Cross-validation test

<!--Data from source chronograms can be used to date tree topologies with no branch lengths, as well as trees with branch lengths as relative substitution rates-->
As a form of cross validation, we took the tree topologies from each study and used time of lineage divergence data from all other source chronograms as fixed calibrations with the program BLADJ (Figs. \ref{fig:cv1} to \ref{fig:cv10}).
<!--I'm confused about why you are talking about branch lengths here... did you use branch lengths? Yes, on the PATHd8 cross validation, but you are right, it is confusing, so I removed that first sentence-->

The ages of internal nodes were recovered with a high precision in almost all cases (except for studies 3, and 5; Fig. \ref{fig:cvbladj}).
Maximum tree ages were only recovered in one case (study 2; Fig. \ref{fig:cv2}).
<!-- We need to think about the root! is it comparable across source chronograms?? not all of them have the same sampling... so probably not the same root and thus not comparable. Address this -->
We also demonstrate the usage of PATHd8 [@Britton2007] as a dating method alternative to BLADJ. PATHd8 requires a tree with initial branch lengths which we obtained using `datelife`'s branch length reconstruction algorithm, that searches for DNA sequence data from the Barcode of Life Data System [BOLD; @ratnasingham2007bold] to generate branch lengths. We were able to successfully generate a tree with BOLD branch lengths for all of the Fringillidae source chronograms. However, dating with PATHd8 using congruified calibrations, was only successful in
<!-- two studies when expanding calibrations to make them agree (studies 3 and 5), and from  -->
three cases (studies 3, 5, and 9, shown in Fig. \ref{fig:cvbold}).
<!-- What made it unsuccessful in the other cases? -->
From these, two trees have a different sampling than the original source chronogram, mainly because DNA BOLD  data for some species is absent from the database.
<!-- ??? Node ages or maximum ages?? -->
Maximum ages are quite different from source chronograms, but this might be explained also by the differences in sampling between source chronograms and BOLD trees.
More examples and code used to generate these trees were developed on an open repository that is available for consultation and reuse at <https://github.com/LunaSare/datelife_examples>.

# Discussion

The main goal of `datelife` is to make state-of-the-art information on time of lineage divergence easily accessible for comparison, reuse, and reanalysis, to researchers in all areas of science and with all levels of expertise in the matter. It is an open service that does not require any expert biological knowledge from users --besides the names of the organisms they want to work with, for any of its functionality.

At the time of writing of this manuscript (`r format(Sys.time(), "%b %d, %Y")`), `datelife`'s database has `r length(opentree_chronograms$trees)` chronograms, pulled entirely from OpenTree's database, the Phylesystem [@mctavish2015phylesystem]. A unique feature of OpenTree's Phylesystem is that the community can add new state-of-the-art chronograms any time. As chronograms are added to Phylesystem, they are incorporated into an updated `datelife`'s database that is assigned a new version number, followed by a package release on CRAN. `datelife`'s chronogram database is updated as new chronogram data is added to Phylesystem, at a minimum of once a month and a maximum of every 6 months.
Users can also upload new chronograms to OpenTree themselves, and trigger an update of their local `datelife` database to incorporate the new chronograms, to have them immediately available for analysis.

Incorporation of more chronograms into `datelife`'s database is crucial to improve its services. One option to increase chronogram number in the database is the Dryad data repository.  Methods to automatically mine chronograms from Dryad could be designed and implemented. However, Dryad's metadata system has no information to automatically detect branch length units, and those would still need to be determined manually by a curator.

The largest, and taxonomically broadest, summary chronogram currently available from OpenTree
was constructed using age data from 2,274 published chronograms [@Hedges2015]. However the source chronograms used as input data for this tree are not available in computer readable format for reuse or reanalysis. As this tree is part of datelife's database, the amount of lineages that can be queried using `datelife` (`r length(unique(unname(unlist(lapply(opentree_chronograms$trees, "[[", "tip.label")))))` unique terminal taxa) is substantial.
Access to the input chronograms used to generate the Hedges et al. summary tree would improve measures of uncertainty in DateLife, but they are available only as image files and not as usable data (timetree.org).
<!--IN VIOLATION OF NSF RULES! but I don't suppose we should say that.-->
We would like to emphasize on the importance of sharing chronogram data for the benefit of the scientific community as a whole, into repositories that require expert input and manual curation, such as OpenTree's Phylesystem [@mctavish2015phylesystem].

By default, `datelife` currently summarizes all source chronograms that overlap with at least two species names. Users can exclude source chronograms if they have reasons to do so.
Strictly speaking, the best chronogram should reflect the real time of lineage divergence accurately and precisely.
To our knowledge, there are no good measures to determine independently if a chronogram is better than another.
Some measures that have been proposed are the proportion of lineage sampling and the number of calibrations used [@magallon2010using, @magallon2015metacalibrated].
Several characteristics of the data used for dating analyses as well as from the output chronogram itself, could be used to score quality of source chronograms.
Some characteristics that are often cited in published studies as a measure of improved age estimates as compared to previously published estimates are: quality of alignment (missing data, GC content), lineage sampling (strategy and proportion), phylogenetic and dating inference method, number of fossils used as calibrations, support for nodes and ages, and magnitude of confidence intervals.
DateLife provides an opportunity to capture concordance and conflict among date estimates, which can also be used as a metric for chronogram reliability.

Scientists usually also favor chronograms constructed using primary calibrations (ages obtained from the fossil or geological record) to ones constructed with secondary calibrations (ages coming from other chronograms)[@schenk2016sec]. It has been observed with simulations that divergence times inferred with secondary calibrations are significantly younger than those inferred with primary calibrations in analyses performed with Bayesian inference methods when priors are implemented in similar ways in both analyses [@schenk2016sec]. However, secondary calibrations
can be applied using other dating methods that do not require setting priors, such as penalized likelihood [@sanderson2003r8s], or as fixed ages,
potentially mitigating the bias reported with Bayesian methods.
Certainly, further studies are required to fully understand the effect of using secondary calibrations on time estimates and downstream analyses.

Furthermore, even chronograms obtained with primary fossil data can vary substantially in time estimates between lineages, as observed from the comparison of source chronograms in the Fringillidae example.
<!--I think that one very odd one didn't have any fossils though! Maybe worth mentioning here?-->
This observation is often encountered in the literature
<!-- Different source chronograms often show substantial variation in clade ages  -->
(see, for example, the ongoing debate about crown group age of angiosperms [@sauquet2021age; @barba2018constraining; @magallon2015metacalibrated; @sanderson2001sources; @ramshaw1972time]. For some studies, especially ones based on branch lengths (e.g., studies of species diversification, timing of evolutionary events, phenotypic trait evolution), using a different chronogram may return different results [@title2016macrophylogenies]. Stitching together these chronograms can create a larger tree that uses information from multiple studies, but the effect of uncertainties and errors at this level on downstream analyses is still largely unknown.

Summarizing chronograms might also imply summarizing fundamentally distinct evolutionary hypotheses.
For example, two different researchers working on the same clade both carefully select and argument their choices of fossil calibrations.
Still, if one researcher decides a fossil will calibrate the ingroup of a clade, while another researcher uses the same one to calibrate outside the clade, the resulting age estimates will often differ substantially, as the placement of calibrations as stem or crown group is proved to deeply affect estimated times of lineage divergence [@sauquet2013practical]. Trying to summarize the resulting chronograms into a single one using simple summary statistics can erase many types of relevant information from the source chronograms. Accordingly, the prevailing view is that we should favor time of lineage divergence estimates obtained from a single analysis, using fossil data as primary sources of calibrations, and using fossils that have been widely discussed and curated as calibrations to date other trees, making sure that all data used in the analysis reflect a coherent evolutionary history [@antonelli2017supersmart]. <!-- Will this be implemented in future `datelife` versions? Probably not-->
However, the exercise of summarizing different chronograms has the potential to help getting a single global evolutionary history for a lineage by putting together evidence from different hypothesis. Choosing the elements of the chronograms that we are going to keep and the ones that we are going to discard is key, since we are potentially loosing important parts of the evolutionary history of a lineage that might only be reflected in source chronograms and not on the summary chronogram [@sauquet2021age]. <!-- Example -->


<!-- still requires more research. -->
<!-- Often, there is a large discrepancy in time of lineage divergence estimates across studies from expert knowledge, that is well known and documented for different groups of organisms [e.g., angiosperms; @magallon2015metacalibrated].  -->
<!-- Comparison of available age data for a wide range of organisms shown here suggest that this is a widespread phenomenon that requires further attention. -->
<!-- In the case of the example presented here, most chronograms are new, meaning that we expect that data was generated with high quality and the software used should be very advanced. However, this is just an assumption.  -->

<!-- Still, even if all source chronograms have been generated by excellent standards and using similar methods, the evolutionary history they depict might be very different.  -->

Nonetheless, in ecology and conservation biology, incorporating at least some data on lineage divergence times represents a relevant improvement for testing alternative hypothesis using phylogenetic distance [@Webb2008].
Hence, we integrated into datelife's workflow different ways of estimating node ages in the absence of calibrations and branch length information for taxa lacking this information.
"Making up" branch lengths is an accepted practice in scientific publications: @Jetz2012, created a time-calibrated tree of all 9,993 bird species, where 67% had molecular data and the rest was simulated; @rabosky2018inverse created a time-calibrated tree of 31,536 ray-finned fishes, of which only 37% had molecular data; @smith2018constructing constructed a tree of 353,185 seed plants where only 23% had molecular data.
Obviously, there are risks in this practice!
Taken to the extreme, one could make a fully resolved, calibrated tree of all modern and extinct taxa using a single taxonomy and a single calibration with the polytomy resolution and branch estimation methods.
There has yet to be a thorough analysis of what can go wrong when one extends inferences beyond the data in this way, so we urge caution; we also urge readers to follow the example of many of the large tree papers cited above and make carefully consider the statistical assumptions being made, and assess the consistency of the results with prior work.

<!-- There is evidence that different chronograms will result in very different macroevolutionary histories inferred from them [@title2016macrophylogenies]. However, it is unknown how using very different age data could affect hypothesis testing in other areas of research and more effort is needed in that directions -->

# Conclusions

Divergence time information is key to many areas of evolutionary studies: trait evolution,
diversification, biogeography, macroecology and more. It is also crucial for science communication and education, but generating chronograms is difficult,
especially for those who want to use phylogenies but who are not systematists, or
do not have the time to acquire and develop the necessary knowledge and data curation skills. Moreover, years of primarily public funded research have resulted in vast amounts of chronograms that are already available on scientific publications, but hidden to the public and scientific community for reuse.

<!--Mention again uses shown here and outline potential uses not shown here-->
The `datelife` R package allows easy and fast summarization of publicly available information
on time of lineage divergence. This provides a straightforward way to get an informed idea on the state of knowledge of the time frame of evolution of different regions of the tree of life, and allows identification of regions that require more research or that have conflicting information.
It is available as an R package, or a web-based R shiny app at dates.opentreeloflife.org/datelife.
<!-- It can also summarize public data, or use it to generate new chronograms based on trees provided by the user. -->
Both summary and newly generated trees are useful to evaluate evolutionary hypotheses in different areas of research. The DateLife project helps with awareness of the existing variation in expert time of divergence data, and will foster exploration of the effect of alternative divergence time hypothesis on the results of analyses, nurturing a culture of more cautious interpretation of evolutionary results.


# Availability

`datelife` is free and open source and it can be used through its current website
<http://www.datelife.org/query/>, through its R package, and through Phylotastic's project web portal <http://phylo.cs.nmsu.edu:3000/>.
`datelife`'s website is maintained using RStudio's shiny server and the shiny package open infrastructure, as well as Docker.
`datelife`'s R package stable version `r availability`
for installation from the CRAN repository (<https://cran.r-project.org/package=datelife>)
using the command `install.packages(pkgs = "datelife")` from within R. Development versions
are available from the GitHub repository (<https://github.com/phylotastic/datelife>)
and can be installed using the command `devtools::install_github("phylotastic/datelife")`.


# Supplementary Material

Code used to generate all versions of this manuscript, the biological examples, as well as the benchmark of functionalities are available at [datelifeMS1](https://github.com/LunaSare/datelifeMS1), [datelife_examples](https://github.com/LunaSare/datelife_examples), and [datelife_benchmark](https://github.com/LunaSare/datelife_benchmark) repositories in LLSR's GitHub account.

# Funding

Funding was provided by the US National Science Foundation (NSF) grants ABI-1458603  to Datelife project and DBI-0905606 to the National Evolutionary Synthesis Center (NESCent), and the Phylotastic project Grant ABI-1458572, and the OpenTree grant ABI-1759846.

# Acknowledgements


The DateLife project was born as a prototype tool aiming to provide these services, and was developed over a series of hackathons at the National Evolutionary Synthesis Center, NC, USA [@Stoltzfus2013].
We thank colleagues from the O'Meara Lab at the University
of Tennesse Knoxville for suggestions, discussions and software testing.
The late National Evolutionary Synthesis Center (NESCent), which sponsored hackathons
that led to initial work on this project. The team that assembled `datelife`'s first proof of concept: Tracy Heath, Jonathan Eastman, Peter Midford, Joseph Brown, Matt Pennell, Mike Alfaro, and Luke Harmon.
The Open Tree of Life project that provides the open, metadata rich repository of
trees used for `datelife`.
The many scientists who publish their chronograms in an open, reusable form, and
the scientists who curate them for deposition in the Open Tree of Life repository.
The NSF for funding nearly all the above, in addition to the ABI grant that funded this project itself.

\newpage

# References

\begingroup
\setlength{\parindent}{-0.5in}
\setlength{\leftskip}{0.5in}

<div id="refs" custom-style="Bibliography"></div>
\endgroup
