---
title             : "DateLife: leveraging databases and analytical tools to reveal the dated Tree of Life"
shorttitle        : "DateLife: revealing the dated Tree of Life"

author:
  - name          : "Luna L. SÃ¡nchez Reyes"
    affiliation   : "1,2"
    corresponding : yes    # Define only one corresponding author
    address       : ""
    email         : "sanchez.reyes.luna@gmail.com"
    role:         # Contributorship roles (e.g., CRediT, https://casrai.org/credit/)
      - Data curation
      - Investigation
      - Software
      - Visualization
      - Validation
      - Writing - Original Draft Preparation
      - Writing - Review & Editing
  - name          : "Emily Jane McTavish"
    affiliation   : "1"
    role:
      - Resources
      - Software
      - Writing - Review & Editing
  - name          : "Brian O'Meara"
    affiliation   : "2"
    role:
      - Conceptualization
      - Funding acquisition
      - Methodology
      - Resources
      - Software
      - Supervision
      - Writing - Review & Editing

affiliation:
  - id            : "1"
    institution   : "University of California, Merced"
  - id            : "2"
    institution   : "University of Tennessee, Knoxville"

authornote: |
  School of Natural Sciences, University of California, Merced, Science and Engineering Building 1.

  Department of Ecology and Evolutionary Biology, University of Tennessee, Knoxville, 425 Hesler Biology Building, Knoxville, TN 37996, USA.

abstract: |
  Time of evolutionary origin is fundamental for research in the natural sciences, as well as for education, science communication and policy.
  Despite an increased availability of fossil and molecular data, and time-efficient analytical techniques, achieving a high-quality reconstruction of time of evolutionary origin as a phylogenetic tree with branch lengths proportional to absolute time (chronogram), is still a difficult and time-consuming task for a majority of interested parties.
  Yet, the amount of published chronograms has increased significantly in the past two decades, and a non-negligeable proportion of these data have been steadily accumulating in public, open databases such as TreeBASE and Open Tree
  of Life, exposing a wealth of expertly-curated and peer-reviewed data on time of evolutionary origin in a programatic and reusable way, for a large quantity and diversity of organisms.
  This trend results from intensive and localized efforts for improving data sharing practices, as well as incentivizing open science in biology. Despite these trends, accessibility for time data is not that good.
  R has become a widely used element of the biological data analyst toolkit. Hence,to improve accessibility of time data we developed an R package that access these data and eases interaction, reanalysis and reuse of it, incorporation of these data into the evolutionary workflow. 


  Here we present `datelife`, a service implemented as an R package and a web site
  (www.datelife.org) for increased accessibility, efficient reuse, summary and reanalysis of expert, peer-reviewed, public data on time of evolutionary origin.

  Main results:
    1. blah
    2. blah
    3. blah

  Results in a general context:
  All source and summary chronograms can be saved in formats that permit easy reuse and reanalysis. Summary and newly generated trees are potentially useful to evaluate evolutionary hypothesis in different areas of research in biology.
  How well this trees work for this purpose still needs to be tested.

  `datelife` will be useful to increase awereness on the existing variation in expert time of divergence data, and might foster exploration of the effect of alternative divergence time hypothesis on the results of analyses, nurturing a culture of more cautious interpretation of evolutionary results.

  <!-- https://tinyurl.com/ybremelq -->

keywords          : "Tree; Phylogeny; Scaling; Dating; Ages; Divergence times; Open Science; Congruification; Supertree; Calibrations"
wordcount         : "571"

bibliography      : ["paper_references.bib"]

floatsintext      : no
figurelist        : no
tablelist         : no
footnotelist      : no
linenumbers       : yes
mask              : no
draft             : no

documentclass     : "apa6"
classoption       : "man"
output            : papaja::apa6_pdf

nocite: |
  @barker2012going, @barker2015new, @burns2014phylogenetics, @claramunt2015new, @gibb2015new, @Hedges2015, @hooper2017chromosomal, @Jetz2012, @price2014niche
---

```{r setup, include = FALSE}
library("papaja")
r_refs("paper_references.bib")
```

```{r analysis-preferences}
# Seed for random number generation
set.seed(50)
knitr::opts_chunk$set(cache.extra = knitr::rand_seed)
data(opentree_chronograms)
rpackage_available <- FALSE
availability <- ifelse(rpackage_available, "is available", "will be available")
dq <- datelife::make_datelife_query(input = "fringillidae", get_spp_from_taxon = TRUE)
```

# Introduction


# Implementation/Description/Workflow

The `datelife` workflow builds off of functions from several R packages (rotl [@Michonneau2016], ape [@Paradis2004],
geiger [@Harmon2008], paleotree [@Bapst2012a], bold [@Chamberlain2018], phytools [@Revell2012], taxize [@Chamberlain2013; @Chamberlain2018], phyloch [@Heibl2008], and phylocomr [@Ooms2018]).

The basic `datelife` workflow is shown in figure \ref{fig:workflow} and consists of:

\begin{enumerate}
\item A user providing at least two taxon names as input. Taxon names can be given at the species level or above. The user can provide them as a simple comma separated character string, or as tip labels on a tree. The tree can be provided in newick format (as a character string) or as a "phylo" R object, and can be with or without branch lengths.
\item A name search is then performed across the chronogram database. Source trees with at least two matching input names are selected, and taxa that do not match the original query are dropped from these source trees.
<!-- The resulting pruned source chronograms are hereafter referred as source chronograms.  --> <!--# This was generating some confussion, so I'm calling them pruned source chronograms now-->
Finally, each pruned source chronogram is transformed to a patristic distance matrix. This format facilitates and greatly speeds up all further analyses and summarizing algorithms. For the sake of keeping track of authorship later, the matrices are associated to the citation of the original study.
\item  At this point, the user can already obtain different summary information from their query, allowing the user to take informed decisions for the next steps of the workflow. Types of summary information provided are: a) all pruned source chronograms, b) age of the mrca (most recent common ancestor) of the pruned source chronograms, c) citations of studies where pruned source chronograms were originally published, d) a summary table with all of the above, e) a single summary chronogram of all or a subset of pruned source chronograms, f) a report of succesful matches of input taxon names across pruned source chronograms, and g) the single pruned source chronogram with the most taxa matched.
\item  Then, source chronogram data can be used as secondary calibration points to date a tree with or without branch lengths containing some or all taxon names from the initial query. %<!--, a taxonomic tree-->
\item  If there is no information available for any queried taxa, users can also create both age and phylogenetic data for this missing taxa with a variety of algorithms described below.
\item  Finally, users can easily save all source and summary chronograms in formats that permit easy reuse and reanalyses (newick and R "phylo" format), as well as view and compare results graphically, or construct their own graphs using \texttt{datelife}'s graphic generation functions.
\end{enumerate}

# Benchmark

`datelife`'s code speed was tested on an Apple iMac
with one 3.4 GHz Intel Core i5 processor. <!--No multiprocessor tests were performed-->
We registered variation in computing time of query processing and search through the database relative to number of queried taxon names.
Query processing time increases roughly linearly with number of input taxon names, and
increases considerably if the taxonomic name resolution service [TNRS; @Boyle2013] is activated.
<!-- Defining tnrs here again, bc we only talk about it two times, so it easy to forget it -->
Up to ten thousand names can be processed and searched in less than 30 minutes with the most time consuming settings.
Once names have been processed as described in methods, a name search through the chronogram database can be performed in less than a minute, even with a very large number of taxon names (Fig. \ref{fig:runtime1}).
<!--Speed of other datelife workflows? NO, bc it reflects speed from other software mainly. But I will put a sentence about it...
Summarizing `datelife` results processing times: shows sdm speed, clustering methods speed;
generating new trees: shows bladj speed;
Adding dates processing time: reflects more mrbayes speed than anything;
get_bold_otol_tree running time: reflects bold downloading data speed.-->
<!--Speed with different number of chronograms in database? NO-->
<!--Speed of web interface and of r package in computers with different capacities? NOOOO-->
`datelife`'s code performance was evaluated with a set of unit tests designed and
implemented with the R package testthat [@RCoreTeam2018] that were run both locally
with the devtools package [@RCoreTeam2018], and on a public server --via
GitHub, using the continuous integration tool Travis CI (<https://travis-ci.org>). At
present, unit tests cover more than 30% of `datelife`'s code (<https://codecov.io/gh/phylotastic/datelife>). <!-- unit test make code robust, verifiable, debuggable, -->


# Results

<!--We used `r cite_r("r-references.bib")` for all our analyses.-->

## Case study

We illustrate the `datelife` workflow using the family of true finches, Fringillidae as an example. To contextualize, a college educator wishes to know the state-of-the-art on time of evolutionary origin of species belonging to the true finches. 
One option is to go to the website at www.datelife.org and perform an interactive run. However, the educator wants the students to practice their R skills. 
The first step is to run a higher-taxon-name 'datelife` query. This will get taxon names for all recognised species within any higher taxon. The Fringillidae has `r length(dq$cleaned_names)` species, according to the Open Tree of Life taxonomy.
Once with a curated set of query taxon names, the next step is to run a `datelife` search. This will find all chronograms that contain at least two queried taxon names, and will save the information on time of lineage divergence as (an R "data frame") table.
 There are `r length(dr)` chronograms containing at least two Fringillidae species, published in `r length(unique(names(dr)))` different studies (Fig. \ref{fig:schronograms1}).
<!-- In case we want to cite the studies in here, but they are already cited in the figure caption: [Fig. \ref{fig:schronograms1}; @barker2012going; @barker2015new; @burns2014phylogenetics; @claramunt2015new; @gibb2015new; @Hedges2015; @hooper2017chromosomal; @Jetz2012; @price2014niche]. -->
The final step is to summarize the available information using the two alternative types of summary chronograms, median and SDM. As explained in the "Description" section, data from source chronograms is first summarised into a single distance matrix (using the median and the SDM method respectively) and then the available node ages are used as fixed ages over a consensus tree topology, to obtain a fully dated tree with the program BLADJ (Fig. \ref{fig:summaries}). Median summary chronograms are older and have wider variation in maximum ages than chronograms obtained with SDM. With both methods, ages are generally consistent with source ages, but there are some biological examples in which this is not true (see Discussion).
<!-- (Going from a summary distance matrix Using this method is better than using clustering algorithms  -->
<!-- and hierarchical clustering method which appears to push ages back -->

## Cross-validation test
Data from source chronograms can be also used to date tree topologies with no branch lengths, as well as trees with branch lengths as relative substitution rates (Figs. \ref{fig:cvbladj} and \ref{fig:cvbold}). As a form of cross validation, we took tree topologies from each study and calibrated them using time of lineage divergence data from all other source chronograms. In the absence of branch lengths, the ages of internal nodes were recovered with a high precision in almost all cases (except for studies 3, and 5; Fig. \ref{fig:cvbladj}).
Maximum tree ages were only recovered in one case (study 2; Fig. \ref{fig:cvbladj}).
<!-- We need to think about the root! is it comparable across source chronograms?? not all of them have the same sampling... so probably not the same root and thus not comparable. Address this -->
We also demonstrate the usage of PATHd8 [@Britton2007] as an alternative method to BLADJ. For this, we run a `datelife` branch length reconstruction that searches for DNA sequence data from the Barcode of Life Data System [BOLD; ratnasingham2007bold] to generate branch lengths. We were able to successfully generate a tree with BOLD branch lengths for all of the Fringillidae source chronograms. However, dating with PATHd8 using congruified calibrations, was only successful in
<!-- two studies when expanding calibrations to make them agree (studies 3 and 5), and from  -->
three cases (studies 3, 5, and 9, shown in Fig. \ref{fig:cvbold}). From these, two trees have a different sampling than the original source chronogram, mainly because DNA BOLD  data for some species is absent from the database. Maximum ages are quite different from source chronograms, but this might be explained also by the differences in sampling between source chronograms and BOLD trees.
More examples and code used to generate these trees were developed on an open repository that is available for consultation and reuse at <https://github.com/LunaSare/datelife_examples>.

# Discussion

# Conclusions

# Availability

`datelife` is free and open source and it can be used through its current website
<http://www.datelife.org/query/>, through its R package, and through Phylotastic's project web portal <http://phylo.cs.nmsu.edu:3000/>.
`datelife`'s website is maintained using RStudio's shiny server and the shiny package open infrastructure, as well as Docker.
`datelife`'s R package stable version `r availability`
for installation from the CRAN repository (<https://cran.r-project.org/package=datelife>)
using the command `install.packages(pkgs = "datelife")` from within R. Development versions
are available from the GitHub repository (<https://github.com/phylotastic/datelife>)
and can be installed using the command `devtools::install_github("phylotastic/datelife")`.


# Supplementary Material

Code used to generate all versions of this manuscript, the biological examples, as well as the benchmark of functionalities are available at [datelifeMS1](https://github.com/LunaSare/datelifeMS1), [datelife_examples](https://github.com/LunaSare/datelife_examples), and [datelife_benchmark](https://github.com/LunaSare/datelife_benchmark) repositories in LLSR's GitHub account.

# Funding

Funding was provided by the US National Science Foundation (NSF) grants ABI-1458603  to Datelife project and DBI-0905606 to the National Evolutionary Synthesis Center (NESCent), and the Phylotastic project Grant ABI-1458572.

# Acknowledgements

We thank colleagues from the O'Meara Lab at the University
of Tennesse Knoxville for suggestions, discussions and software testing.
The late National Evolutionary Synthesis Center (NESCent), which sponsored hackathons
that led to initial work on this project. The team that assembled `datelife`'s first proof of concept: Tracy Heath, Jonathan Eastman, Peter Midford, Joseph Brown, Matt Pennell, Mike Alfaro, and Luke Harmon.
The Open Tree of Life project that provides the open, metadata rich repository of
trees used for `datelife`.
The many scientists who publish their chronograms in an open, reusable form, and
the scientists who curate them for deposition in the Open Tree of Life repository.
The NSF for funding nearly all the above, in addition to the ABI grant that funded this project itself.

\newpage

# References

\begingroup
\setlength{\parindent}{-0.5in}
\setlength{\leftskip}{0.5in}

<div id="refs" custom-style="Bibliography"></div>
\endgroup
