---
output:
  pdf_document: default
  html_document: default
---

 The primary information needed for these time estimates comes from the fossil record. Coupled with phylogenies with branch lengths based on molecular and/or morphological data, the time of divergence of extant and extinct lineages can be reconstructed with molecular dating methods.

Wide interest from the scientific community to make information from phylogenies in general and chronograms in particular available for consultation and reuse has spurred the creation of public platforms with various goals and characteristics. TreeBASE [@piel2009treebase; @vos2012nexml], the Dryad repository (<http://datadryad.org/>), and the Open Tree of Life (OToL) repository [@Hinchliff2015; @mctavish2015phylesystem] are platforms that store and make available published phylogenies and chronograms for easy scientific reuse. Trees in all of them can be queried using automatised web procedures, which permit personalized, large scale searches that are also very fast.
OToL stores trees with branch length information from a wide range of living organisms, implementing a metadata structure that stores the branch length units (i.e., time or relative susbtitution rates). Treebase and Dryad repositories also contain trees from all groups of life, but the former did not store branch length information until recently (and lacks consistent metadata on what any branch lengths stored mean) and Dryad stores many other types of biological data using metadata that does not allow automatic distinction of types of trees and branch length units, impairing the automatised access to time of lineage divergence information.

Besides keeping a repository to easily store and share expert phylogenetic and chronogram knowledge, OToL also has the primary goal of synthesising all trees in their repository to expose to the community a single tree of all life depicting the phylogenetic relationships among known lineages.
All or parts of this synthetic tree can be reused for any purpose. However, it currently only focus on synthesizing tree topology, meaning that it does not expose branch length data of any type.
<!-- let alone time of divergence data from their stored chronograms. -->
The Timetree of Life project focuses on the synthesis of a single chronogram of life [@Hedges2006] and presents a very accessible, attractive interface. However, the thousands of chronograms this NSF-funded project have compiled for synthesis are only publicly available for visual examination in their website or for download as images, but large scale download remains prohibited by their site. The latest version of their synthetic chronogram [@Kumar2017] can be queried only through their website in a non-automatised fashion, and only subsets of it can be reused for analyses with the permission of the authors.
Other platforms such as SuperSmart [@antonelli2017supersmart] and phylogenerator [@pearse2013phylogenerator] are focused in automatised _de novo_ chronogram inference, by reusing DNA sequence data to reconstruct phylogenetic trees. However, expert fossil information necessary for subsequent molecular dating analyses still needs to be compiled and curated by the user, rendering them a challenging tool to obtain data on time of lineage divergence for the non-specialist. Additionally, neither Supersmart nor phylogenerator are designed to provide information from already created expert chronograms, since it might not be their objective.

<!-- discussion:
The Angiosperm Phylogeny Website [APW; @stevens2001apw] also specializes in constructing a single synthetic tree, but it focuses in showing phylogenetic relationships between seed plant families only, and it does not store or make available the source trees.

Platforms dedicated to store data on fossils and fossil calibrations: PBDB and The fossil calibration database
can provide data on time of lineage divergence, but it is often not in a phylogenetic context.

To our knowledge, there is no system allowing the comparison of available source information (apw does this a little bit, but it is a verbal comparison more than a graphical or visual comparison). Other types of technical comparisons: which trees are better (which trees represent reality better), but this is of course a very difficult issue.
 -->

<!-- , for the benefit of the broader community. -->
<!-- as well as tools for comparison of source and summary trees.  NOT YET u_u-->
<!-- R packages for benchmarking of functionalities and demonstrating services were also developed and made available.>

<!--(BOLD and OToL species names, are they standardized with each other? I think yes-->

\begin{center}
\textsc{Description}
\end{center}

A `datelife` search currently accepts scientific names only. It can be any named clade or binomial specific. <!--common names (implemented in phylotastic)-->
The search is performed at the species level, so when input names correspond
to named clades, `datelife` pulls all accepted species names within the
clade from OToL's reference taxonomy [@rees2017automated] and uses all of those.
Searches at the infraspecies level are not currently allowed, so input names belonging to subspecies or any other infraspecific category are collapsed to the species level.
`datelife` processes input names with the taxon name resolution service [TNRS; @Boyle2013],
<!--which "corrects misspelled names and authorities, standardizes variant spellings,
and converts nomenclatural synonyms to accepted names" (Boyle et al. 2013)-->
which corrects potentially misspelled names and typos, and standardizes spelling
variations and synonyms <!--to a taxonomic reference of users choosing, default
to otol taxonomy -->, increasing the probability of correctly finding the
queried taxa in `datelife`'s chronogram database.

Then, a name matching procedure is performed across `datelife`'s chronogram database which is assembled from OToL's tree repository [known as the phylesystem; @mctavish2015phylesystem]. Compared to other existing open tree repositories OToL's metadata rich tree store is the only one that supports search, identification, and handling of chronograms in an automatised fashion. Also, the tip names in their stored trees correspond to scientific names at or below the species level. Finally, all their chronograms come from peer-reviewed published studies generated by specialists in the targeted lineages, arguably representing expert knowledge on time of lineage divergence.
<!-- The TreeBASE repository stores trees without branch lengths, so chronograms stored there do not provide relevant information on time of lineage divergence. The Dryad repository stores all types of biological data besides chronograms, and their metadata does not allow differentiating between branch lengths units.  -->
<!-- branch length units that might represent other types of time of divergence that are not proportional to absolute (geologic) time.  -->

Information from pruned source chronograms can then be summarised with simple summary statistics of tree branch length, such as the median or the mean. A  much slower, Super Distance Matrix (SDM) approach for supertree reconstruction with branch lengths [@Criscuolo2006] is also implemented via the ape package [@Paradis2004].<!-- and the hierarchical clustering method of ttol??-->
<!-- describe what SDM does and why clustering does not work here -->
The resulting summary patristic distance matrix could be clustered with classic algorithms to return a tree. However, the resulting trees are often non-ultrameric and do not reflect the source chronogram data (see datelife_examples package). Instead, we obtained a distribution of age data from the summary matrix available for nodes on a consensus tree. The Branch Length Adjuster (BLADJ) algorithm [@Webb2008] was then used to fix known node ages and then distribute nodes with unknown ages evenly over the consensus tree, minimizing age variance in the resulting chronogram.

For tree dating, the congruification algorithm described by @Eastman2013
is implemented to find shared nodes between trees (congruent nodes). The ages of these nodes are then used as calibrations to date any given tree. Currently implemented methods for tree dating are BLADJ, MrBayes [@Huelsenbeck2001; @Ronquist2003] and PATHd8 [@Britton2007], a non-clock, rate-smoothing dating method.

<!-- and simulations are performed with various algorithms: -->
<!-- MrBayes can be used when adding taxa at -->
<!-- random, following a reference taxonomy or a topological constraint. It draws ages -->
<!-- from a pure birth model, as implemented by Jetz and collaborators [-@Jetz2012]. -->


<!-- Details on each step are further developed in `datelife`'s R package documentation -->
<!-- `datelife workflow` vignette at (<https://LINK>). -->

\begin{center}
\textsc{Benchmark}
\end{center}
<!--**Benchmark: Testing DateLife computing performance**-->
<!-- Good to test 3 features on a package: performance, speed and scalability-->



\begin{center}
\textsc{Example}
\end{center}
<!--**Biological example: Testing DateLife accuracy**-->
<!--Bird (or reptile) chronograms, too long time...
finches is good
nothofagus
Look for all chronograms containing any birds
Or, look for chronograms containing small and old lineages
Determine which clade of birds has the more chronograms (have been dated
more times) and use that as biological example. Fringillidae is a good candidate
Find a clade with at least one chronogram containing all clade's species.
(Penguins look good, but they are giving weird results in SDM)

Remove this chronogram from datelife Results.

Make sdm and median trees and Compare

add taxa with different methods and Compare

Use ltts to compare for now.

think of a test to compare trees, topology- and date-wise

test improvement in synthetic otol tree after using bold sequences. Is it really better resolved? Probably not.
Do not include this analysis in paper, maybe in examples, but i think it is not worthy.
-->


<!--Show code here, in the paper? Or show it only in the examples? Or only in the vignette? Write down what about datelife will be shown in each of these.
Paper should show outputs from web site and package.
Vignette will show the R code used to generate the outputs displayed here and in the website.
datelife examples will explain more on what is done in every step.
-->

\begin{center}
\textsc{Flaws, Limitations and Prospects}
\end{center}


\begin{center}
\textsc{Conclusions}
\end{center}



<!-- `datelife` is also part of the reproducibility of science movement. It strives to facilitate usage of public data sets as well as encouraging the making of information publicly available. Acknowledging source information and describing workflows and analyses. -->

<!-- it allows exploration of alternative taxonomic and phylogenetic schemes; -->
<!-- it allows rapid ; -->

\begin{center}
\textsc{Availability}
\end{center}



\begin{center}
\textsc{Supplementary Material}
\end{center}

<!-- Vignettes?
Dryad also?? since it is all in github already, maybe not necessary-->

\begin{center}
\textsc{Funding}
\end{center}



<!-- Any separate funding from the University of Tennessee, Knoxville?? -->


\begin{center}
\textsc{Acknowledgements}
\end{center}
